<!DOCTYPE html>

<html lang="en">

    <head>
        <title>Ruogu Fang 方若谷</title>
        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
        <meta name="author" content="ruogu fang">
        <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
        <meta name="description" content="Ruogu Fang Homepage" />
        <meta name="keywords" content="Ruogu Fang, medical image, machine learning, deep learning, computer vision, health analysis, SMILE group, big data" />
        <link rel="shortcut icon" href="img/misc/ufl.png">

        <!--CSS styles-->
        <link rel="stylesheet" href="css/bootstrap.css">
        <link rel="stylesheet" href="css/font-awesome.min.css">
        <link rel="stylesheet" href="css/perfect-scrollbar-0.4.5.min.css">
        <link rel="stylesheet" href="css/magnific-popup.css">
        <link rel="stylesheet" href="css/style.css">
        <link id="theme-style" rel="stylesheet" href="css/styles/default.css">
        <!--/CSS styles-->

        <!--Javascript files-->
        <script type="text/javascript" src="js/jquery-1.11.3.min.js"></script>
        <script type="text/javascript" src="js/TweenMax.min.js"></script>
        <script type="text/javascript" src="js/jquery.touchSwipe.min.js"></script>
        <script type="text/javascript" src="js/jquery.carouFredSel-6.2.1-packed.js"></script>
        <script type="text/javascript" src="js/modernizr.custom.63321.js"></script>
        <script type="text/javascript" src="js/jquery.dropdownit.js"></script>
        <script type="text/javascript" src="js/ScrollToPlugin.min.js"></script>
        <script type="text/javascript" src="js/bootstrap.min.js"></script>
        <script type="text/javascript" src="js/jquery.mixitup.min.js"></script>
        <script type="text/javascript" src="js/masonry.min.js"></script>
        <script type="text/javascript" src="js/perfect-scrollbar-0.4.5.with-mousewheel.min.js"></script>
        <script type="text/javascript" src="js/jquery.nicescroll.min.js"></script>
        <script type="text/javascript" src="js/magnific-popup.js"></script>
        <script type="text/javascript" src="js/custom.js"></script>
        <!--/Javascript files-->
    </head>

    <body>
        <div id="wrapper">

            <!--Sidebar-->
            <a href="#sidebar" class="mobilemenu"><i class="fa fa-reorder"></i></a>
            <div id="sidebar">
                <div id="sidebar-wrapper">
                    <div id="sidebar-inner">

                        <!-- Profile/logo section-->
                        <div id="profile" class="clearfix">
                            <div class="portrate hidden-xs"></div>
                            <div class="title">
                                <h2>Ruogu Fang Reanna 方若谷</h2>
                                <h3>University of Florida</h3>
                            </div>
                        </div>
                        <!-- /Profile/logo section-->

                        <!-- Main navigation-->
                        <div id="main-nav">
                            <ul id="navigation">
                                <li>
                                  <a href="#biography">
                                    <i class="fa fa-user"></i>
                                    <div class="text">About Me</div>
                                  </a>
                                </li>

                                <li>
                                  <a href="#team">
                                    <i class="fa fa-home"></i>
                                    <div class="text">Team</div>
                                  </a>
                                </li>

                                <li>
                                  <a href="#research">
                                    <i class="fa fa-book"></i>
                                    <div class="text">Research</div>
                                  </a>
                                </li>

                                <li>
                                  <a href="#publications">
                                    <i class="fa fa-edit"></i>
                                    <div class="text">Publications</div>
                                  </a>
                                </li>

                                <li>
                                  <a href="#teaching">
                                    <i class="fa fa-clock-o"></i>
                                    <div class="text">Teaching</div>
                                  </a>
                                </li>

                                <li>
                                  <a href="#software">
                                    <i class="fa fa-tv"></i>
                                    <div class="text">Software</div>
                                  </a>
                                </li>

                                <li>
                                  <a href="#gallery">
                                    <i class="fa fa-picture-o"></i>
                                    <div class="text">Gallery</div>
                                  </a>
                                </li>

                                <li>
                                  <a href="#join">
                                      <i class="fa fa-send"></i>
                                      <div class="text">Join</div>
                                  </a>
                                </li>

                                <li>
                                  <a href="#contact">
                                    <i class="fa fa-calendar"></i>
                                      <div class="text">Contact Me</div>
                                  </a>
                                </li>

                                <li class="external">
                                  <a href="cv_rfang.pdf" target="_blank">
                                    <i class="fa fa-download"></i>
                                      <div class="text">Download CV</div>
                                  </a>
                                </li>
                            </ul>
                        </div>
                        <!-- /Main navigation-->

                        <!-- Sidebar footer -->
                        <div id="sidebar-footer">
                            <div class="social-icons">
                                <ul>
                                    <li><a href="https://scholar.google.com/citations?user=LVb46zEAAAAJ&hl=en" target="_blank"><i class="fa fa-google"></i></a></li>
                                    <!--<li><a href="#" ><i class="fa fa-facebook"></i></a></li>-->
                                    <li><a href="https://twitter.com/RuoguFang"><i class="fa fa-twitter"></i></a></li>
                                    <li><a href="https://www.linkedin.com/in/ruogu-fang-a015bb15" target="_blank"><i class="fa fa-linkedin"></i></a></li>
                                </ul>
                            </div>
                          <div id="copyright">Copyright @2020 Ruogu Fang</div>
                        </div>
                        <!-- /Sidebar footer -->
                    </div>
                </div>
            </div>
            <!--/Sidebar-->

            <!--Main-->
            <div id="main">

                <!--About me-->
                <div id="biography" class="page home" data-pos="home">

                    <!--Front Page Header-->
                    <div class="pageheader">
                        <div class="headercontent">
                            <div class="section-container">
                                <div class="row">
                                    <div class="col-sm-2 visible-sm"></div>
                                    <div class="col-sm-8 col-md-5">
                                        <div class="biothumb">
                                            <img alt="image" src="img/personal/personal-ufl.jpg" class="img-responsive">
<!--
                                            <div class="overlay">
                                                <h1 class="">Ruogu (Reanna) Fang 方若谷</h1>
                                                <ul class="list-unstyled">
                                                    <li>J. Crayton Pruitt Family<br>Department of Biomedical Engineering</li>
                                                    <li>University of Florida</li>
                                                </ul>
                                            </div>
-->
                                        </div>
                                        &nbsp;
                                        <p style="color:red"><b>PhD Positions Open</b>: We are actively recruiting PhD students in Biomedical Engineering, Electrical and Computer Engineering, and Computer Science! Multiple fully funded PhD positions are open on artificial intelligence in medicine, brain-inspired AI, and AI for neurological diseases.</p>
                                    </div>
                                    <div class="clearfix visible-sm visible-xs"></div>
                                    <div class="col-sm-12 col-md-7">
                                        <img alt="image" src="img/misc/smilebanner.png" class="img-responsive" style="padding-top:30px;padding-bottom:30px">
                                    </div>
                                    <div class="clearfix visible-sm visible-xs"></div>
                                    <div class="col-sm-12 col-md-7">
                                        <h3 class="title">Bio</h3>
                                        <p>An AI researcher in medicine and healthcare, <b>Dr. Ruogu Fang</b> is a tenure-track Assistant Professor in the J. Crayton Pruitt Family Department of Biomedical Engineering at University of Florida. Her research spans Artificial Intelligence, Brain Dynamics, and Medical Image Analysis. She focuses on questions such as: How to use machine learning to quantify brain dynamics, predict treatment outcome, and precision theurapeutics. Fang's current research is rooted in the confluence of AI and multimodal medical imaging. She is the recipient of <a href="https://www.nsf.gov/awardsearch/showAward?AWD_ID=1564892">NSF Research Iniatiation Iniative (CRII) Award</a>, <a href="https://www.nsf.gov/awardsearch/showAward?AWD_ID=1908299&HistoricalAwards=false">NSF CISE IIS Award</a>, <a href="http://chenlab.ece.cornell.edu/people/ruogu/kin_verify">Best Paper Award from IEEE International Conference on Image Processing</a>, Ralph Lowe Junior Faculty Enhancement Award from <a href="https://www.orau.org/">Oak Ridge Associated Universities (ORAU)</a>, and Robin Sidhu Memorial Young Scientist Award from Society of Brain Mapping and Therapeutics, among the others. Fang's research has been featured by <a href="https://www.forbes.com/sites/jackierocheleau/2020/11/25/scientists-are-looking-into-the-eyes-of-patients-to-diagnose-parkinsons-disease/">Forbes</a>, <a href="https://press.rsna.org/timssnet/media/pressreleases/14_pr_target.cfm?ID=2229">RSNA</a>, and published in Lancet Digital Health. Dr. Fang directs SMILE lab, standing for Smart Medical Informatics Learning and Evaluation, which also reflects for her expectation for every member in the lab to smile while exploring artificial intelligence in medicine and neuroimaging.</p>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>
                    <!--/Front Page Header-->

                    <!--Front Page Content-->
                    <div class="pagecontents">

                    <!--News-->
                    <div class="section color-2">
                            <div class="section-container">
                                <div class="row">

                                    <div class="col-md-10 col-md-offset-1">
                                        <div class="title text-center">
                                            <h3>News</h3>
                                        </div>
                                    <tr>
                                        <td>
                                            <div style="height: 350px; overflow: auto;">
                                                <ul>
                                                     <li style="margin-bottom:10px">12/2020: Dr. Fang, together with <a href="https://www.bme.ufl.edu/dept-member/ding_mingzhou/">Dr. Mingzhou Ding</a> and <a href="https://people.clas.ufl.edu/akeil/">Dr. Andreas Keil</a> have been awarded <a href="https://news.ufl.edu/2020/12/artificial-intelligence-research-catalyst-fund-/">UF Research Artificial Intelligence Research Catalyst Fund</a> on the project "VCA-DNN: Neuroscience-Inspired Artificial Intelligence for Visual Emotion Recognition"! [<a href="https://www.bme.ufl.edu/fang-and-ding-selected-for-uf-catalyst-fund/" target="_blank">Link</a>]</li>
                                                    
                                                    <li style="margin-bottom:10px">12/15/2020: Our paper on "Modular machine learning for Alzheimer's disease classification from retinal vasculature" has been accepted by Nature Scientific Reports! [<a href="https://www.nature.com/articles/s41598-020-80312-2">Link</a>]</li>
                                                    <li style="margin-bottom:10px">12/2020: Dr. Fang has been interviewed by <a href="https://www.forbes.com/sites/jackierocheleau/2020/11/25/scientists-are-looking-into-the-eyes-of-patients-to-diagnose-parkinsons-disease/?sh=557314f52faa">Forbes</a>, <a href="https://press.rsna.org/timssnet/media/pressreleases/14_pr_target.cfm?ID=2229">RSNA</a>, <a href="https://www.diagnosticimaging.com/view/eye-exam-plus-ai-could-replace-advanced-imaging-in-parkinson-s-detection">Diagnostic Imaging</a> on her research about <a href="https://www.eng.ufl.edu/newengineer/in-the-headlines/scientists-are-looking-into-the-eyes-of-patients-to-diagnose-parkinsons-disease/">AI for Parkinson's Disease Diagnosis via Eye Exam.</a></li>
                                                    
                                                    <li style="margin-bottom:10px">11/2020: Dr. Fang is interviewed by UFII on AI for brain functions. [<a href="https://informatics.research.ufl.edu/engineering-professor-uses-ai-to-study-the-brain/">Link</a>]</li>
                                                    
                                                    <li style="margin-bottom:10px">10/30/2020: Paper published in Brain Simulation on <a href="https://www.sciencedirect.com/science/article/pii/S1935861X20302680">Machine learning and individual variability in electric field characteristics predict tDCS treatment response</a>". [<a href="https://www.brainstimjrnl.com/article/S1935-861X(20)30268-0/pdf">PDF</a>]</li>
                                                    
                                                    <li style="margin-bottom:10px">10/14/2020: Congratulations to SMILer Gianna Sweeting on receving the Engineering College Scholarship from the Herbert Wertheim College of Engineering (HWCOE)! [<a href="https://www.bme.ufl.edu/bme-student-selected-as-a-hwcoe-scholarship-recipient/">News</a>]</li>
                                                    
                                                    <li style="margin-bottom:10px">9/4/2020: Congratulations to BME alumna Yao Xiao Ph.D on receiving the prestigious 2020 BMES Career Development Award! [<a href="https://www.bme.ufl.edu/bme-alumna-awarded-bmes-2020-career-development-award/">News</a>]</li>
                                                    <li style="margin-bottom:10px">8/5/2020: Congrats on Skylar Stolte on her abstract entitled "Artificial Intelligence For Characterizing Heart Failure In Cardiac Magnetic Resonance Images" being accepted by <a href="https://professional.heart.org/professional/EducationMeetings/MeetingsLiveCME/ScientificSessions/UCM_316900_Scientific-Sessions.jsp">American Heart Association Scientific Sessions</a> 2020! </li> 
                                                    <li style="margin-bottom:10px">7/16/2020: Our paper on "Physiological wound assessment from coregistered and segmented tissue hemoglobin maps" is published at Journal of the Optical Society of America A. It is a collaborative work with Dr. Anuradha Godavarty at Florida International University. [<a href="https://www.osapublishing.org/josaa/abstract.cfm?uri=josaa-37-8-1249">Paper</a>]</li>
                                                    <li style="margin-bottom:10px">7/6/2020: Congratulaions to SMILE Member PhD student <a href="https://rocmind.com/">Peng Liu</a> on receving <a href="https://research.ufl.edu/informatics-institute.html">UFII</a> Graduate Student Fellowship on his project on Neuroscience-Inspired Artificial Intelligence![<a href="https://www.bme.ufl.edu/bme-student-receives-ufii-graduate-student-fellowship/">News</a>]</li></li>
                                                    <li style="margin-bottom:10px">6/25/2020: Yao Xiao presented at the 2020 Annual Meeting of the Society for Imaging Informatics in Medicine (SIIM) on 'Multi-Series CT Image Super-Resolution by using Transfer Generative Adversarial Network'! [<a href="https://cdn.ymaws.com/siim.org/resource/resmgr/siim20/abstracts-research/xiao-fang_multi-series_ct_im.pdf" target="_blank">Link</a>]</li>
                                                    <li style="margin-bottom:10px">6/12/2020: SMILer Kyle B. See is highlighted in BME Student Spotlight! [<a href="https://www.bme.ufl.edu/student-spotlight-kyle-see/" target="_blank">News</a>]</li>
                                                    <li style="margin-bottom:10px">5/30/2020: Congratulations to Skylar Stolte for her paper, 'A Survey on Medical Image Analysis in Diabetic Retinopathy', being accepted for publication in Medical Image Analysis! [<a href="https://www.sciencedirect.com/science/article/pii/S1361841520301067">Link</a>]</li>
                                                    <li style="margin-bottom:10px">5/21/2020: Congratulations to Kyle See for passing the PhD Departmental Comprehensive Exam!</li>
                                                    <li style="margin-bottom:10px">5/4/2020: Garrett Fullerton and Simon Kato have been selected for NSF REU.</li>
                                                    <li style="margin-bottom:10px">4/5/2020: Yao Xiao presented at the IEEE International Symposium on Biomedical Imaging (ISBI'20) on 'Transfer-GAN: Multimodal CT Image Super-Resolution via Transfer Generative Adversarial Networks'! [<a href="https://ieeexplore.ieee.org/document/9098322" target="_blank">Paper</a>]</li>
                                                    <li style="margin-bottom:10px">4/30/2020: Yao Xiao has been selected as a Graduate Student Commencement Speaker at the College of Engineering Graduation Ceremony. [<a href="https://www.bme.ufl.edu/hwcoe-commencement-graduation-student-speaker-yao-xiao/" target="_blank">News</a>] [<a href="http://virtualgrad.marchingorder.com/ufl/">Speech beings at 16:28</a>]</li>
                                                    <li style="margin-bottom:10px">4/7/2020: SMILer Charlie Tran has been selected to participate in the University of Florida's Ronald E. McNair Post-Baccalaureate Achievement Program. [<a href="http://mcnair.aa.ufl.edu/about/" target="_blank">News</a>]</li>
                                                    <li style="margin-bottom:10px">4/1/2020: SMILE Lab Alumnus Daniel El Basha has been awarded the NSF Graduate Research Fellowship! [<a href="https://www.research.gov/grfp/AwardeeList.do?method=loadAwardeeList" target="_blank">News</a>]</li>
                                                    <li style="margin-bottom:10px">3/25/2020: Yao Xiao on successfully defending her PhD dissertation!</li>
                                                    <li style="margin-bottom:10px">2/25/2020: SMILer Garret Fullerton has been accepted into UF University Scholars Program to work on machine learning for medical image optimization. [<a href="https://www.bme.ufl.edu/nine-bme-students-selected-as-university-scholars/" target="_blank">News</a>]</li>
                                                    <li style="margin-bottom:10px">2/17/2020: Yao Xiao presented at the SPIE Medical Imaging (SPIE MI'20) on 'Transfer generative adversarial network for multimodal CT image super-resolution'! [<a href="https://www.spiedigitallibrary.org/conference-proceedings-of-spie/11313/2549533/Transfer-generative-adversarial-network-for-multimodal-CT-image-super-resolution/10.1117/12.2549533.full?SSO=1" target="_blank">Talk</a>]</li>
                                                    <li style="margin-bottom:10px">2/7/2020: Congrats to Yao Xiao for her abstract, 'Multi-Series CT Image Super-Resolution by using Transfer Generative Adversarial Network', being accepted for presenting at the 2020 Annual Meeting of the Society for Imaging Informatics in Medicine (SIIM)! [<a href="https://cdn.ymaws.com/siim.org/resource/resmgr/siim20/abstracts-research/xiao-fang_multi-series_ct_im.pdf" target="_blank">Paper</a>]</li>
                                                    <li style="margin-bottom:10px">1/6/2020: Congrats to Yao Xiao for her paper, 'Transfer-GAN: Multimodal CT Image Super-Resolution via Transfer Generative Adversarial Networks', being accepted for publication in the IEEE International Symposium on Biomedical Imaging (ISBI'20), and awarded the ISBI (NIH, NIBIB, NCI-funded) Student Travel Award, UF GSC Student Travel Award! [<a href="https://ieeexplore.ieee.org/document/9098322" target="_blank">Paper</a>]</li>
                                                    <li style="margin-bottom:10px">12/4/2019: Dr. Fang presented at the Annual Conference of Radiological Society of North America (RSNA) on “Multimodal CT Image Super-Resolution via Transfer Generative Adversarial Network” with Ph.D. student Yao Xiao as the leading author. [<a href="https://rsna2019.rsna.org/" target="_blank">Link</a>]</li>
                                                    <li style="margin-bottom:10px">10/16/2019: Congrats to Yao Xiao for her abstract, 'Transfer generative adversarial network for multimodal CT image super-resolution', being accepted for presenting at the SPIE Medical Imaging (SPIE MI'20)! [<a href="https://www.spiedigitallibrary.org/conference-proceedings-of-spie/11313/2549533/Transfer-generative-adversarial-network-for-multimodal-CT-image-super-resolution/10.1117/12.2549533.full?SSO=1" target="_blank">Paper</a>]</li>
                                                    <li style="margin-bottom:10px">9/9/2019: Dr. Fang receives an NSF award entitled "III:Small: Modeling Multi-Level Connectivity of Brain Dynamics". [<a href="https://www.bme.ufl.edu/fang-and-ding-awarded-nih-grant-to-study-brain-dynamics/" target="_blank">News</a>]</li>
                                                    <li style="margin-bottom:10px">8/30/2019: Congrats to Peng Liu on passing his dissertation qualification!</li>
												    <li style="margin-bottom:10px">8/26/2019: Kudos to Peng Liu and Dr. Fang for their first patent at UF. A Software Using a Genetic Algorithm to Automatically Build Convolutional Neural Networks for Medical Image Denoising! [<a href="https://ufinnovate.technologypublisher.com/tech/Software_Using_a_Genetic_Algorithm_to_Automatically_Build_Convolutional_Neural_Networks_for_Medical_Image_Denoising" target="_blank">News</a>]</li>
												    <li style="margin-bottom:10px">8/20/2019: Congrats to our STTP students on their fantastic project presentations!</li>
												    <li style="margin-bottom:10px">7/26/2019: 'Development and Validation of Automated Imaging Differentiation in Parkinsonism: A Multi-Site Machine Learning Study' accepted for publication in The Lancet Digital Health. Congrats to SMiLE Lab and Dr. David Vaillancourt!</li>
												    <li style="margin-bottom:10px">8/20/2019: Two SSTP high school students Imaan Randhawa and Aarushi Walia successfully completed their summer research in SMiLE Lab under the supervision of Dr. Fang and PhD student Yao Xiao.</li>
												    <li style="margin-bottom:10px">6/28/2019: Dr. Fang recieves UF Informatics Institute Seed Fund Grant for research in smartphone based Diabetic Retinopath detection. [<a href="https://www.bme.ufl.edu/fang-receives-uf-informatics-institute-seed-fund-grant/" target="_blank">News</a>]</li>
												    <li style="margin-bottom:10px">6/13/2019: Kyle See is awarded an NIH CTSI TL1 Predoctoral Fellowship to study methods and mechanisms of cognition and motor control through data science. [<a href="https://www.bme.ufl.edu/bme-doctoral-students-awarded-nih-ctsi-tl1-predoctoral-fellowship/" target="_blank">News</a>]</li>
												    <li style="margin-bottom:10px">5/10/2019: Dr. Fang receives collaborative CTSI Pilot Award to study cancer therapy-induced cardiotoxicity. [<a href="https://www.bme.ufl.edu/fang-receives-collaborative-ctsi-pilot-award-for-precision-medicine/" target="_blank">News</a>]</li>
												    <li style="margin-bottom:10px">5/7/2019: Congrats to Yao Xiao for successfully defending her proposal</li>
												    <li style="margin-bottom:10px">4/16/2019: Congrats to Peng Liu for his paper ,'Deep Evolutionary Networks with Expedited Genetic Algorithms for Medical Image Denoising', being accepted for publication in Medical Image Analysis! [<a href="https://www.bme.ufl.edu/fang-and-researchers-manuscript-recently-published-in-medical-image-analysis/" target="_blank">News</a>]</li>
												    <li style="margin-bottom:10px">4/16/2019: Kudo to Yao Xiao for her paper, 'STIR-Net: Spatial-Temporal Image Restoration Net for CTPerfusion Radiation Reduction', being accepted for publication in Frontiers in Neurology, section Stroke. [<a href="https://www.bme.ufl.edu/fang-and-researchers-manuscript-recently-published-in-frontiers-in-neurology/" target="_blank">News</a>]</li>
												    <li style="margin-bottom:10px">4/16/2019: SMiLE Lab's Senior Design group presents their final project for Smartphone Based Diabetic Retinopathy Diagnosis.</li>
												    <li style="margin-bottom:10px">3/7/2019: Maximillian Diaz accepted to UF University Scholars Program to work on retina based Parkinson's diagnosis. [<a href="https://www.bme.ufl.edu/nine-bme-students-selected-as-university-scholars-for-2019-2020/" target="_blank">News</a>]</li>
												    <li style="margin-bottom:10px">1/10/2019: Dr. Fang named a senior member of IEEE. [<a href="https://www.bme.ufl.edu/fang-elevated-to-ieee-senior-member/" target="_blank">News</a>]</li>
												    <li style="margin-bottom:10px">5/17/2018: Dr. Fang awarded University of Florida Informatics Institute and the Clinical and Translational Science Institute (CTSI) pilot funding for precision medicine. [<a href="https://www.bme.ufl.edu/fang-awarded-ctsi-pilot-program-funding/" target="_blank">News</a>]</li>
												    <li style="margin-bottom:10px">4/24/2018: SMiLE Lab recieves two first place awards at the Diabetic Retinopathy Segmentation and Grading Challenge. [<a href="https://www.bme.ufl.edu/smile-lab-wins-first-place-in-international-competition-on-retinal-image-analysis/" target="_blank">News</a>]</li>
												    <li style="margin-bottom:10px">3/26/2018: Akash and Akshay Mathavan accepted to UF University Scholars Program to work on environmental risk factors for ALS. [<a href="https://www.bme.ufl.edu/thirteen-uf-bme-students-selected-as-university-scholars-for-2018-2019/" target="_blank">News</a>]</li>
                                                    <li style="margin-bottom:10px">10/30/2017: Two master students, Yangjunyi Li and Yun Liang, join SMILE Group. Welcome!</li>
                                                    <li style="margin-bottom:10px">10/1/2017: Two UF BME senior students Kyle See and Daniel El Basha join SMILE lab to work on big biomedical data anlytics research supported by NSF REU. Welcome! [<a href="https://www.bme.ufl.edu/fangs-students-awarded-reu-supplement-from-nsf/" target="_blank">News</a>]</li>
                                                    <!--<li>09/01/2017: One paper is accepted by the journal IET Image Processing on <a href="http://digital-library.theiet.org/content/journals/10.1049/iet-ipr.2017.0273">``Automatic Choroid Layer Segmentation: A Slicing Approach''</a>. Congrats to Saleha!</li>-->
                                                    <li style="margin-bottom:10px">08/2017: Yao is awarded <a href="http://www.miccai2017.org/">MICCAI 2017</a> Student Travel Award. Congrats to Yao! [<a href="https://www.bme.ufl.edu/bme-student-receives-two-prestigious-travel-awards/" target="_blank">News</a>]</li>
                                                    <li style="margin-bottom:10px">08/2017: Four papers from SMILE Lab will be presented at <a href="http://www.bmes.org/">BMES 2017</a> in Pheonix, Arizona.</li>
                                                    <li style="margin-bottom:10px">08/2017: SMILE Lab will move to J. Crayton Pruitt Family Department of Biomedical Engineering at University of Florida.</li>
                                                    <li style="margin-bottom:10px">07/2017: One paper is accepted by <a href="http://mlmi2017.web.unc.edu/" target="_blank">Machine Learning and Medical Imaging (MLMI) Workshop</a> at <a href="http://www.miccai2017.org/" target="_blank">MICCAI 2017</a>. Congrats to Yao! </li>
                                                    <li style="margin-bottom:10px">07/16/2017: Dr. Fang is invited to give a talk at the <a href="http://mic.sjtu.edu.cn/mics-2017/home" target="_blank">4th Medical Image Computing Seminar (MICS)</a> held at Shanghai Jiaotong University, Shanghai, China.</li>
                                                    <li style="margin-bottom:10px">06/2017: Yao won the NSF Travel Award to attend <a href="http://conferences.computer.org/chase2017/" target="_blank">CHASE 2017</a> in Philadelphia, PA. Congrats to Yao! [<a href="https://www.bme.ufl.edu/bme-student-receives-two-prestigious-travel-awards/" target="_blank">News</a>]</li>
                                                    <li style="margin-bottom:10px">05/2017: Dr. Fang is invited by NSF as a panelist of Smart and Connect Health.</li>
                                                    <li style="margin-bottom:10px">05/2017: One paper is accepted by <a href="http://conferences.computer.org/chase2017/" target="_blank">CHASE 2017</a>. Congrats to Yao! </li>
                                                    <li style="margin-bottom:10px">04/2017: One paper is accepted by <a href="http://www.miccai2017.org/" target="_blank">MICCAI 2017</a>. Congrats to Ling! </li>
                                                    <li style="margin-bottom:10px">04/2017: Dr. Fang is invited to give a talk at the <a href="http://www.worldbrainmapping.org/" target="_blank">Society for Brain Mapping and Therapeutics</a> Annual Meeting in Los Angeles, CA. </li>
                                                    <li style="margin-bottom:10px">03/2017: Dr. Fang attended NSF Smart and Connected Health PI Meeting held in Boston, MA. </li>
                                                    <li style="margin-bottom:10px">10/2016: PhD student Maryamossadat Aghili has been awarded travel grants from NIPS WiML 2016 Program Committee and FIU GPSC to attend WiML of NIPS 2016 in Barcelona. </li>
                                                    <li style="margin-bottom:10px">09/2016: Dr. Fang is invited by NIH to review in BDMA study section.</li>
                                                    <li style="margin-bottom:10px">09/2016: One paper is accepted by Pattern Recognition!</li>
                                                    <li style="margin-bottom:10px">08/2016: Dr. Fang is selected as an Early Career Grant Reviewer by NIH.</li>
                                                    <li style="margin-bottom:10px">08/2016: SMILE REU and RET students and teachers won the 2016 Best REU and RET Poster Awards at SCIS of FIU.</li>
                                                    <li style="margin-bottom:10px">07/2016: Our paper "Abdominal Adipose Tissues Extraction Using Multi-Scale Deep Neural Network" is accepted by NeuroComputing!</li>
                                                    <li style="margin-bottom:10px">04/2016: PhD student Maryamossadat Aghili has been awarded travel grants to attend Grad Cohort Conference (CRAW).</li>
                                                    <li style="margin-bottom:10px">05/2016: Dr. Fang's research appeared on FIU News: <a href="http://news.fiu.edu/2016/05/professor-uses-computer-science-to-reduce-patients-exposure-to-radiation-from-ct-scans/101015" target="_blank">Professor uses computer science to reduce patients' exposure to radiation from CT scans</a>!</li>
                                                    <li style="margin-bottom:10px">05/2016: <b>Dr. Fang has been selected to receive the <a href="http://www.orau.org/university-partnerships/faculty-student-programs/powe/powe-winners.aspx" target="_blank">2016 Ralph E. Powe Junior Faculty Enhancement Award</a> from Oak Ridge Associated Universities!</b></li>
                                                    <li style="margin-bottom:10px">05/2016: <b>Dr. Fang received funding from NSF on CRII (Pre-CAREER) Award on "<a href="http://www.nsf.gov/awardsearch/showAward?AWD_ID=1564892" target="_blank">Characterizing, Modeling and Evaluating Brain Dynamics</a>"!</b></li>
                                                    <li style="margin-bottom:10px">04/2016: Our paper "TENDER: TEnsor Non-local Deconvolution Enabled Radiation Reduction in CT Perfusion" is accepted by NeuroComputing!</li>
                                                    <li style="margin-bottom:10px">04/09/2016: <b>Dr. Fang is awarded the first <a href="http://research.fiu.edu/research-faculty-honors-highlights/" target="_blank">Robin Sidhi Memorial Young Scientist Award</a> from the Annual Congress of <a href="http://www.worldbrainmapping.org/" target="_blank">Society of Brain Mapping and Therapeutics</a> for recognizing her continuous devotion to bridge information technology and health informatics. </b></li>
                                                    <li style="margin-bottom:10px">03/2016: Our paper "<a href=http://users.cis.fiu.edu/~rfang/publications/ACM16_Survey.pdf" target="_blank">"Computational Health Informatics in the Big Data Age: A Survey</a>" is accepted by the prestigous ACM Computing Survey!</li>
                                                    <li style="margin-bottom:10px">01/2016: Dr. Fang is invited by NSF as a panel reviewer of Smart and Connect Health.</li>
                                                    <li style="margin-bottom:10px">12/2015: Dr. Fang serves as the Publicity Chair of the <a href="http://www.icmla-conference.org/icmla15/#">14th IEEE International Conference on Machine Learning and Applications 2015</a>.</li>
                                                    <li style="margin-bottom:10px">04/2015: Dr. Fang is invited by NSF as a panel reviewer of Smart and Connect Health.</li>
                                                </ul>
                                            </div>
                                        </td>
                                    </tr>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>
                    <!--/News-->


                    <!--Honors, Awards, Grants-->
                    <div class="section color-1">
                        <div class="section-container">
                            <div class="row">
                                <div class="col-md-10 col-md-offset-1">
                                    <div class="title text-center">
                                        <h3>Honors, Awards and Grants</h3>
                                    </div>
                                    <ul class="timeline">
                                        <li class="open">
                                            <div class="date">2020</div>
                                            <div class="circle"></div>
                                            <div class="data">
                                                <div class="subject"><a href="https://news.ufl.edu/2020/12/artificial-intelligence-research-catalyst-fund-/">NVIDIA-UF Artificial Intelligence Catalyst Award</a></div>
                                            </div>
                                        </li>
                                        
                                        <li class="open">
                                            <div class="date">2019</div>
                                            <div class="circle"></div>
                                            <div class="data">
                                                <div class="subject"><a href="https://www.nsf.gov/awardsearch/showAward?AWD_ID=1908299&HistoricalAwards=false">NSF IIS Information Integration and Informatics Award</a></div>
                                            </div>
                                        </li>
                                        
                                       <li class="open">
                                            <div class="date">2017</div>
                                            <div class="circle"></div>
                                            <div class="data">
                                                <div class="subject">ACM Future Academy of Computing</div>
                                            </div>
                                        </li>

                                        <li class="open">
                                            <div class="date">2016</div>
                                            <div class="circle"></div>
                                            <div class="data">
                                                <div class="subject">NSF Best REU and RET Poster Awards</div>
                                                <div class="text row">
                                                    <div class="col-md-2">
                                                        <img alt="image" class="thumbnail img-responsive" src="img/personal/reu.jpg" >
                                                    </div>
                                                    <div class="col-md-10">
                                                        Dr. Fang's mentee, Paul Naghshineh from George Washington University, won the Best REU Poster Award at the annual REU Symposium at FIU. Christian McDonald with Edda I. Rivera, teachers from Miami Jackson Senior High School and John A. Ferguson Senior High School, under Dr. Fang's mentorship, won the Best RET Poster Award. Congratulations to Paul, Christian and Edda!
                                                    </div>
                                                </div>
                                            </div>
                                        </li>
                                        <li class="open">
                                            <div class="date">2016</div>
                                            <div class="circle"></div>
                                            <div class="data">
                                                <div class="subject">Ralph E. Powe Junior Faculty Enhancement Award</div>
                                                <div class="text row">
                                                    <div class="col-md-2">
                                                        <img alt="image" class="thumbnail img-responsive" src="img/personal/ORAU.jpg" >
                                                    </div>
                                                    <div class="col-md-10">
                                                        Dr. Fang has been selected to receive the 2016 Ralph E. Powe Junior Faculty Enhancement Award from Oak Ridge Associated Universities (two nominations from each institute, 35 awardees out of 132 applicants).
                                                    </div>
                                                </div>
                                            </div>
                                        </li>
                                        <li>
                                            <div class="date">2016</div>
                                            <div class="circle"></div>
                                            <div class="data">
                                                <div class="subject"><a href="https://www.nsf.gov/awardsearch/showAward?AWD_ID=1564892">NSF IIS CRII Award on "Characterizing, Modeling and Evaluating Brain Dynamics"</a></div>
                                                <div class="text row">
                                                    <div class="col-md-2">
                                                        <img alt="image" class="thumbnail img-responsive" src="img/personal/nsf.jpg" >
                                                    </div>
                                                    <div class="col-md-10">
                                                    <p>Brain dynamics, which reflects the healthy or pathological states of the brain with quantifiable, reproducible, and indicative dynamics values, remains the least understood and studied area of brain science despite its intrinsic and critical importance to the brain. Unlike other brain information such as the structural and sequential dimensions that have all been extensively studied with models and methods successfully developed, the 5th dimension, dynamics, has only very recently started receiving systematic analysis from the research community. The state-of-the-art models suffer from several fundamental limitations that have critically inhibited the accuracy and reliability of the dynamic parameters' computation. First, dynamic parameters are derived from each voxel of the brain spatially independently, and thus miss the fundamental spatial information since the brain is connected? Second, current models rely solely on single-patient data to estimate the dynamic parameters without exploiting the big medical data consisting of billions of patients with similar diseases.</p>
                                                    <p>This project aims to develop a framework for data-driven brain dynamics characterization, modeling and evaluation that includes the new concept of a 5th dimension – brain dynamics – to complement the structural 4-D brain for a complete picture. The project studies how dynamic computing of the brain as a distinct problem from the image reconstruction and de-noising of convention models, and analyzes the impact of different models for the dynamics analysis. A data-driven, scalable framework will be developed to depict the functionality and dynamics of the brain. This framework enables full utilization of 4-D brain spatio-temporal data and big medical data, resulting in accurate estimations of the dynamics of the brain that are not reflected in the voxel-independent models and the single patient models. The model and framework will be evaluated on both simulated and real dual-dose computed tomography perfusion image data and then compared with the state-of-the-art methods for brain dynamics computation by leveraging collaborations with Florida International University Herbert Wertheim College of Medicine, NewYork-Presbyterian Hospital / Weill Cornell Medical College (WCMC) and Northwell School of Medicine at Hofstra University. The proposed research will significantly advance the state-of-the-art in quantifying and analyzing brain structure and dynamics, and the interplay between the two for brain disease diagnosis, including both the acute and chronic diseases. This unified approach brings together fields of Computer Science, Bioengineering, Cognitive Neuroscience and Neuroradiology to create a framework for precisely measuring and analyzing the 5th dimension – brain dynamics – integrated with the 4-D brain with three dimensions from spatial data and one dimension from temporal data. Results from the project will be incorporated into graduate-level multi-disciplinary courses in machine learning, computational neuroscience and medical image analysis. This project will open up several new research directions in the domain of brain analysis, and will educate and nurture young researchers, advance the involvement of underrepresented minorities in computer science research, and equip them with new insights, models and tools for developing future research in brain dynamics in a minority serving university.</p>
                                                    </div>
                                                </div>
                                            </div>
                                        </li>
                                        <li>
                                            <div class="date">2016</div>
                                            <div class="circle"></div>
                                            <div class="data">
                                                <div class="subject">Robin Sidhu Memorial Young Scientist Award</div>
                                                <div class="text row">
                                                    <div class="col-md-2">
                                                        <img alt="image" class="thumbnail img-responsive" src="img/personal/robin.jpg" >
                                                    </div>
                                                    <div class="col-md-10">
                                                    Dr. Fang was awarded the Robin Sidhu Memorial Young Scientist Award at the Annual Congress of Society of Brain Mapping and Theurapeutics. News reported on PR Newswire, Yahoo,<a href="http://news.fiu.edu/2016/05/professor-uses-computer-science-to-reduce-patients-exposure-to-radiation-from-ct-scans/101015" target="_blank" >FIU NEWS<i class="fa fa-external-link"></i></a>
                                                    </div>
                                                </div>
                                            </div>
                                        </li>
                                        <li>
                                            <div class="date">2015</div>
                                            <div class="circle"></div>
                                            <div class="data">
                                                <div class="subject">National Science Foundation CISE CAREER Workshop Travel Award </div>
                                                <div class="text row">
                                                    <div class="col-md-2">
                                                        <img alt="image" class="thumbnail img-responsive" src="img/personal/nsf.jpg" >
                                                    </div>
                                                    <div class="col-md-10">
                                                        Dr. Fang received a Travel Award to attend CISE NSF CAREER Proposal Writing Workshop in Arlington, VA.
                                                    </div>
                                                </div>
                                            </div>
                                        </li>
                                        <li>
                                            <div class="date">2014</div>
                                            <div class="circle"></div>
                                            <div class="data">
                                                <div class="subject">Hsien Wu and Daisy	Yen	Wu Memorial	Award</div>
                                                <div class="text row">
                                                    <div class="col-md-2">
                                                        <img alt="image" class="thumbnail img-responsive" src="img/personal/cu.jpg" >
                                                    </div>
                                                    <div class="col-md-10">
                                                    In recognition	of the	excellent progress in the academic program and high potential for a successful academic career (5 awardees out	of all graduate students at Cornell University).
                                                    </div>
                                                </div>
                                            </div>
                                        </li>
                                        
                                        <!---
                                        <li>
                                        <div class="date">2014</div>
                                        <div class="circle"></div>
                                            <div class="data">
                                                <div class="subject">Student Travel	Award at MICCAI</div>
                                                <div class="text row">
                                                    <div class="col-md-2">
                                                        <img alt="image" class="thumbnail img-responsive" src="img/personal/miccai.jpg" >
                                                    </div>
                                                    <div class="col-md-10">
                                                        Student	Travel Award at the	International Conference on	Medical	Image Computing	and Computer Assisted Intervention (MICCAI)	2014.
                                                    </div>
                                                </div>
                                            </div>
                                        </li>
                                        --->
                                        
                                        <li>
                                            <div class="date">2012</div>
                                            <div class="circle"></div>
                                            <div class="data">
                                                <div class="subject">IBM Cornell ECE Women's Conference	Travel Grant</div>
                                                <div class="text row">
                                                    <div class="col-md-2">
                                                        <img alt="image" class="thumbnail img-responsive" src="img/personal/ibm.jpg" >
                                                    </div>
                                                    <div class="col-md-10">
                                                        IBM	Cornell	ECE	Women's	Conference Travel Grant to attend the 15th International Conference	on Medical Image Computing and Computer	Assisted Intervention, 2012.
                                                    </div>
                                                </div>
                                            </div>
                                        </li>

                                        <li>
                                        <div class="date">2010</div>
                                        <div class="circle"></div>
                                            <div class="data">
                                                <div class="subject">Best Paper Award, IEEE International Conference on Image Processing (ICIP)</div>
                                                <div class="text row">
                                                    <div class="col-md-2">
                                                        <img alt="image" class="thumbnail img-responsive" src="img/personal/icip.jpg" >
                                                    </div>
                                                    <div class="col-md-10">
                                                        Best Paper Award at	the	17th International Conference on Image Processing, 2010. (Top 1 out	of 1190 accepted papers, first author publication)
                                                    </div>
                                                </div>
                                            </div>
                                        </li>

                                        <li>
                                            <div class="date">2010</div>
                                            <div class="circle"></div>
                                            <div class="data">
                                                <div class="subject">Best PhD Poster Award, Cornell Engineering </div>
                                                <div class="text row">
                                                    <div class="col-md-2">
                                                        <img alt="image" class="thumbnail img-responsive" src="img/personal/cu.jpg" >
                                                    </div>
                                                <div class="col-md-10">
                                                    Cornell	Engineering	Research Conference, 2010
                                                </div>
                                            </div>
                                            </div>
                                        </li>
                                        
                                        <li>
                                            <div class="date">2009</div>
                                            <div class="circle"></div>
                                            <div class="data">
                                                <div class="subject">Irwin and Joan	Jacobs Fellowship</div>
                                                <div class="text row">
                                                    <div class="col-md-2">
                                                        <img alt="image" class="thumbnail img-responsive" src="img/personal/cu.jpg" >
                                                    </div>
                                                <div class="col-md-10">
                                                    Cornell	University,	awarded	to students	who	exemplify strength and potential in academics, service,	and	leadership,	2009-2010
                                                </div>
                                                </div>
                                            </div>
                                        </li>


                                    </ul>
                                </div>
                            </div>
                        </div>
                    </div>
                    <!--/Honors, Awards, Grants-->

                    <!--Academic Positions-->
                        <div class="section color-1">
                            <div class="section-container">
                                <div class="row">

                                    <!--First Column-->
                                    <div class="col-md-5">

                                        <div class="title text-center">
                                            <h3>Education</h3>
                                        </div>

                                        <ul class="ul-card">
                                            <li>
                                                <div class="dy">
                                                    <span class="degree">Ph.D.</span>
<!--                                                    <span class="year">2014</span>-->
                                                </div>
                                                <div class="description">
                                                    <p class="waht">Ph.D. in Electrical	and Computer Engineering</p>
                                                    <p class="where">Cornell University, Ithaca, NY.</p>
                                                </div>
                                            </li>

                                            <!--<li>
                                                <div class="dy">
                                                    <span class="degree">B.A.</span><span class="year">2007</span>
                                                </div>
                                                <div class="description">
                                                    <p class="waht">Full-time Exchange in Electrical and	Communication Engineering</p>
                                                    <p class="where">University	of Hong	Kong,Hong Kong</p>
                                                </div>
                                            </li>-->

                                            <li>
                                                <div class="dy">
                                                    <span class="degree">B.E.</span>
<!--                                                    <span class="year">2009</span>-->
                                                </div>
                                                <div class="description">
                                                    <p class="waht">B.E. in Information Engineering</p>
                                                    <p class="where">Zhejiang University, Hangzhou, China</p>
                                                </div>
                                            </li>
                                        </ul>
                                    </div>
                                    
                                    

                                    <!--Second Column-->
                                   <div class="col-md-5 col-md-offset-1">

                                        <div class="title text-center">
                                            <h3>Academic Positions</h3>
                                        </div>

                                        <!--1st Position-->
                                        <ul class="ul-dates">
                                            <li>
                                                <div class="dates">
                                                    <span>BME</span>
                                                </div>
                                                <div class="content">
                                                    <h4>Tenure-Track Assistant Professor</h4>
                                                    <p><em>University of Florida</em> J. Crayton Pruitt Family Department of Biomedical Engineering</p>
                                                </div>
                                            </li>
                                        </ul>
                                        <!--/1st Position-->

                                        <!--2nd Position-->
                                        <ul class="ul-dates">
                                            <li>
                                                <div class="dates">
<!--                                                    <span>Present</span>-->
<!--                                                    <span>2017</span>-->
                                                    <span>ECE</span>
                                                </div>
                                                <div class="content">
                                                    <h4>Affliated Faculty</h4>
                                                    <p><em>University of Florida</em> Department of Electrical and Computer Engineering</p>
                                                </div>
                                            </li>
                                        </ul>
                                        <!--/2nd Position-->

                                        <!--3rd Position-->
                                         <ul class="ul-dates">
                                            <li>
                                                <div class="dates">
<!--                                                    <span>Present</span>-->
<!--                                                    <span>2018</span>-->
                                                    <span>COM</span>
                                                </div>
                                                <div class="content">
                                                    <h4>Affiliated Faculty</h4>
                                                    <p><em>University of Florida</em> Department of Radiology, College of Medicine</p>
                                                </div>
                                            </li>
                                        </ul>
                                        <!--/3rd Position-->

                                        <!--/4th Position-->
                                        <ul class="ul-dates">
                                            <li>
                                                <div class="dates">
<!--                                                    <span>Present</span>-->
<!--                                                    <span>2019</span>-->
                                                    <span>CISE</span>
                                                </div>
                                                <div class="content">
                                                    <h4>Affiliated Faculty</h4>
                                                    <p><em>University of Florida</em> Department of Computer & Information Science & Engineering</p>
                                                </div>
                                            </li>
                                        </ul>
                                        <!--/4th Position-->

                                        <!--5th Position-->
                                         <ul class="ul-dates">
                                            <li>
                                                <div class="dates">
<!--                                                    <span>Present</span>-->
<!--                                                    <span>2020</span>-->
                                                    <span>CAM</span>
                                                </div>
                                                <div class="content">
                                                    <h4>Affiliated Faculty</h4>
                                                    <p><em>University of Florida</em> The Center for Cognitive Aging and Memory (CAM)</p>
                                                </div>
                                            </li>
                                        </ul>
                                        <!--/5th Position-->

                                        <!--
                                        <ul class="ul-dates">
                                            <li>
                                                <div class="dates">
                                                    <span>2017</span>
                                                    <span>2014</span>
                                                </div>
                                                <div class="content">
                                                    <h4>Tenure-Track Assistant Professor</h4>
                                                    <p><em>Florida International University</em> School of Computing and Information Sciences</p>
                                                </div>
                                            </li>
                                        </ul>
                                        -->
                                    </div>
                                    <!--/Second Column-->

                                </div>
                            </div>
                        </div>
                        <!--/Academic Positions-->
            
                    <!--Acknowledgements-->
                    <div class="section color-2">
                        <div class="section-container">
                            <div class="row">
                                <div class="col-md-10 col-md-offset-1">
                                    <div class="title text-center">
                                        <h3>Acknowledgements</h3>
                                    </div>
                                    <ul>
                                        <li class="open">
                                            <div class="data">
                                                <div class="">Our work is supported by:</div><br>
                                                <img alt="image" src="img/acknowlegements/nsf_logo.gif" >
                                                <img alt="image" height="166" src="img/acknowlegements/nih.jpg" >
                                                <img alt="image" height="166" src="img/acknowlegements/DHS.png" >
                                                <img alt="image" height="166" src="img/acknowlegements/DoD.png" >
                                                <img alt="image" height="166" src="img/acknowlegements/orau.jpg" >
                                                <img alt="image" height="166" src="img/acknowlegements/nvidia.jpg" >
                                                <img alt="image" height="166" src="img/acknowlegements/sbmt.png" >
                                                <img alt="image" height="166" src="img/acknowlegements/uf.png" >
                                                <img alt="image" height="166" src="img/acknowlegements/UFII.png" >
                                            </div>
                                        </li>
                                    </ul>
                                </div>
                            </div>
                        </div>
                    </div>
                </div>
                <!--/About me-->

                <!--Team-->
                <div id="team" class="page">
                    <div class="pageheader">
                        <div class="headercontent">
                            <div class="section-container">
                                <h2 class="title">Team</h2>
                                <div class="row">
                                    <div class="col-md-24">
                                        <p></p>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>
                    <div class="pagecontents">
                        <div class="section color-1">
                            <div class="section-container">
                                <!--
                                <div class="title text-center">
                                    <h3>Openings</h3>
                                </div>
                                <div class="section color-2" id="pub-grid">
                                    <div class="pitems">
                                        <div class="item mix Journal" data-year="2017">
                                            <div class="pubmain">
                                                <div class="pubassets">
                                                    <a href="#" class="pubcollapse">
                                                        <i class="fa fa-expand"></i>
                                                    </a>
                                                </div>
                                                <h4 class="pubtitle">NSF Research Experiences for Undergraduates (REU)</h4>
                                                <div>Big Biomedical Informatics Analytics</div>
                                            </div>
                                            <div class="pubdetails">
                                                <h4>Details</h4>
                                                <ul>
                                                    <li>The Research Experiences for Undergraduates (REU) site at University of Florida (UF) provides research experience to undergraduate students a six-month program. </li>
                                                    <li>Students will be paid <strong>$5,000</strong> for doing some cutting-edge research.</li>
                                                    <li>Students will work on cutting-edge research in the area of machine learning and data mining in big medical image and informatics analytics under the supervision of Dr. Fang and her graduate students.</li>
                                                    <li>Improve oral and written comprehension and communication of technical knowledge.</li>
                                                    <li>Spark an interest in continuing STEM research in graduate schools.</li>
                                                </ul>
                                                <h4>Requirements</h4>
                                                <ul>
                                                    <li>Applicant must be a U.S. citizen or permanent resident of the United States.</li>
                                                    <li>Applicant must be and remain an undergraduate student in good standing.</li>
                                                </ul>
                                                <p>All students that meet these requirements are welcome to apply! We are particularly interested in broadening participation of underrepresented groups in STEM, and women and minorities are especially encouraged to apply. We also aim to provide research experiences for students who have limited exposure and opportunities to participate in research. </p>
                                                <h4>How to apply</h4>
                                                <ul>
                                                    <li>Send an email to Dr. Fang entitled "REU Application_YourName" with your (i) a CV; (ii) information describing your research skills and experience; (iii) your goals and research interests for your REU research; (iv) contact details for two references. Full consideration will be given to applicants submitted before Oct. 1, 2017.</li>
                                                    <li>We consider applications on a rolling basis until all positions are filled.</li>
                                                </ul>
                                            </div>
                                        </div>
                                    </div>
                                </div>
                                -->

                                <!--Current Members-->
                                <div class="title text-center">
                                    <h3>Current Members</h3>
                                </div>
<!--Peng-->                     <a href="https://sites.google.com/view/liupeng" target="_blank">
                                <div class="member">
                                    <img src="img/lab/pengliu.jpg" width="200" height="200" class="img-circle lab-img">
                                    <div class="bio">
                                        <h3><b>Peng Liu</b></h3>
                                        <h4><i>PhD Candidate</i></h4>
                                        <h4>Biomedical Engineering</h4>
                                        <div class="bmail">pliu1@ufl.edu</div>
                                        <!-- <div class="btags2 btw">Biology-Inspired Artificial Intelligence</div>
                                        <div class="btags2 btw">Medical Image Analysis</div>
                                        <div class="btags2 btw">AI-Inspired Neuroscience</div> -->
                                    </div>
                                </div>
                                </a>

<!-- JQ
                                <div class="member">
                                    <img src="img/lab/jq.jpg" width="200" height="200" class="img-circle lab-img">
                                    <div class="bio">
                                        <h3><b>Jianqiao Tian</b></h3>
                                        <h4><i>PhD Student</i></h4>
                                        <h4>Biomedical Engineering</h4>
                                        <div class="bmail"> j.tian@ufl.edu</div>
                                    </div>
                                </div>
-->


<!-- Kyle-->                    <div class="member">
                                    <img src="img/lab/kylesee.jpg" width="200" height="200" class="img-circle lab-img">
                                    <div class="bio">
                                        <h3><b>Kyle See</b></h3>
                                        <h4><i>PhD Student, GSPA</i></h4>
                                        <h4>Biomedical Engineering</h4>
                                        <h5>NIH CTSI TL1 Predoctoral Fellowship</h5>
                                        <div class="bmail"> kylebsee@ufl.edu</div>
                                        <!-- <div class="btags2 btw">Machine Learning in Somatotopy</div>
                                        <div class="btags2 btw">Autoimmune Disease Analytics</div> -->
                                    </div>
                                </div>

<!-- Skylar-->	                <div class="member">
                                    <img src="img/lab/skylarstolte.png" width="200" height="200" class="img-circle lab-img">
                                    <div class="bio">
                                        <h3><b>Skylar Stolte</b></h3>
                                        <h4><i>PhD Student, GSPA</i></h4>
                                        <h4>Biomedical Engineering</h4>
                                        <div class="bmail">skylastolte4444@ufl.edu</div>
                                        <!-- <div class="btags2 btw">Cardiotoxicity Prediction</div>
                                        <div class="btags2 btw">Longitudinal Glaucoma Study</div> -->
                                    </div>
                                </div>

<!-- Rachel Ho-->	            <div class="member">
                                    <img src="img/lab/rachelho.jpg" width="200" height="200" class="img-circle lab-img">
                                    <div class="bio">
                                        <h3><b>Rachel Ho</b></h3>
                                        <h4><i>PhD Student, TL1 Scholar</i></h4>
                                        <h4>Applied Physiology and Kinesiology</h4>
                                        <div class="bmail">rachel.judy@ufl.edu</div>
                                        <h5>Co-advised with Dr. Steven Coombes via TL1 Program</h5>
                                        <!-- <div class="btags2 btw">Longitudinal Glaucoma Study</div> -->
                                    </div>
                                </div>
                                
<!--Shreya-->                   <div class="member">
                                    <img src="img/lab/shreyaverma.jpg" width="200" height="200" class="img-circle lab-img">
                                    <div class="bio">
                                        <h3><b>Shreya Verma</b></h3>
                                        <h4><i>Master Student</i></h4>
                                        <h4>Biomedical Engineering</h4>
                                        <div class="bmail">shreyaverma@ufl.edu</div>
                                    </div>
                                </div>
                                
<!--Bhavin-->                   <div class="member">
                                    <img src="img/lab/BhavinSoni.jpg" width="200" height="200" class="img-circle lab-img">
                                    <div class="bio">
                                        <h3><b>Bhavin Soni</b></h3>
                                        <h4><i>Master Student</i></h4>
                                        <h4>Biomedical Engineering</h4>
                                        <div class="bmail">bhavinsoni@ufl.edu</div>
                                    </div>
                                </div>


<!--Max                         <div class="member">
                                    <img src="img/lab/maxdiaz.png" width="200" height="200" class="img-circle lab-img">
                                    <div class="bio">
                                        <h3><b>Maximillian Diaz</b></h3>
                                        <h4>Undergraduate Student</h4>
                                        <h5>University Scholar Program</h5>
                                        <div class="bmail"> mdiaz40@ufl.edu</div>
                                        <div class="btags2 btw">Early Neural Disease Diagnosis</div>
                                        <div class="btags2 btw">Medical Image Analysis</div>
                                    </div>
                                </div>
-->

<!--Sumanth                     <div class="member">
                                    <img src="img/lab/sumanthaluri.jpg" width="200" height="200" class="img-circle lab-img">
                                    <div class="bio">
                                        <h3><b>Sumanth Aluri</b></h3>
                                        <h4><i>Undergraduate Student</i></h4>
                                        <h4>Computer Science</h4>
                                        <div class="bmail"> aluris@ufl.edu</div>
                                    </div>
                                </div>
-->

<!--Charlie-->                  <div class="member">
                                    <img src="img/lab/charlietran.jpg" width="200" height="200" class="img-circle lab-img">
                                    <div class="bio">
                                        <h3><b>Charlie Tran</b></h3>
                                        <h4><i>Undergraduate Student</i></h4>
                                        <h4>Electrical and Computer Engineering</h4>
                                        <h5>McNair Scholars Program</h5>
                                        <div class="bmail"> charlietran@ufl.edu</div>
                                    </div>
                                </div>

<!--Rachel                      <div class="member">
                                    <img src="img/lab/rachelpeebles.jpg" width="200" height="200" class="img-circle lab-img">
                                    <div class="bio">
                                        <h3><b>Rachel Peebles</b></h3>
                                        <h4>Undergraduate Student</h4>
                                        <div class="bmail">rachel.peebles@ufl.edu</div>
                                    </div>
                                </div>
-->

<!--Garret-->                   <div class="member">
                                    <img src="img/lab/garrettfullerton.jpg" width="200" height="200" class="img-circle lab-img">
                                    <div class="bio">
                                        <h3><b>Garrett Fullerton</b></h3>
                                        <h4><i>Undergraduate Student</i></h4>
                                        <h4>Biomedical Engineering</h4>
                                        <h4>NSF REU Awardee</h4>
                                        <h5>University Scholar Program</h5>
                                        <div class="bmail"> gfullerton@ufl.edu</div>
                                    </div>
                                </div>

<!--Neeva                    <div class="member">
                                    <img src="img/lab/neevasethi.jpg" width="200" height="200" class="img-circle lab-img">
                                    <div class="bio">
                                        <h3><b>Neeva Sethi</b></h3>
                                        <h4><i>Undergraduate Student</i></h4>
                                        <h4>Computer Science</h4>
                                        <div class="bmail">neevasethi@ufl.edu</div>
                                    </div>
                                </div>
-->
                                
<!--Gianna-->                   <div class="member">
                                    <img src="img/lab/giannasweeting.jpg" width="200" height="200" class="img-circle lab-img">
                                    <div class="bio">
                                        <h3><b>Gianna Sweeting</b></h3>
                                        <h4><i>Undergraduate Student</i></h4>
                                        <h4>Biomedical Engineering</h4>
                                        <div class="bmail">sweetinggianna@ufl.edu</div>
                                    </div>
                                </div>

<!--Ahmet                       <div class="member">
                                    <img src="img/lab/ahmetbilgili.jpg" width="200" height="200" class="img-circle lab-img">
                                    <div class="bio">
                                        <h3><b>Ahmet Bilgili</b></h3>
                                        <h4><i>Undergraduate Student</i></h4>
                                        <h4>Biomedical Engineering</h4>
                                        <div class="bmail">ahmet.bilgili@ufl.edu</div>
                                    </div>
                                </div>
-->

<!--Jason-->                    <div class="member">
                                    <img src="img/lab/jasonchen.png" width="200" height="200" class="img-circle lab-img">
                                    <div class="bio">
                                        <h3><b>Jason Chen</b></h3>
                                        <h4><i>Undergraduate Student</i></h4>
                                        <h4>Computer Science</h4>
                                        <div class="bmail">yanghaoyuchen@ufl.edu</div>
                                    </div>
                                </div>

<!--Muhamed                  <div class="member">
                                    <img src="img/lab/muhamedmaher.jpg" width="200" height="200" class="img-circle lab-img">
                                    <div class="bio">
                                        <h3><b>Muhamed Maher</b></h3>
                                        <h4><i>Undergraduate Student</i></h4>
                                        <h4>Biomedical Engineering</h4>
                                        <div class="bmail">mhobi@ufl.edu</div>
                                    </div>
                                </div>
-->

<!--Edward-->                   <div class="member">
                                    <img src="img/lab/edwardzhang.jpg" width="200" height="200" class="img-circle lab-img">
                                    <div class="bio">
                                        <h3><b>Edward Zhang</b></h3>
                                        <h4><i>Undergraduate Student</i></h4>
                                        <h4>Computer Engineering & Finance</h4>
                                        <div class="bmail">edwardzhang@ufl.edu</div>
                                    </div>
                                </div>

<!--Adeeb-->                    <div class="member">
                                    <img src="img/lab/adeebrashid.jpeg" width="200" height="200" class="img-circle lab-img">
                                    <div class="bio">
                                        <h3><b>Adeeb Rashid</b></h3>
                                        <h4><i>Undergraduate Student</i></h4>
                                        <h4>Biomedical Engineering & Computer Science</h4>
                                        <div class="bmail">adeeb.rashid@ufl.edu</div>
                                    </div>
                                </div>

<!--James                    <div class="member">
                                    <img src="img/lab/jamesbondnguyen.jpg" width="200" height="200" class="img-circle lab-img">
                                    <div class="bio">
                                        <h3><b>James-Bond Nguyen</b></h3>
                                        <h4><i>Undergraduate Student</i></h4>
                                        <h4>Biomedical Engineering</h4>
                                        <div class="bmail">jbnguyen@ufl.edu</div>
                                    </div>
                                </div>
-->

<!--Simon-->                    <div class="member">
                                    <img src="img/lab/simonkato.jpg" width="200" height="200" class="img-circle lab-img">
                                    <div class="bio">
                                        <h3><b>Simon Kato</b></h3>
                                        <h4><i>Undergraduate Student</i></h4>
                                        <h4>Math and Statistics</h4>
                                        <h4>NSF REU Awardee</h4>
                                        <div class="bmail">skato1@ufl.edu</div>
                                    </div>
                                </div>

<!--Abdul-->                    <div class="member">
                                    <img src="img/lab/abduldozic.jpg" width="200" height="200" class="img-circle lab-img">
                                    <div class="bio">
                                        <h3><b>Abdul-Vehab Dozic</b></h3>
                                        <h4><i>Undergraduate Student</i></h4>
                                        <h4>Physics</h4>
                                        <div class="bmail">adozic@ufl.edu</div>
                                    </div>
                                </div>

<!--Riley                       <div class="member">
                                    <img src="img/lab/peiwengao.jpg" width="200" height="200" class="img-circle lab-img">
                                    <div class="bio">
                                        <h3><b>Peiwan (Riley) Gao</b></h3>
                                        <h4><i>Undergraduate Student</i></h4>
                                        <h4>Computer Science</h4>
                                        <div class="bmail">peiwengao@ufl.edu</div>
                                    </div>
                                </div>
--->

<!--Christopher                 <div class="member">
                                    <img src="img/lab/christopherfernandez.jpg" width="200" height="200" class="img-circle lab-img">
                                    <div class="bio">
                                        <h3><b>Christopher Fernandez</b></h3>
                                        <h4><i>Undergraduate Student</i></h4>
                                        <h4>Computer Science</h4>
                                        <div class="bmail">fernandezc@ufl.edu</div>
                                    </div>
                                </div>
-->

<!--Sruthika-->                 <div class="member">
                                    <img src="img/lab/sruthikabaviriseaty.jpg" width="200" height="200" class="img-circle lab-img">
                                    <div class="bio">
                                        <h3><b>Sruthika Baviriseaty</b></h3>
                                        <h4><i>Undergraduate Student</i></h4>
                                        <h4>Biomedical Engineering</h4>
                                        <div class="bmail">sbaviriseaty@ufl.edu</div>
                                    </div>
                                </div>

<!--Jared                       <div class="member">
                                    <img src="img/lab/jaredhixenbaugh.jpg" width="200" height="200" class="img-circle lab-img">
                                    <div class="bio">
                                        <h3><b>Jared Hixenbaugh</b></h3>
                                        <h4><i>Undergraduate Student</i></h4>
                                        <h4>Chemical Engineering</h4>
                                        <div class="bmail">jhixenbaugh@ufl.edu</div>
                                    </div>
                                </div>
-->

<!--Yiru-->                     <div class="member">
                                    <img src="img/lab/yirumu.jpg" width="200" height="200" class="img-circle lab-img">
                                    <div class="bio">
                                        <h3><b>Yiru Mu</b></h3>
                                        <h4><i>Undergraduate Student</i></h4>
                                        <h4>Biomedical Engineering</h4>
                                        <div class="bmail">yiru.mu@ufl.edu</div>
                                    </div>
                                </div>

<!--Nathan-->                   <div class="member">
                                    <img src="img/lab/nathanbarkdull.jpg" width="200" height="200" class="img-circle lab-img">
                                    <div class="bio">
                                        <h3><b>Nathan Barkdull</b></h3>
                                        <h4><i>Undergraduate Student</i></h4>
                                        <h4>Physics and Math</h4>
                                        <div class="bmail">nathan.barkdull@ufl.edu</div>
                                    </div>
                                </div>

                                <!--/Current Members-->





                                <!--Alumni-->
                                <div class="title text-center">
                                    <h3>Alumni</h3>
                                </div>

<!--Yao-->                      <a href="https://sites.google.com/view/yaoxiao" target="_blank">
                                <div class="member">
                                    <!--<img src="img/lab/yaoxiao.jpg" width="200" height="200" class="img-circle lab-img">-->
                                    <div class="bio">
                                        <h3><b>Yao Xiao</b></h3>
                                        <h5>Doctor of Philosophy, 2017-2020</h5>
                                        <h4>Postdoc Fellow at MD Anderson, BMES 2020 Career Development Award, Graduate Student Speaker at College of Engineering Commencement 2020</h4>
                                    </div>
                                </div>
                                </a>

<!--Daniel-->	                <div class="member">
                                    <!--<img src="img/lab/Daniel-basha.jpg" width="200" height="200" class="img-circle lab-img">-->
                                    <div class="bio">
                                        <h3><b>Daniel El Basha</b></h3>
                                        <h5>Undergraduate Researcher, NSF REU Student, 2017-2019</h5>
                                        <h4>PhD student at MD Anderson, NSF GRFP</h4>
                                    </div>
                                </div>

<!--Akshay-->	                <div class="member">
                                    <!--<img src="img/lab/Akshay.png" width="200" height="200" class="img-circle lab-img">-->
                                    <div class="bio">
                                        <h3><b>Akshay Mathavan</b></h3>
                                        <h5>Undergraduate Researcher, University Scholar Program, 2018-2019</h5>
                                        <h4>Doctor of Medicine student, UF</h4>
                                    </div>
                                </div>

<!--Akash-->		            <div class="member">
                                    <!--img src="img/lab/Akash.png" width="200" height="200" class="img-circle lab-img">-->
                                    <div class="bio">
                                        <h3><b>Akash Mathavan</b></h3>
                                        <h5>Undergraduate Researcher, University Scholar Program, 2018-2019</h5>
                                        <h4>Doctor of Medicine student, UF</h4>
                                    </div>
                                </div>


<!--Jun-->                      <a href="https://cn.linkedin.com/in/yangjunyi-li-6b8b97ab" target="_blank">
                                <div class="member">
                                    <!--<img src="img/lab/JunYi.jpg" width="200" height="200" class="img-circle lab-img">-->
                                    <div class="bio">
                                        <h3><b>Yangjunyi Li</b></h3>
                                        <h5>Masters Student, UF BME, 2017-2018</h5>
                                        <h4>Novartis Institutes for BioMedical Research</h4>
                                    </div>
                                </div>
                                </a>

<!--Yun-->	                    <a href="https://www.linkedin.com/in/yun-liang-51b17445" target="_blank">
                                <div class="member">
                                    <!--<img src="img/lab/YunLiang.jpg" class="img-circle lab-img">-->
                                    <div class="bio">
                                        <h3><b>Yun Liang</b></h3>
                                        <h5>PhD Student (As Intern), UF BME, 2018-2019</h5>
                                        <h4>PhD student, UF BME</h4>
                                    </div>
                                </div>
                                </a>

<!--Micheal-->                  <a href="https://www.linkedin.com/in/adeyosoyemic/" target="_blank">
                                <div class="member">
                                    <!--<img src="img/lab/micheal.jpg" class="img-circle lab-img">-->
                                    <div class="bio">
                                        <h3><b>Micheal Adeyosoye</b></h3>
                                        <h5>MS-PhD, FIU Bridge to PhD Fellowship, 2016-2017</h5>
                                        <h4>PhD Student, FIU</h4>
                                    </div>
                                </div>
                                </a>

<!--Jingan-->                   <a href="https://www.linkedin.com/in/jinganqu/" target="_blank">
                                <div class="member">
                                    <!--<img src="img/lab/jinganqu.jpg" class="img-circle lab-img">-->
                                    <div class="bio">
                                        <h3><b>Jingan Qu</b></h3>
                                        <h5>Masters Student, 2015-2017</h5>
                                        <h4>Data Scientist, Applied Research Center FIU</h4>
                                    </div>
                                </div>
                                </a>

<!--Daniel-->                   <a href="https://www.linkedin.com/in/danieldotparra/" target="_blank">
                                <div class="member">
                                    <!--<img src="img/lab/dani.jpg" class="img-circle lab-img">-->
                                    <div class="bio">
                                        <h3><b>Daniel Parra</b></h3>
                                        <h5>Masters Student, 2015-2016</h5>
                                        <h4>Software Engineer, ProActive Technologies, LLC</h4>
                                    </div>
                                </div>
                                </a>

<!--Haodi-->                    <div class="member">
                                    <!-- <img src="img/lab/haodi-jiang.jpg" class="img-circle lab-img"> -->
                                    <div class="bio">
                                        <h3><b>Haodi Jiang</b></h3>
                                        <h5>Masters Student, 2014-2015</h5>
                                        <h4>PhD student at NJIT</h4>
                                    </div>
                                </div>

<!--Sherman-->                  <a href="https://www.linkedin.com/in/sherman-ng-a3801523/" target="_blank">
                                <div class="member">
                                    <!-- <img src="img/lab/shermanng.jpg" class="img-circle lab-img"> -->
                                    <div class="bio">
                                        <h3><b>Sherman Ng</b></h3>
                                        <h5>M.Eng. 2010-2011 @ Cornell ECE</h5>
                                        <h4>Software Engineer, Microsoft</h4>
                                    </div>
                                </div>
                                </a>

<!--Yuanyuan-->                 <a href="https://www.linkedin.com/in/yyzcs/" target="_blank">
                                <div class="member">
                                    <!-- <img src="img/lab/yuanyuan.jpg" class="img-circle lab-img"> -->
                                    <div class="bio">
                                        <h3><b>Yuanyuan Zhu</b></h3>
                                        <h5>Visiting Undergraduate Student, 2016 Summer</h5>
                                        <h4>Masters Student, computer Science, University of South California</h4>
                                    </div>
                                </div>
                                </a>

<!--Xing-->                     <div class="member">
                                    <!-- <img src="img/lab/pang.jpg" class="img-circle lab-img"> -->
                                    <div class="bio">
                                        <h3><b>Xing Pang</b></h3>
                                        <h5>Visiting Graduate Student, 2015-2016</h5>
                                        <h4>Nanjing University of Science and Technology</h4>
                                    </div>
                                </div>

<!--Edda-->                     <a href="http://teacherweb.com/FL/JohnAFergusonHighSchool/EddaRivera/t.aspx" target="_blank">
                                <div class="member">
                                    <!-- <img src="img/lab/edda.jpg" class="img-circle lab-img"> -->
                                    <div class="bio">
                                        <h3><b>Edda Rivera</b></h3>
                                        <h5>NSF DoD RET Teacher, Best NSF RET Poster Award at FIU SCIS RET Presentation, 2016 Summer</h5>
                                        <h4>High School Teacher, John A. Ferguson Senior High School</h4>
                                    </div>
                                </div>
                                </a>

<!--Christian-->                <a href="http://teacherweb.com/FL/JohnAFergusonHighSchool/EddaRivera/t.aspx" target="_blank">
                                <div class="member">
                                    <!-- <img src="img/lab/chris.jpg" class="img-circle lab-img"> -->
                                    <div class="bio">
                                        <h3><b>Christian McDonald</b></h3>
                                        <h5>NSF DoD RET Teacher, Best NSF RET Poster Award at FIU SCIS RET Presentation, 2016 Summer</h5>
                                        <h4>High School Teacher, Miami Jackson Senior High School</h4>
                                    </div>
                                </div>
                                </a>

<!--Paul-->                     <a href="https://www.linkedin.com/in/paul-naghshineh-77a756112/" target="_blank">
                                <div class="member">
                                    <!-- <img src="img/lab/Paul.jpg" class="img-circle lab-img"> -->
                                    <div class="bio">
                                        <h3><b>Paul Naghshineh</b></h3>
                                        <h5>NSF DoD REU Student, 2016 Summer</h5>
                                        <h4>Gaumard Scientific</h4>
                                    </div>
                                </div>
                                </a>

                            </div>
                        </div>
                    </div>
                </div>
                <!--/Team-->

                <!--Research-->
                <div id="research" class="page">

                    <!--Research Summary-->
                    <div class="pageheader">
                        <div class="headercontent">
                            <div class="section-container">
                                <h2 class="title">Research Summary</h2>
                                <div class="row">
                                    <div class="col-md-8">
                                        <p>Smart learning and evaluation has revolutionized traditional medical data computing paradigm by converging machine learning, computer vision, big-data strategy and data mining into the conventional medical and clinical world. It has opened up tremendous opportunities in patient-centric medical imaging, robust neural computing, personal health tracking, and large-scale health informatics, etc. The medical informatics ranges over image, video, audio, text and various modalities such as CT, MRI, PET/CT, EEG and so on.</p>
                                        <p>Smart Medical Informatics Learning and Evaluation (SMILE) Group focuses on scientific approaches to bridge data and medicine. Our effort spans from safety-aware medical imaging, robust hemodynamics estimation, to innovation of the computing and analysis algorithms. In our medical informatics learning and evaluation research, we explore and leverage the large amount of medical data in various forms and modalities, to generate scientific enlightening or clinically useful information. With these words, we aim to push the fundamental limits of the medical analysis and to revolutionize the functional imaging and data interpretation.</p>
                                    </div>
                                    <div class="col-md-4">
                                        <div class="subtitle text-center">
                                            <h3>Interests</h3>
                                        </div>
                                        <ul class="ul-boxed list-unstyled">
                                            <li>Artificial Intelligence</li>
                                            <li>Brain Dynamics</li>
                                            <li>Medical Image Analysis</li>
                                            <li>Mutlimodal Data Mining</li>
                                            <li>Medical Machine Learning</li>
                                            <li>Neurological Diseases</li>
                                        </ul>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>
                    <!--/Research Summary-->

                    <!--NSF Funded Projects-->
                    <div class="section color-2">
                        <div class="section-container">
                            <div class="title text-center">
                                <h3>NSF Funded Projects</h3>
                            </div>
                            <div class="row">
                                <div class="col-md-12">
                                    <ul class="ul-withdetails">
                                        <li>
                                            <div class="row">
                                                <div class="col-sm-6 col-md-3">
                                                    <div class="image">
                                                        <img alt="image" src="img/acknowlegements/nsf_logo.gif" class="img-responsive">
                                                        <div class="imageoverlay">
                                                            <a href="https://www.bme.ufl.edu/labs/fang/projects/IIISmall2019.html" class="tooltips" title="External link" target="_blank"><i class="fa fa-search"></i></a>
                                                        </div>
                                                    </div>
                                                </div>
                                                <div class="col-sm-6 col-md-9">
                                                    <div class="meta">
                                                        <h3>IIS-1908299 III: Small: Collaborative Research: Modeling Multi-Level Connectivity of Brain Dynamics</h3>
                                                        <a href="./projects/IIISmall2019.html" target="_blank"><h4>Project Page</h4></a>
                                                        <a href="https://www.nsf.gov/awardsearch/showAward?AWD_ID=1908299&HistoricalAwards=false"><h4>NSF Award Abstract</h4></a>
                                                    </div>
                                                </div>
                                            </div>
                                            <div class="details">
                                                <p>The temporal dynamics of blood flows through the network of cerebral arteries and veins provides a window into the health of the human brain. Since the brain is vulnerable to disrupted blood supply, brain dynamics serves as a crucial indicator for many kinds of neurological diseases such as stroke, brain cancer, and Alzheimer's disease. Existing efforts at characterizing brain dynamics have predominantly centered on 'isolated' models in which data from single-voxel, single-modality, and single-subject are characterized. However, the brain is a vast network, naturally connected on structural and functional levels, and multimodal imaging provides complementary information on this natural connectivity. Thus, the current isolated models are deemed not capable of offering the platform necessary to enable many of the potential advancements in understanding, diagnosing, and treating neurological and cognitive diseases, leaving a critical gap between the current computational modeling capabilities and the needs in brain dynamics analysis. This project aims to bridge this gap by exploiting multi-scale structural (voxel, vasculature, tissue) connectivity and multi-modal (anatomical, angiography, perfusion) connectivity to develop an integrated connective computational paradigm for characterizing and understanding brain dynamics.</p>
                                            </div>
                                        </li>

                                        <li>
                                            <div class="row">
                                                <div class="col-sm-6 col-md-3">
                                                    <div class="image">
                                                        <img alt="image" src="img/acknowlegements/nsf_logo.gif" class="img-responsive">
                                                        <div class="imageoverlay">
                                                            <a href="https://www.nsf.gov/awardsearch/showAward?AWD_ID=1564892" target="_blank">
                                                                <i class="fa fa-search"></i>
                                                            </a>
                                                        </div>
                                                    </div>
                                                </div>
                                                <div class="col-sm-6 col-md-9">
                                                    <div class="meta">
                                                        <h3>IIS-1564892 CRII: SCH: Characterizing, Modeling and Evaluating Brain Dynamics</h3>
                                                        <a href="https://www.nsf.gov/awardsearch/showAward?AWD_ID=1564892" target="_blank"><h4>NSF Award Abstract</h4></a>
                                                    </div>
                                                </div>
                                            </div>
                                            <div class="details">
                                                <p>The temporal dynamics of blood flows through the network of cerebral arteries and veins provides a window into the health of the human brain. Since the brain is vulnerable to disrupted blood supply, brain dynamics serves as a crucial indicator for many kinds of neurological diseases such as stroke, brain cancer, and Alzheimer's disease. Existing efforts at characterizing brain dynamics have predominantly centered on 'isolated' models in which data from single-voxel, single-modality, and single-subject are characterized. However, the brain is a vast network, naturally connected on structural and functional levels, and multimodal imaging provides complementary information on this natural connectivity. Thus, the current isolated models are deemed not capable of offering the platform necessary to enable many of the potential advancements in understanding, diagnosing, and treating neurological and cognitive diseases, leaving a critical gap between the current computational modeling capabilities and the needs in brain dynamics analysis. This project aims to bridge this gap by exploiting multi-scale structural (voxel, vasculature, tissue) connectivity and multi-modal (anatomical, angiography, perfusion) connectivity to develop an integrated connective computational paradigm for characterizing and understanding brain dynamics.</p>
                                            </div>
                                        </li>

                                        <!---
                                        <li>
                                            <div class="row">
                                                <div class="col-sm-6 col-md-3">
                                                    <div class="image">
                                                        <img alt="image" src="img/lab/TEnsor.png" class="img-responsive">
                                                        <div class="imageoverlay">
                                                            <i class="fa fa-search"></i>
                                                        </div>
                                                    </div>
                                                </div>
                                                <div class="col-sm-6 col-md-9">
                                                    <div class="meta">
                                                        <h3>TENDER: TEnsor Non-local Deconvolution Enabled Radiation Reduction in CT Perfusion</h3>
                                                    </div>
                                                </div>
                                            </div>
                                            <div class="details">
                                                <p>Stroke is the leading cause of long-term disability and the second leading cause of mortality in the world, and exerts an enormous burden on the public health. CT remains one of the most widely used imaging modality for stroke diagnosis. However when coupled with CT perfusion, the excessive radiation exposure in repetitive imaging to assess treatment response and prognosis has raised significant public concerns regarding its potential hazards to both short- and longterm health outcomes. Tensor total variation has been proposed to reduce the necessary radiation dose in CT perfusion without comprising the image quality by fusing the information of the local anatomical structure with the temporal blood flow model. However the local search in the framework fails to leverage the non-local information in the spatio-temporal data. In this paper, we propose TENDER, an efficient framework of non-local tensor deconvolution to maintain the accuracy of the hemodynamic parameters and the diagnostic reliability in low radiation dose CT perfusion. The tensor total variation is extended using non-local spatio-temporal cubics for regularization to integrate contextual and non-local information. We also propose an efficient framework consisting of fast nearest neighbor search, accelerated optimization and parallel computing to improve the efficiency and scalability of the non-local spatio-temporal algorithm. Evaluations on clinical data of subjects with cerebrovascular disease and normal subjects demonstrate the advantage of non-local tensor deconvolution for reducing radiation dose in CT perfusion.</p>
                                            </div>
                                        </li>

                                        <li>
                                            <div class="row">
                                                <div class="col-sm-6 col-md-3">
                                                    <div class="image">
                                                        <img alt="image" src="img/lab/Health.png" class="img-responsive">
                                                        <div class="imageoverlay">
                                                            <i class="fa fa-search"></i>
                                                        </div>
                                                    </div>
                                                </div>
                                                <div class="col-sm-6 col-md-9">
                                                    <div class="meta">
                                                        <h3>Computational Health Informatics in the Big Data Age: A Survey</h3>
                                                    </div>
                                                </div>
                                            </div>
                                            <div class="details">
                                                <p>The explosive growth and widespread accessibility of digital health data have led to a surge of research activity in the healthcare and data sciences fields. The conventional approaches for health data management have achieved limited success as they are incapable of handling the huge amount of complex data with high volume, high velocity, and high variety. This article presents a comprehensive overview of the existing challenges, techniques, and future directions for computational health informatics in the big data age, with a structured analysis of the historical and state-of-the-art methods. We have summarized the challenges into four Vs (i.e., volume, velocity, variety, and veracity) and proposed a systematic data-processing pipeline for generic big data in health informatics, covering data capturing, storing, sharing, analyzing, searching, and decision support. Specifically, numerous techniques and algorithms in machine learning are categorized and compared. On the basis of this material, we identify and discuss the essential prospects lying ahead for computational health informatics in this big data age.</p>
                                            </div>
                                        </li>

                                        <li>
                                            <div class="row">
                                                <div class="col-sm-6 col-md-3">
                                                    <div class="image">
                                                        <img alt="image" src="img/lab/Near-Infrared.png" class="img-responsive">
                                                        <div class="imageoverlay">
                                                            <i class="fa fa-search"></i>
                                                        </div>
                                                    </div>
                                                </div>
                                                <div class="col-sm-6 col-md-9">
                                                    <div class="meta">
                                                        <h3>Near-Infrared Optical Imaging and Wound Segmentation in Lower Extremity Ulcers</h3>
                                                    </div>
                                                </div>
                                            </div>
                                            <div class="details">
                                                <p>Near-Infrared (NIR) optical imaging can reveal tissue oxygenation of the wound, complementing the visual inspection of the surface granulation. Herein, graph cuts algorithm is applied to segment NIR images of the wound from its peripheries.</p>
                                            </div>
                                        </li>

                                        <li>
                                            <div class="row">
                                                <div class="col-sm-6 col-md-3">
                                                    <div class="image">
                                                        <img alt="image" src="img/lab/BRFSS.jpg" class="img-responsive">
                                                        <div class="imageoverlay">
                                                            <i class="fa fa-search"></i>
                                                        </div>
                                                    </div>
                                                </div>
                                                <div class="col-sm-6 col-md-9">
                                                     <div class="meta">
                                                        <h3>Risk Factor Analysis: BRFSS</h3>
                                                        <p>Members: Yao Xiao, Daniel Parra</p>
                                                    </div>
                                                </div>
                                            </div>
                                            <div class="details">
                                                <p>The Behavioral Risk Factor Surveillance System (BRFSS) is the nation's premier system of health-related telephone surveys that collect state data about U.S. residents regarding their health-related risk behaviors, chronic health conditions, and use of preventive services. Established in 1984 with 15 states, BRFSS now collects data in all 50 states as well as the District of Columbia and three U.S. territories. BRFSS completes more than 400,000 adult interviews each year, making it the largest continuously conducted health survey system in the world.</p>
                                            </div>
                                        </li>

                                        <li>
                                            <div class="row">
                                                <div class="col-sm-6 col-md-3">
                                                    <div class="image">
                                                        <img alt="image" src="img/lab/Neuron.jpg"  class="img-responsive">
                                                        <div class="imageoverlay">
                                                            <i class="fa fa-search"></i>
                                                        </div>
                                                    </div>
                                                </div>
                                                <div class="col-sm-6 col-md-9">
                                                    <div class="meta">
                                                        <h3>Big Neuron Analytics</h3>
                                                        <p>Members: Peng Liu, Jingan Qu, Maryam Aghili</p>
                                                    </div>
                                                </div>
                                            </div>
                                            <div class="details">
                                                <p>We are going to analysize different neuron from many species by  data mining and deep learning algorithm seperately. </p>
                                            </div>
                                        </li>
                                        -->

                                    </ul>
                                </div>
                            </div>
                        </div>
                    </div>
                    <!--/NSF Funded Projects-->

                </div>
                <!--/Research-->

                <!--Publications-->
                <div id="publications" class="page">
                    <div class="page-container">
                        <div class="pageheader">
                            <div class="headercontent">
                                <div class="section-container">
                                    <h2 class="title">Selected Publications</h2>
                                    <div class="row">
                                        <div class="col-md-12">
                                            <p></p>
                                        </div>
                                    </div>
                                </div>
                            </div>
                        </div>
                        <div class="pagecontents">
                            <div class="section color-1" id="filters">
                                <div class="section-container">
                                    <div class="row">
                                        <div class="col-md-3">
                                            <h3>Filter by type:</h3>
                                        </div>
                                        <div class="col-md-6">
                                            <select id="cd-dropdown" name="cd-dropdown" class="cd-select">
                                            <!-- Publication Catergories Start-->
                                            <option class="filter" value="all" selected>All types (91)</option>
                                            <option class="filter" value="Journal">Journal (26)</option>
                                            <option class="filter" value="Conference">Conference (59)</option>
                                            <option class="filter" value="Thesis">Thesis (1)</option>
                                            <option class="filter" value="Patents">Patents (2)</option>
                                            <option class="filter" value="Book Chapters">Book Chapters (2)</option>
                                            <option class="filter" value="Presentations">Presentations (1)</option>
                                            <!-- Publication Catergories End-->
                                            </select>
                                        </div>
                                        <div class="col-md-3" id="sort">
                                            <span>Sort by year:</span>
                                            <div class="btn-group pull-right">
                                                <button type="button" data-sort="data-year" data-order="desc" class="sort btn btn-default"><i class="fa fa-sort-numeric-asc"></i></button>
                                                <button type="button" data-sort="data-year" data-order="asc" class="sort btn btn-default"><i class="fa fa-sort-numeric-desc"></i></button>
                                            </div>
                                        </div>
                                    </div>
                                </div>
                            </div>
                            <div class="section color-2" id="pub-grid">
                                <div class="section-container">
                                    <div class="row">
                                        <div class="col-md-12">

                                            <!-- Publications  Start -->
                                            <div class="pitems">

				                                <!--- Publication Example Code
                                                Journal
                                                <div class="item mix Journal" data-year="2020">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                            <a href="https://www.osapublishing.org/josaa/abstract.cfm?uri=josaa-37-8-1249" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/osa-josaa.jpg" class="attachment-medium size-medium wp-post image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Physiological wound assessment from coregistered and segmented tissue hemoglobin maps</h4>
                                                            <span class="label label-primary">Journal</span>
                                                            <div class="pubauthor">E. A. Robledo, R. Schutzman, <strong>R. Fang</strong>, C. Fernandez, R. Kwasinski, K. Leiva, F. Perez-Clavijo, and A. Godavarty</div>
                                                            <div class="pubcite">Journal of the Optical Society of America A</div>
                                                            <div class="pubdate">Publication Year: July, 2020</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                            <p>A handheld near-infrared optical scanner (NIROS) was recently developed to map for effective changes in oxy- and deoxyhemoglobin concentration in diabetic foot ulcers (DFUs) across weeks of treatment. Herein, a coregistration and image segmentation approach was implemented to overlay hemoglobin maps onto the white light images of ulcers. Validation studies demonstrated over 97% accuracy in coregistration. Coregistration was further applied to a healing DFU across weeks of healing. The potential to predict changes in wound healing was observed when comparing the coregistered and segmented hemoglobin concentration area maps to the visual area of the wound.
                                                    </div>
                                                </div>

                                                Conference
                                                <div class="item mix Conference" data-year="2020">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="https://neuromodec.com/nyc-neuromodulation-online-2020/index.html" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/Neuromodec.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Determinants of Treatment Response to Transcranial Direct Current Stimulation</h4>
                                                            <h5><strong>Outstanding Presentation by Early Career Scientist Award</strong></h5>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor">Albizu A, <strong>Fang R</strong> Indahlastari A, Nissim NR, OShea A, Woods AJ</div>
                                                            <div class="pubcite">5th Annual NYC Neuromodulation Conference</div>
                                                            <div class="pubdate">Publication Year: April 20-22, 2020</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                </div>


                                                Book
                                                <div class="item mix Book Chapters" data-year="2019">
                                                    <div class="pubmain">
                                                        <img width="125" src="img/pubs/Springer.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        <h4 class="pubtitle">Deep Spatial-Temporal Convolutional Neural Networks for Medical Image Restoration Deep Learning and Convolutional Neural Networks for Medical Image Computing </h4>
                                                        <div class="pubauthor">Yao Xiao, Skylar Stolte, Peng Liu, Yun Liang, Pina Sanelli, Ajay Gupta, Jana Ivanidze, and <strong>Ruogu Fang</strong> </div>
                                                        <div class="pubcite"><span class="label label-warning">Book Chapter </span>Deep Learning and Convolutional Neural Networks for Medical Image Computing, Springer Publisher, 2019. </div>
                                                    </div>
                                                </div>

                                                Patents
                                                <div class="item mix Patents" data-year="2018">
                                                    <div class="pubmain">
                                                    <div class="pubassets">
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                        </div>
                                                        <h4 class="pubtitle">
                                                            Neural Network Evolution Using Expedited Genetic Algorithm for Medical Image Denoising
                                                        </h4>
                                                        <div class="pubauthor">Peng Liu and <strong> Ruogu Fang</strong></div>
                                                        <div class="pubcite">
                                                            <span class="label label-info">Patents</span>Ref. No: UF-17344, Filed on 9/10/2018, Provisional Patent. US 62/728,995
                                                        </div>
                                                    </div>
                                                </div>

                                                -->

                                                <!--Preferred Image Sizes: Width: 125px-->
                                                <!--2020-->
                                                <div class="item mix Journal" data-year="2020">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <!--<a href="" class="tooltips" title="PDF" target="_blank">
                                                            <i class="fa fa-file-pdf-o"></i>-->
                                                            
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                            <a href="https://www.nature.com/articles/s41598-020-80312-2" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/SciReports.jpg" class="attachment-medium size-medium wp-post image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Modular machine learning for Alzheimer's disease classification from retinal vasculature</h4>
                                                            <span class="label label-warning">Journal</span>
                                                            <div class="pubauthor">Jianqiao Tian, Glenn Smith, Han Guo, Boya Liu, Zehua Pan, Zijie Wang, Shuangyu Xiong, <u><b>Ruogu Fang</b></u></div>
                                                            <div class="pubcite">Nature Scientific Reports</div>
                                                            <div class="pubdate">Publication Year: 2020/2021</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Alzheimer's disease is the leading cause of dementia. The long progression period in Alzheimer's disease provides a possibility for patients to get early treatment by having routine screenings. However, current clinical diagnostic imaging tools do not meet the specific requirements for screening procedures due to high cost and limited availability. In this work, we took the initiative to evaluate the retina, especially the retinal vasculature, as an alternative for conducting screenings for dementia patients caused by Alzheimer's disease. Highly modular machine learning techniques were employed throughout the whole pipeline. Utilizing data from the UK Biobank, the pipeline achieved an average classification accuracy of 82.44%. Besides the high classification accuracy, we also added a saliency analysis to strengthen this pipeline's interpretability. The saliency analysis indicated that within retinal images, small vessels carry more information for diagnosing Alzheimer's diseases, which aligns with related studies.</p>
                                                    </div>
                                                </div>
                                                    
                                                <div class="item mix Journal" data-year="2020">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="https://www.brainstimjrnl.com/article/S1935-861X(20)30268-0/pdf" class="tooltips" title="PDF" target="_blank">
                                                            <i class="fa fa-file-pdf-o"></i>
                                                            
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                            <a href="https://www.sciencedirect.com/science/article/pii/S1935861X20302680" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/brainstim.gif" class="attachment-medium size-medium wp-post image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Machine learning and individual variability in electric field characteristics predict tDCS treatment response.</h4>
                                                            <span class="label label-warning">Journal</span>
                                                            <div class="pubauthor">Alejandro Albizu, <strong>Ruogu Fang</strong>, Aprinda Indahlastari, Andrew OShea, Skylar E. Stolte, Kyle B. See, Emanuel M. Boutzoukas, Jessica N. Kraft, Nicole R. Nissim and Adam J. Woods</div>
                                                            <div class="pubcite">Brain Stimulation</div>
                                                            <div class="pubdate">Publication Year: November/December, 2020</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <h5>Background</h5>
                                                        <p>Transcranial direct current stimulation (tDCS) is widely investigated as a therapeutic tool to enhance cognitive function in older adults with and without neurodegenerative disease. Prior research demonstrates that electric current delivery to the brain can vary significantly across individuals. Quantification of this variability could enable person-specific optimization of tDCS outcomes. This pilot study used machine learning and MRI-derived electric field models to predict working memory improvements as a proof of concept for precision cognitive intervention.</p>
                                                        
                                                        <h5>Methods</h5>
                                                        <p>Fourteen healthy older adults received 20 minutes of 2 mA tDCS stimulation (F3/F4) during a two-week cognitive training intervention. Participants performed an N-back working memory task pre-/post-intervention. MRI-derived current models were passed through a linear Support Vector Machine (SVM) learning algorithm to characterize crucial tDCS current components (intensity and direction) that induced working memory improvements in tDCS responders versus non-responders.</p>
                                                        
                                                        <h5>Main results</h5>
                                                        <p>SVM models of tDCS current components had 86% overall accuracy in classifying treatment responders vs. non-responders, with current intensity producing the best overall model differentiating changes in working memory performance. Median current intensity and direction in brain regions near the electrodes were positively related to intervention responses.</p>

                                                        <h5>Conclusions</h5>
                                                        <p>This study provides the first evidence that pattern recognition analyses of MRI-derived tDCS current models can provide individual prognostic classification of tDCS treatment response with 86% accuracy. Individual differences in current intensity and direction play important roles in determining treatment response to tDCS. These findings provide important insights into mechanisms of tDCS response as well as proof of concept for future precision dosing models of tDCS intervention.</p>
                                                        
                                                        <p><img width="600" src="https://ars.els-cdn.com/content/image/1-s2.0-S1935861X20302680-fx1_lrg.jpg">
                                                    </div>
                                                </div>
                                                
                                                
                                                <div class="item mix Conference" data-year="2020">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <!--Update Link after meeting-->
                                                            <a href="https://www.rsna.org/annual-meeting" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/RSNA.png" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Machine Learning for Parkinsons Disease Diagnosis Using Fundus Eye Images</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor">Maximillian Diaz, Jianqiao Tian, <u><b>Ruogu Fang</b></u></div>
                                                            <div class="pubcite">Annual Meeting of Radiology Society of North America</div>
                                                            <div class="pubdate">Publication Year: December, 2020</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2020">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="https://neuromodec.com/nyc-neuromodulation-online-2020/index.html" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/AHA.png" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Artificial Intelligence For Characterizing Heart Failure In Cardiac Magnetic Resonance Images</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor">Skylar Stolte, Yonghui Wu, William R Hogan, Yan Gong, <u><b>Ruogu Fang</b></u></div>
                                                            <div class="pubcite">American Heart Association Scientific Session</div>
                                                            <div class="pubdate">Publication Year: November 13-17, 2020</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                </div>
                                                
                                                
                                                <div class="item mix Journal" data-year="2020">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                            <a href="https://www.osapublishing.org/josaa/abstract.cfm?uri=josaa-37-8-1249" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/osa-josaa.jpg" class="attachment-medium size-medium wp-post image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Physiological wound assessment from coregistered and segmented tissue hemoglobin maps</h4>
                                                            <span class="label label-warning">Journal</span>
                                                            <div class="pubauthor">E. A. Robledo, R. Schutzman, <strong>R. Fang</strong>, C. Fernandez, R. Kwasinski, K. Leiva, F. Perez-Clavijo, and A. Godavarty</div>
                                                            <div class="pubcite">Journal of the Optical Society of America A</div>
                                                            <div class="pubdate">Publication Year: July, 2020</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                            <p>A handheld near-infrared optical scanner (NIROS) was recently developed to map for effective changes in oxy- and deoxyhemoglobin concentration in diabetic foot ulcers (DFUs) across weeks of treatment. Herein, a coregistration and image segmentation approach was implemented to overlay hemoglobin maps onto the white light images of ulcers. Validation studies demonstrated over 97% accuracy in coregistration. Coregistration was further applied to a healing DFU across weeks of healing. The potential to predict changes in wound healing was observed when comparing the coregistered and segmented hemoglobin concentration area maps to the visual area of the wound.
                                                    </div>
                                                </div>


                                                <div class="item mix Journal" data-year="2020">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                            <a href="https://www.sciencedirect.com/science/article/pii/S1361841520301067" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/MIA.gif" class="attachment-medium size-medium wp-post image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">A Survey on Medical Image Analysis in Diabetic Retinopathy</h4>
                                                            <span class="label label-warning">Journal</span>
                                                            <div class="pubauthor">Skylar Stolte, <u><b>Ruogu Fang</b></u></div>
                                                            <div class="pubcite">Medical Image Analysis, In Press</div>
                                                            <div class="pubdate">Publication Year: May 30, 2020</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                            <p>Diabetic Retinopathy (DR) represents a highly-prevalent complication of diabetes in which individuals suffer from damage to the blood vessels in the retina. The disease manifests itself through lesion presence, starting with microaneurysms, at the nonproliferative stage before being characterized by neovascularization in the proliferative stage. Retinal specialists strive to detect DR early so that the disease can be treated before substantial, irreversible vision loss occurs. The level of DR severity indicates the extent of treatment necessary - vision loss may be preventable by effective diabetes management in mild (early) stages, rather than subjecting the patient to invasive laser surgery. Using artificial intelligence (AI), highly accurate and efficient systems can be developed to help assist medical professionals in screening and diagnosing DR earlier and without the full resources that are available in specialty clinics. In particular, deep learning facilitates diagnosis earlier and with higher sensitivity and specificity. Such systems make decisions based on minimally handcrafted features and pave the way for personalized therapies. Thus, this survey provides a comprehensive description of the current technology used in each step of DR diagnosis. First, it begins with an introduction to the disease and the current technologies and resources available in this space. It proceeds to discuss the frameworks that different teams have used to detect and classify DR. Ultimately, we conclude that deep learning systems offer revolutionary potential to DR identification and prevention of vision loss.
                                                    </div>
                                                </div>


                                                <div class="item mix Presentations" data-year="2020">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="https://neuromodec.com/nyc-neuromodulation-online-2020/index.html" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/Neuromodec.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Determinants of Treatment Response to Transcranial Direct Current Stimulation</h4>
                                                            <h5><strong>Outstanding Presentation by Early Career Scientist Award</strong></h5>
                                                            <span class="label label-default">Presentation</span>
                                                            <div class="pubauthor">Albizu A, <strong>Fang R</strong>, Indahlastari A, Nissim NR, OShea A, Woods AJ</div>
                                                            <div class="pubcite">5th Annual NYC Neuromodulation Conference</div>
                                                            <div class="pubdate">Publication Year: April 20-22, 2020</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2020">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="https://www.the-ins.org/meetings/denver2020/" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" height="125" src="img/pubs/INS.png" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Building Personalized Medicine Models for Therapeutic Applications of Transcranial Electrical Stimulation</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor">Albizu A, Indahlastari A, Nissim NR, OShea A, <strong>Ruogu Fang</strong>, Woods AJ</div>
                                                            <div class="pubcite">48th Annual Meeting of the International Neuropsychological Society</div>
                                                            <div class="pubdate">Publication Year: February 2020</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                </div>

                                                <div class="item mix Journal" data-year="2020">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                            <a href="https://www.sciencedirect.com/science/article/abs/pii/S1361841520300219" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/MIA.gif" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Domain-Invariant Interpretable Fundus Image Quality Assessment</h4>
                                                            <span class="label label-warning">Journal</span>
                                                            <div class="pubauthor">Yaxin Shen, Bin Sheng, <strong>Ruogu Fang</strong>, Huating Li, Ling Dai, Skylar Stolte*, Jing Qin, Weiping Jia, Dinggang Shen</div>
                                                            <div class="pubcite">Medical Image Analysis, volume 61</div>
                                                            <div class="pubdate">Publication Year: 2020</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Objective and quantitative assessment of fundus image quality is essential for the diagnosis of retinal diseases. The major factors in fundus image quality assessment are image artifact, clarity, and field definition. Unfortunately, most of existing quality assessment methods focus on the quality of overall image, without interpretable quality feedback for real-time adjustment. Furthermore, these models are often sensitive to the specific imaging devices, and cannot generalize well under different imaging conditions. This paper presents a new multi-task domain adaptation framework to automatically assess fundus image quality. The proposed framework provides interpretable quality assessment with both quantitative scores and quality visualization for potential real-time image recapture with proper adjustment. In particular, the present approach can detect optic disc and fovea structures as landmarks, to assist the assessment through coarse-to-fine feature encoding. The framework also exploit semi-tied adversarial discriminative domain adaptation to make the model generalizable across different data sources. Experimental results demonstrated that the proposed algorithm outperforms different state-of-the-art approaches and achieves an area under the ROC curve of 0.9455 for the overall quality classification.
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2020">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="https://siim.org/page/siim2020" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/SIIM.png" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Multi-Series CT Image Super-Resolution by using Transfer Generative Adversarial Network</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor">Yao Xiao, Manuel M. Arreola, Izabella Barreto, Wesley E. Bolch, W. Christopher Fox, Keith Peters, Dhanashree A. Rajderkar, John H. Rees, <u><b>Ruogu Fang</b></u></div>
                                                            <div class="pubcite">Society for Imaing Informatics in Medicine Annual Meeting in Austin, TX</div>
                                                            <div class="pubdate">Publication Year: June 24-26, 2020</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2020">
                                                    <div class="pubmain">
                                                        <div class="pubassets">

                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/IEEEImaging.png" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Transfer-GAN: Multimodal CT Image Super-Resolution via Transfer Generative Adversarial Networks</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor">Yao Xiao, Keith R. Peters, W. Christopher Fox, John H. Rees, Dhanashree A. Rajderkar, Manuel M. Arreola, Izabella Barreto, Wesley E. Bolch, <u><b>Ruogu Fang</b></u></div>
                                                            <div class="pubcite">IEEE International Symposium on Biomedical Imaging in Iowa City, IA</div>
                                                            <div class="pubdate">Publication Year: April 3-7, 2020</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2020">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="https://spie.org/conferences-and-exhibitions/past-conferences-and-exhibitions/medical-imaging-2020" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/SPIE.png" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Transfer Generative Adversarial Network for Multimodal CT Image Super-Resolution</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor">Yao Xiao, <strong>Ruogu Fang</strong></div>
                                                            <div class="pubcite">SPIE Medical Imaging in Houston, TX</div>
                                                            <div class="pubdate">Publication Year: February 15-20, 2020</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2020">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="http://snamc2020.jpn.org/" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img height="50" src="img/pubs/SNAMC2020.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Monte Carlo Dosimetry For CT Brain Perfusion Studies Utilizing Volumetric Acquisitions</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor">Justin L Brown, Daniel El Basha∗, Nathalie Correa, Yao Xiao, Izabella Barreto, <strong>Ruogu Fang</strong>, Chan Kim, Wesley E. Bolch </div>
                                                            <div class="pubcite">Joint International Conference on Supercomputing in Nuclear Applications + Monte Carlo</div>
                                                            <div class="pubdate">Publication Year: 2020</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2020">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" height="170" src="img/pubs/ASCPT.png" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">ZMAT4 and DOCK9 Variants Associated with Heart Failure in Breast Cancer Patients in the UK Biobank data</h4><font color="red"><h5><strong>Presidential Trainee Award, 2020 David J. Goldstein Trainee Award</strong></h5></font>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor">Marwa Tantawy, Sonal Singh, Guang Yang, Matt Gitzendanner, Yiqing Chen, Yonghui Wu, <strong>Ruogu Fang</strong>, William Hogan, Yan Gong</div>
                                                            <div class="pubcite">American Society for Clinical Pharmacology and Therapeutics Annual Meeting in Houston, TX</div>
                                                            <div class="pubdate">Publication Year: March 18-21, 2020</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                </div>

                                                <div class="item mix Journal" data-year="2020">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                            <a href="https://www.sciencedirect.com/science/article/pii/S0167610519306269" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/JWEIA.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Low-Rise Gable Roof Buildings Pressure Prediction using Deep Neural Networks</h4>
                                                            <span class="label label-warning">Journal </span>
                                                            <div class="pubauthor">Jianqiao Tian, Kurtis Gurley, Maximillian Diaz, Pedro L. Fernández Cabán, Forrest J. Masters, <u><b>Ruogu Fang</b></u></div>
                                                            <div class="pubcite">Journal of Wind Engineering & Industrial Aerodynamics</div>
                                                            <div class="pubdate">Publication Year: 2020</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>This paper presents a deep neural network (DNN) based approach for predicting mean and peak wind pressure coefficients on the surface of a scale model low-rise, gable roof residential building. Pressure data were collected on the model at multiple prescribed wind directions and terrain roughnesses. The resultant pressure coefficients quantified from a subset of these directions and terrains were used to train an DNN to predict coefficients for directions and terrains excluded from the training. The approach is able to leverage a variety of input conditions to predict pressure coefficients with high accuracy, while the prior work has limited flexibility with the number of input variables and yielded lower prediction accuracy. A two-step nested DNN procedure is introduced to improve the prediction of peak coefficients. The optimal correlation coefficients of return predictions were 0.9993 and 0.9964, for mean and peak coefficient prediction, respectively. The concept of super resolution based on global prediction was also discussed. With a sufficiently large database, the proposed DNN-based approach can augment existing experimental methods to improve the yield of knowledge while reducing the number of tests required to gain that knowledge.
                                                    </div>
                                                </div>

                                                <div class="item mix Journal" data-year="2020">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                            <a href="https://www.sciencedirect.com/science/article/pii/S1361841519301033?via%3Dihub" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/MIA.gif" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">IDRiD: Diabetic Retinopathy – Segmentation and Grading Challenge</h4>
                                                            <span class="label label-warning">Journal</span>
                                                            <div class="pubauthor">Prasanna Porwal, Samiksha Pachade, Manesh Kokare, Girish Deshmukh, Jaemin Son, Woong Bae, Lihong Liu, Jianzong Wang, Xinhui Liu, Liangxin Gao, TianBo Wu, Jing Xiao, Fengyan Wang, Baocai Yin, Yunzhi Wang, Gopichandh Danala, Linsheng He, Yoon Ho Choi, Yeong Chan Lee, Sang-Hyuk Jung, Zhongyu Li, Xiaodan Sui, Junyan Wu, Xiaolong Li, Ting Zhou, Janos Toth, Agnes Baran, Avinash Kori, Sai Saketh Chennamsetty, Mohammed Safwan, Varghese Alex, Xingzheng Lyu,r, Li Cheng,D, Qinhao Chu, Pengcheng Li, Xin Ji, Sanyuan Zhang, Yaxin Shen, Ling Dai$, Oindrila Saha, Rachana Sathish, Tânia Melo, Teresa Araújo, Balazs Harangi, Bin Sheng, <strong>Ruogu Fang</strong>, Debdoot Sheet, Andras Hajdu, Yuanjie Zheng, Ana Maria Mendonça, Shaoting Zhang, Aurélio Campilho, Bin Zheng, Dinggang Shen, Luca Giancardo, Gwenolé Quellec, Fabrice Mériaudeau</div>
                                                            <div class="pubcite">Medical Image Analysis, volume 59</div>
                                                            <div class="pubdate">Publication Year: 2020</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Diabetic Retinopathy (DR) is the most common cause of avoidable vision loss, predominantly affecting the working-age population across the globe. Screening for DR, coupled with timely consultation and treatment, is a globally trusted policy to avoid vision loss. However, implementation of DR screening programs is challenging due to the scarcity of medical professionals able to screen a growing global diabetic population at risk for DR. Computer-aided disease diagnosis in retinal image analysis could provide a sustainable approach for such large-scale screening effort. The recent scientific advances in computing capacity and machine learning approaches provide an avenue for biomedical scientists to reach this goal. Aiming to advance the state-of-the-art in automatic DR diagnosis, a grand challenge on “Diabetic Retinopathy – Segmentation and Grading” was organized in conjunction with the IEEE International Symposium on Biomedical Imaging (ISBI - 2018). In this paper, we report the set-up and results of this challenge that is primarily based on Indian Diabetic Retinopathy Image Dataset (IDRiD). There were three principal sub-challenges: lesion segmentation, disease severity grading, and localization of retinal landmarks and segmentation. These multiple tasks in this challenge allow to test the generalizability of algorithms, and this is what makes it different from existing ones. It received a positive response from the scientific community with 148 submissions from 495 registrations effectively entered in this challenge. This paper outlines the challenge, its organization, the dataset used, evaluation methods and results of top-performing participating solutions. The top-performing approaches utilized a blend of clinical information, data augmentation, and an ensemble of models. These findings have the potential to enable new developments in retinal image analysis and image-based DR screening in particular.
                                                    </div>
                                                </div>

                                                <div class="item mix Journal" data-year="2020">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i></a>
                                                            <a href="https://www.sciencedirect.com/science/article/pii/S1361841519301100" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/MIA.gif" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">REFUGE Challenge: A Unified Framework for Evaluating Automated Methods for Glaucoma Assessment from Fundus Photographs</h4>
                                                            <span class="label label-warning">Journal </span>
                                                            <div class="pubauthor">Jose Ignacio Orlando, Huazhu Fu, Joa ̃o Barbossa Breda, Karel van Keer, Deepti R. Bathula, Andre ́s Diaz-Pinto, <strong>Ruogu Fang</strong>, Pheng-Ann Heng, Jeyoung Kim, JoonHo Lee, Joonseok Lee, Xiaoxiao Li, Peng Liu, Shuai Lu, Balamurali Murugesan, Valery Naranjo, Sai Samarth R. Phaye, Sharath M. Shankaranarayana, Apoorva Sikka, Jaemin Son, Anton van den Hengel, Shujun Wang, Junyan Wu, Zifeng Wu, Guanghui Xu, Yongli Xu, Pengshuai Yin, Fei Li, Xiulan Zhang, Yanwu Xu, Hrvoje Bogunovic ́</div>
                                                            <div class="pubcite">Medical Image Analysis, volume 59</div>
                                                            <div class="pubdate">Publication Year: 2020</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Glaucoma is one of the leading causes of irreversible but preventable blindness in working age populations. Color fundus photography (CFP) is the most cost-effective imaging modality to screen for retinal disorders. However, its application to glaucoma has been limited to the computation of a few related biomarkers such as the vertical cup-to-disc ratio. Deep learning approaches, although widely applied for medical image analysis, have not been extensively used for glaucoma assessment due to the limited size of the available data sets. Furthermore, the lack of a standardize benchmark strategy makes difficult to compare existing methods in a uniform way. In order to overcome these issues we set up the Retinal Fundus Glaucoma Challenge, REFUGE (https://refuge.grand-challenge.org), held in conjunction with MICCAI 2018. The challenge consisted of two primary tasks, namely optic disc/cup segmentation and glaucoma classification. As part of REFUGE, we have publicly released a data set of 1200 fundus images with ground truth segmentations and clinical glaucoma labels, currently the largest existing one. We have also built an evaluation framework to ease and ensure fairness in the comparison of different models, encouraging the development of novel techniques in the field. 12 teams qualified and participated in the online challenge. This paper summarizes their methods and analyzes their corresponding results. In particular, we observed that two of the top-ranked teams outperformed two human experts in the glaucoma classification task. Furthermore, the segmentation results were in general consistent with the ground truth annotations, with complementary outcomes that can be further exploited by ensembling the results.
                                                    </div>
                                                </div>




                                                <!--2019-->
                                                <div class="item mix Book Chapters" data-year="2019">
                                                    <div class="pubmain">
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/Springer.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Deep Spatial-Temporal Convolutional Neural Networks for Medical Image Restoration Deep Learning and Convolutional Neural Networks for Medical Image Computing</h4>
                                                            <span class="label label-success">Book Chapter</span>
                                                            <div class="pubauthor">Yao Xiao, Skylar Stolte, Peng Liu, Yun Liang, Pina Sanelli, Ajay Gupta, Jana Ivanidze, <u><b>Ruogu Fang</b></u></div>
                                                            <div class="pubcite">Deep Learning and Convolutional Neural Networks for Medical Image Computing, Springer Publisher</div>
                                                            <div class="pubdate">Publication Year: 2019</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                </div>

                                                <div class="item mix Book Chapters" data-year="2019">
                                                    <div class="pubmain">
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/CRC.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle"> Big Data in Computational Health Informatics</h4>
                                                            <span class="label label-success">Book Chapter</span>
                                                            <div class="pubauthor"><u><b>Ruogu Fang</b></u>, Samira Pouyanfar, Yimin Yang, Yao Xiao, Jianqiao Tian, Shu-Ching Chen, and S.S. Iyengar </div>
                                                            <div class="pubcite">Big Data in Multimodal Medical Imaging, CRC Publisher</div>
                                                            <div class="pubdate">Publication Year: 2019</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                </div>

                                                <div class="item mix Journal" data-year="2019">
                                                    <div class="pubmain">
                                                        <!--
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse"><i class="fa fa-expand"></i></a>
                                                            <a href="https://www.frontiersin.org/articles/10.3389/fneur.2019.00647/full" class="tooltips" title="External link" target="_blank"><i class="fa fa-external-link"></i></a>
                                                        </div>
                                                        -->
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/Lancet.png" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Development and Validation of the Automated Imaging Diﬀerentiation in Parkinsonism (AID-P): A Multi-Site Machine Learning Study </h4>
                                                            <span class="label label-warning">Journal</span>
                                                            <div class="pubauthor">Derek B. Archer, ..., <strong>Ruogu Fang</strong>, ..., David E. Vaillancourt</div>
                                                            <div class="pubcite">The Lancet Digital Health</div>
                                                            <div class="pubdate">Publication Year: 2019</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <!--
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Computed Tomography Perfusion (CTP) imaging is a cost-effective and fast approach to provide diagnostic images for acute stroke treatment. Its cine scanning mode allows the visualization of anatomic brain structures and blood flow; however, it requires contrast agent injection and continuous CT scanning over an extended time. In fact, the accumulative radiation dose to patients will increase health risks such as skin irritation, hair loss, cataract formation, and even cancer. Solutions for reducing radiation exposure include reducing the tube current and/or shortening the X-ray radiation exposure time. However, images scanned at lower tube currents are usually accompanied by higher levels of noise and artifacts. On the other hand, shorter X-ray radiation exposure time with longer scanning intervals will lead to image information that is insufficient to capture the blood flow dynamics between frames. Thus, it is critical for us to seek a solution that can preserve the image quality when the tube current and the temporal frequency are both low. We propose STIR-Net in this paper, an end-to-end spatial-temporal convolutional neural network structure, which exploits multi-directional automatic feature extraction and image reconstruction schema to recover high-quality CT slices effectively. With the inputs of low-dose and low-resolution patches at different cross-sections of the spatio-temporal data, STIR-Net blends the features from both spatial and temporal domains to reconstruct high-quality CT volumes. In this study, we finalize extensive experiments to appraise the image restoration performance at different levels of tube current and spatial and temporal resolution scales.The results demonstrate the capability of our STIR-Net to restore high-quality scans at as low as 11% of absorbed radiation dose of the current imaging protocol, yielding an average of 10% improvement for perfusion maps compared to the patch-based log likelihood method.
                                                    </div>
                                                    -->
                                                </div>

                                                <div class="item mix Journal" data-year="2019">
                                                    <div class="pubmain">
                                                        <!--
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse"><i class="fa fa-expand"></i></a>
                                                            <a href="https://www.frontiersin.org/articles/10.3389/fneur.2019.00647/full" class="tooltips" title="External link" target="_blank"><i class="fa fa-external-link"></i></a>
                                                        </div>
                                                        -->
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/JAMIA.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">  Identifying Relations of Medications with Adverse Drug Events Using Recurrent Convolutional Neural Networks and Gradient Boosting </h4>
                                                            <span class="label label-warning">Journal</span>
                                                            <div class="pubauthor">Xi Yang, Jiang Bian, <strong>Ruogu Fang</strong>, Ragnhildur I. Bjarnadottir, William R. Hogan, and Yonghui Wu</div>
                                                            <div class="pubcite">Journal of the American Medical Informatics Association</div>
                                                            <div class="pubdate">Publication Year: 2019</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <!--
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Computed Tomography Perfusion (CTP) imaging is a cost-effective and fast approach to provide diagnostic images for acute stroke treatment. Its cine scanning mode allows the visualization of anatomic brain structures and blood flow; however, it requires contrast agent injection and continuous CT scanning over an extended time. In fact, the accumulative radiation dose to patients will increase health risks such as skin irritation, hair loss, cataract formation, and even cancer. Solutions for reducing radiation exposure include reducing the tube current and/or shortening the X-ray radiation exposure time. However, images scanned at lower tube currents are usually accompanied by higher levels of noise and artifacts. On the other hand, shorter X-ray radiation exposure time with longer scanning intervals will lead to image information that is insufficient to capture the blood flow dynamics between frames. Thus, it is critical for us to seek a solution that can preserve the image quality when the tube current and the temporal frequency are both low. We propose STIR-Net in this paper, an end-to-end spatial-temporal convolutional neural network structure, which exploits multi-directional automatic feature extraction and image reconstruction schema to recover high-quality CT slices effectively. With the inputs of low-dose and low-resolution patches at different cross-sections of the spatio-temporal data, STIR-Net blends the features from both spatial and temporal domains to reconstruct high-quality CT volumes. In this study, we finalize extensive experiments to appraise the image restoration performance at different levels of tube current and spatial and temporal resolution scales.The results demonstrate the capability of our STIR-Net to restore high-quality scans at as low as 11% of absorbed radiation dose of the current imaging protocol, yielding an average of 10% improvement for perfusion maps compared to the patch-based log likelihood method.
                                                    </div>
                                                    -->
                                                </div>

                                                <div class="item mix Journal" data-year="2019">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse"><i class="fa fa-expand"></i></a>
                                                            <a href="https://www.sciencedirect.com/science/article/pii/S1361841518307734?dgcid=author" class="tooltips" title="External link" target="_blank"><i class="fa fa-external-link"></i></a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/MIA.gif" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Deep Evolutionary Networks with Expedited Genetic Algorithms for Medical Image Denoising </h4>
                                                            <span class="label label-warning">Journal</span>
                                                            <div class="pubauthor">Peng Liu, Mohammad D El Basha, Yangjunyi Li, Yao Xiao, Pina C.Sanelli, <u><b>Ruogu Fang</b></u></div>
                                                            <div class="pubcite">Medical Image Analysis, vol. 54, pp. 306-315</div>
                                                            <div class="pubdate">Publication Year: 2019</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Deep convolutional neural networks offer state-of-the-art performance for medical image analysis. However, their architectures are manually designed for particular problems. On the one hand, a manual designing process requires many trials to tune a large number of hyperparameters and is thus quite a time-consuming task. On the other hand, the fittest hyperparameters that can adapt to source data properties (e.g., sparsity, noisy features) are not able to be quickly identified for target data properties. For instance, the realistic noise in medical images is usually mixed and complicated, and sometimes unknown, leading to challenges in applying existing methods directly and creating effective denoising neural networks easily. In this paper, we present a Genetic Algorithm (GA)-based network evolution approach to search for the fittest genes to optimize network structures automatically. We expedite the evolutionary process through an experience-based greedy exploration strategy and transfer learning. Our evolutionary algorithm procedure has flexibility, which allows taking advantage of current state-of-the-art modules (e.g., residual blocks) to search for promising neural networks. We evaluate our framework on a classic medical image analysis task: denoising. The experimental results on computed tomography perfusion (CTP) image denoising demonstrate the capability of the method to select the fittest genes for building high-performance networks, named EvoNets. Our results outperform state-of-the-art methods consistently at various noise levels.
                                                    </div>
                                                </div>

                                                <div class="item mix Journal" data-year="2019">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse"><i class="fa fa-expand"></i></a>
                                                            <a href="https://www.frontiersin.org/articles/10.3389/fneur.2019.00647/full" class="tooltips" title="External link" target="_blank"><i class="fa fa-external-link"></i></a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                        <img width="125" src="img/pubs/Frontiers.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">STIR-Net: Spatial-Temporal Image Restoration Net for CT Perfusion Radiation Reduction</h4>
                                                            <span class="label label-warning">Journal</span>
                                                            <div class="pubauthor">Yao Xiao, Peng Liu, Yun Liang, Skylar Stolte, Pina Sanelli, Ajay Gupta, Jana Ivanidze, <u><b>Ruogu Fang</b></u></div>
                                                            <div class="pubcite">Frontiers in Nuerology</div>
                                                            <div class="pubdate">Publication Year: 2019</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Computed Tomography Perfusion (CTP) imaging is a cost-effective and fast approach to provide diagnostic images for acute stroke treatment. Its cine scanning mode allows the visualization of anatomic brain structures and blood flow; however, it requires contrast agent injection and continuous CT scanning over an extended time. In fact, the accumulative radiation dose to patients will increase health risks such as skin irritation, hair loss, cataract formation, and even cancer. Solutions for reducing radiation exposure include reducing the tube current and/or shortening the X-ray radiation exposure time. However, images scanned at lower tube currents are usually accompanied by higher levels of noise and artifacts. On the other hand, shorter X-ray radiation exposure time with longer scanning intervals will lead to image information that is insufficient to capture the blood flow dynamics between frames. Thus, it is critical for us to seek a solution that can preserve the image quality when the tube current and the temporal frequency are both low. We propose STIR-Net in this paper, an end-to-end spatial-temporal convolutional neural network structure, which exploits multi-directional automatic feature extraction and image reconstruction schema to recover high-quality CT slices effectively. With the inputs of low-dose and low-resolution patches at different cross-sections of the spatio-temporal data, STIR-Net blends the features from both spatial and temporal domains to reconstruct high-quality CT volumes. In this study, we finalize extensive experiments to appraise the image restoration performance at different levels of tube current and spatial and temporal resolution scales.The results demonstrate the capability of our STIR-Net to restore high-quality scans at as low as 11% of absorbed radiation dose of the current imaging protocol, yielding an average of 10% improvement for perfusion maps compared to the patch-based log likelihood method.
                                                    </div>
                                                </div>

                                                <div class="item mix Journal" data-year="2019">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse"><i class="fa fa-expand"></i></a>
                                                            <a href="https://www.nature.com/articles/s41598-019-39795-x" class="tooltips" title="External link" target="_blank"><i class="fa fa-external-link"></i></a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/NATURE.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Automatic Choroid Layer Segmentation from Optical Coherence Tomography Images Using Deep Learning </h4>
                                                            <span class="label label-warning">Journal</span>
                                                            <div class="pubauthor">Saleha Masood, <strong>Ruogu Fang</strong>, Huating Li, Bin Sheng, Ping Li, Akash Mathavan, Xiangning Wang, Po Yang, Qiang Wu, Jing Qin, and Weiping Jia</div>
                                                            <div class="pubcite">Nature Scientiﬁc Reports, 9(1):3058</div>
                                                            <div class="pubdate">Publication Year: 2019</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>The choroid layer is a vascular layer in human retina and its main function is to provide oxygen and support to the retina. Various studies have shown that the thickness of the choroid layer is correlated with the diagnosis of several ophthalmic diseases. For example, diabetic macular edema (DME) is a leading cause of vision loss in patients with diabetes. Despite contemporary advances, automatic segmentation of the choroid layer remains a challenging task due to low contrast, inhomogeneous intensity, inconsistent texture and ambiguous boundaries between the choroid and sclera in Optical Coherence Tomography (OCT) images. The majority of currently implemented methods manually or semi-automatically segment out the region of interest. While many fully automatic methods exist in the context of choroid layer segmentation, more effective and accurate automatic methods are required in order to employ these methods in the clinical sector. This paper proposed and implemented an automatic method for choroid layer segmentation in OCT images using deep learning and a series of morphological operations. The aim of this research was to segment out Bruch’s Membrane (BM) and choroid layer to calculate the thickness map. BM was segmented using a series of morphological operations, whereas the choroid layer was segmented using a deep learning approach as more image statistics were required to segment accurately. Several evaluation metrics were used to test and compare the proposed method against other existing methodologies. Experimental results showed that the proposed method greatly reduced the error rate when compared with the other state-of-the-art methods.
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2019">
                                                    <div class="pubmain">
                                                        <div class="pubassets">

                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/RSNA.png" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Multimodal CT Image Super-Resolution via Transfer Generative Adversarial Network</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor">Yao Xiao, Manual Arreola, Izabella Barrreto, W. Christopher Fox, Keith Peters, <u><b>Ruogu Fang</b></u> </div>
                                                            <div class="pubcite">Annual Meeting of Radiology Society of North American</div>
                                                            <div class="pubdate">Publication Year: 2019</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2019">
                                                    <div class="pubmain">
                                                        <div class="pubassets">

                                                        <a href="https://github.com/cswin/AWC" class="tooltips" title="Github" target="_blank">
                                                            <i class="fa fa-github"></i>
                                                        </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/MICCAI.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">CFEA: Collaborative Feature Ensembling Adaptation for Domain Adaptation in Unsupervised Optic Disc and Cup Segmentation</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor">Peng Liu, Bin Kong, Zhongyu Li, Shaoting Zhang, <u><b>Ruogu Fang</b></u></div>
                                                            <div class="pubcite">Medical Image Analysis and Computer Assisted Intervention</div>
                                                            <div class="pubdate">Publication Year: 2019</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2019">
                                                    <div class="pubmain">
                                                        <div class="pubassets">

                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/MICCAI.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Abdominal Adipose Tissue Segmentation in MRI with Double Loss Function Collaborative Learning</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor">Siyuan Pan, Yuxin Xue, Bin Sheng, Xuhong Hou, Huating Li, <strong>Ruogu Fang</strong>, Weiping Jia, and Jing Qin </div>
                                                            <div class="pubcite">Medical Image Analysis and Computer Assisted Intervention</div>
                                                            <div class="pubdate">Publication Year: 2019</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2019">
                                                    <div class="pubmain">
                                                        <div class="pubassets">

                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/BMES.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Multimodal CT Image Super-Resolution via Transfer-GAN.</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor">Yao Xiao, <u><b>Ruogu Fang</b></u></div>
                                                            <div class="pubcite">Biomedical Engineering Society Annual Meeting, Philadelphia, PA</div>
                                                            <div class="pubdate">Publication Year: 2019</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2019">
                                                    <div class="pubmain">
                                                        <div class="pubassets">

                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/BMES.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Deep Learning-based Alzheimers Disease Classification of FDG-PET and AV45 PET Images.</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor">Jianqiao Tian, Max Diaz, <u><b>Ruogu Fang</b></u></div>
                                                            <div class="pubcite">Biomedical Engineering Society Annual Meeting, Philadelphia, PA</div>
                                                            <div class="pubdate">Publication Year: 2019</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2019">
                                                    <div class="pubmain">
                                                        <div class="pubassets">

                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/BMES.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Artificial Intelligence For Automated Diagnosis of Glaucoma In Stereoscopic Images</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor">Skylar Stolte, <u><b>Ruogu Fang</b></u></div>
                                                            <div class="pubcite">Biomedical Engineering Society Annual Meeting, Philadelphia, PA</div>
                                                            <div class="pubdate">Publication Year: 2019</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2019">
                                                    <div class="pubmain">
                                                        <div class="pubassets">

                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/BMES.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Classification Of Neural Stimulations In The Brain With Super Voxels</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor">Kyle See, <u><b>Ruogu Fang</b></u></div>
                                                            <div class="pubcite">Biomedical Engineering Society Annual Meeting, Philadelphia, PA</div>
                                                            <div class="pubdate">Publication Year: 2019</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2019">
                                                    <div class="pubmain">
                                                        <div class="pubassets">

                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/BMES.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Retinal Disease Diagnosis Using Mobile Devices</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor">Skylar Stolte, Kyle See, Daniel El Basha, <u><b>Ruogu Fang</b></u></div>
                                                            <div class="pubcite">Biomedical Engineering Society Annual Meeting, Philadelphia, PA</div>
                                                            <div class="pubdate">Publication Year: 2019</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                </div>

                                                <!-- 2018-->
                                                <div class="item mix Journal" data-year="2018">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                            <a href="https://www.hindawi.com/journals/cin/2018/8234734/" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/CIN.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Mining Big Neuron Morphological Data</h4>
                                                            <span class="label label-warning">Journal</span>
                                                            <div class="pubauthor">Maryamossadat Aghili, <u><b>Ruogu Fang</b></u></div>
                                                            <div class="pubcite">Computational Intelligence and Neuroscience</div>
                                                            <div class="pubdate">Publication Year: 2018</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>The advent of automatic tracing and reconstruction technology has led to a surge in the number of neurons 3D reconstruction data and consequently the neuromorphology research. However, the lack of machine-driven annotation schema to automatically detect the types of the neurons based on their morphology still hinders the development of this branch of science. Neuromorphology is important because of the interplay between the shape and functionality of neurons and the far-reaching impact on the diagnostics and therapeutics in neurological disorders. This survey paper provides a comprehensive research in the field of automatic neurons classification and presents the existing challenges, methods, tools, and future directions for automatic neuromorphology analytics. We summarize the major automatic techniques applicable in the field and propose a systematic data processing pipeline for automatic neuron classification, covering data capturing, preprocessing, analyzing, classification, and retrieval. Various techniques and algorithms in machine learning are illustrated and compared to the same dataset to facilitate ongoing research in the field.
                                                    </div>
                                                </div>

                                                <div class="item mix Journal" data-year="2018">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                            <a href="https://ieeexplore.ieee.org/document/8362709" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/IEEECyber.png" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Retinal Vessel Segmentation Using Minimum Spanning Superpixel Tree Detector</h4>
                                                            <span class="label label-warning">Journal</span>
                                                            <div class="pubauthor">Bin Sheng, Ping Li, Shuangjia Mo, Huating Li, Xuhong Hou, Qiang Wu, Jing Qin, <strong>Ruogu Fang</strong>, and David Dagan Feng</div>
                                                            <div class="pubcite">IEEE Transaction on Cybernetics, 99, pp.1-13</div>
                                                            <div class="pubdate">Publication Year: 2018</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>The retinal vessel is one of the determining factors in an ophthalmic examination. Automatic extraction of retinal vessels from low-quality retinal images still remains a challenging problem. In this paper, we propose a robust and effective approach that qualitatively improves the detection of low-contrast and narrow vessels. Rather than using the pixel grid, we use a superpixel as the elementary unit of our vessel segmentation scheme. We regularize this scheme by combining the geometrical structure, texture, color, and space information in the superpixel graph. And the segmentation results are then refined by employing the efficient minimum spanning superpixel tree to detect and capture both global and local structure of the retinal images. Such an effective and structure-aware tree detector significantly improves the detection around the pathologic area. Experimental results have shown that the proposed technique achieves advantageous connectivity-area-length (CAL) scores of 80.92% and 69.06% on two public datasets, namely, DRIVE and STARE, thereby outperforming state-of-the-art segmentation methods. In addition, the tests on the challenging retinal image database have further demonstrated the effectiveness of our method. Our approach achieves satisfactory segmentation performance in comparison with state-of-the-art methods. Our technique provides an automated method for effectively extracting the vessel from fundus images
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2018">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                            <a href="https://link.springer.com/chapter/10.1007/978-3-030-00928-1_2" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/MICCAI.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle"> Neural Network Evolution Using Expedited Genetic Algorithm for Medical Image Denoising</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor">Peng Liu, Yangjunyi Li, Mohammad D El Basha, <u><b>Ruogu Fang</b></u></div>
                                                            <div class="pubcite">Medical Image Analysis and Computer Assisted Intervention, Granada, Spain.</div>
                                                            <div class="pubdate">Publication Year: 2018</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Convolutional neural networks offer state-of-the-art performance for medical image denoising. However, their architectures are manually designed for different noise types. The realistic noise in medical images is usually mixed and complicated, and sometimes unknown, leading to challenges in creating effective denoising neural networks. In this paper, we present a Genetic Algorithm (GA)-based network evolution approach to search for the fittest genes to optimize network structures. We expedite the evolutionary process through an experience-based greedy exploration strategy and transfer learning. The experimental results on computed tomography perfusion (CTP) images denoising demonstrate the capability of the method to select the fittest genes for building high-performance networks, named EvoNets, and our results compare favorably with state-of-the-art methods
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2018">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                            <a href="https://link.springer.com/chapter/10.1007/978-3-030-00919-9_4" class="tooltips" title="External Link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/MLMI.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">  Multi-task Fundus Image Quality Assessment via Transfer Learning and Landmarks Detection</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor">Yaxin Shen, <strong>Ruogu Fang</strong>, Bin Sheng, Ling Dai, Huating Li, Jing Qin, Qiang Wu, and Weiping Jia </div>
                                                            <div class="pubcite">Machine Learning in Medical Imaging, Granada, Spain.</div>
                                                            <div class="pubdate">Publication Year: 2018</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>The quality of fundus images is critical for diabetic retinopathy diagnosis. The evaluation of fundus image quality can be affected by several factors, including image artifact, clarity, and field definition. In this paper, we propose a multi-task deep learning framework for automated assessment of fundus image quality. The network can classify whether an image is gradable, together with interpretable information about quality factors. The proposed method uses images in both rectangular and polar coordinates, and fine-tunes the network from trained model grading of diabetic retinopathy. The detection of optic disk and fovea assists learning the field definition task through coarse-to-fine feature encoding. The experimental results demonstrate that our framework outperform single-task convolutional neural networks and reject ungradable images in automated diabetic retinopathy diagnostic systems.
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2018">
                                                    <div class="pubmain">
                                                        <div class="pubassets">

                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/BMES.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Low-Dose CT Perfusion Image Restoration and Radiation Reduction.</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor">Yao Xiao, Peng Liu, <u><b>Ruogu Fang</b></u></div>
                                                            <div class="pubcite">Biomedical Engineering Society Annual Meeting, Atlanta, GA</div>
                                                            <div class="pubdate">Publication Year: 2018</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2018">
                                                    <div class="pubmain">
                                                        <div class="pubassets">

                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/BMES.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Eﬃcient Multi Modality Medical Image Joint Recosntruction via Vectorized Gradient</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor">Yao Xiao, Yun Liang, Yunmei Chen, Xiaojing Ye, <u><b>Ruogu Fang</b></u> </div>
                                                            <div class="pubcite">Biomedical Engineering Society Annual Meeting, Atlanta, GA</div>
                                                            <div class="pubdate">Publication Year: 2018</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2018">
                                                    <div class="pubmain">
                                                        <div class="pubassets">

                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/BMES.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Multi-Modality Brain Image Co-Registration</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor">Skylar Stolte, Yao Xiao, <u><b>Ruogu Fang</b></u></div>
                                                            <div class="pubcite">Biomedical Engineering Society Annual Meeting, Atlanta, GA</div>
                                                            <div class="pubdate">Publication Year: 2018</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2018">
                                                    <div class="pubmain">
                                                        <div class="pubassets">

                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/BMES.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Decision Tree-based Classiﬁcation for Diﬀerentiating System Lupus Erythematosus and Mixed Connective Tissue Disease</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor">Kyle B. See, <u><b>Ruogu Fang</b></u></div>
                                                            <div class="pubcite">Biomedical Engineering Society Annual Meeting, Atlanta, GA</div>
                                                            <div class="pubdate">Publication Year: 2018</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2018">
                                                    <div class="pubmain">
                                                        <div class="pubassets">

                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/TAPIA.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">STDN: Spatial-Temporal Denoising Net for Radiation Optimization in CT Perfusion</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor">Yao Xiao, Liupeng, Yun Liang, <u><b>Ruogu Fang</b></u></div>
                                                            <div class="pubcite">ACM Richard Tapia Celebration of Diversity in Computing, September, Orlando, FL</div>
                                                            <div class="pubdate">Publication Year: 2018</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2018">
                                                    <div class="pubmain">
                                                        <div class="pubassets">

                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/ASNR.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Multi-Modality PET-MRI Image Joint Reconstruction</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor">Yao Xiao, Yun Liang, Yunmei Chen, Xiaojing Ye, <u><b>Ruogu Fang</b></u></div>
                                                            <div class="pubcite">ASNR 56th Annual Meeting & The Foundation of the ASNR Symposium, Vancouver, Canada</div>
                                                            <div class="pubdate">Publication Year: 2018</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2018">
                                                    <div class="pubmain">
                                                        <div class="pubassets">

                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/ASNR.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Image Super-Resolution and Radiation Reduction via Deep Learning</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor">Yao Xiao, Pina C. Sanelli, <u><b>Ruogu Fang</b></u></div>
                                                            <div class="pubcite">ASNR 56th Annual Meeting & The Foundation of the ASNR Symposium, Vancouver, Canada</div>
                                                            <div class="pubdate">Publication Year: 2018</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2018">
                                                    <div class="pubmain">
                                                        <div class="pubassets">

                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/ASNR.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Regulated-convolutional Networks for Low-dose Cerebral CT Perfusion Restoration</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor"> Peng Liu, <u><b>Ruogu Fang</b></u></div>
                                                            <div class="pubcite">ASNR 56th Annual Meeting & The Foundation of the ASNR Symposium, Vancouver, Canada</div>
                                                            <div class="pubdate">Publication Year: 2018</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2018">
                                                    <div class="pubmain">
                                                        <div class="pubassets">

                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/IEEEImaging.png" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle"> SDCNET: Smoothed Dense-Convolution Network For Restoring Low-Dose Cerebral CT Perfusion</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor"> Peng Liu, <u><b>Ruogu Fang</b></u></div>
                                                            <div class="pubcite">IEEE International Symposium on Biomedical Imaging, Washington D.C.</div>
                                                            <div class="pubdate">Publication Year: 2018</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                </div>

                                                <div class="item mix Patents" data-year="2018">
                                                    <div class="pubmain">
                                                        <div class="pubassets">

                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/Patent.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Neural Network Evolution Using Expedited Genetic Algorithm for Medical Image Denoising</h4>
                                                            <span class="label label-info">Patents</span>
                                                            <div class="pubauthor">Peng Liu, <u><b>Ruogu Fang</b></u></div>
                                                            <div class="pubcite">Ref. No: UF-17344, Filed on 9/10/2018, Provisional Patent. US 62/728,995</div>
                                                            <div class="pubdate">Publication Year: 2018</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                </div>




                                                <!--2017-->
                                                <div class="item mix Journal" data-year="2017">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                            <a href="http://digital-library.theiet.org/content/journals/10.1049/iet-ipr.2017.0273" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/IET.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Automatic Choroid Layer Segmentation Using Normalized Graph Cut </h4>
                                                            <span class="label label-warning">Journal</span>
                                                            <div class="pubauthor">Saleha Masood, Bin Sheng, Ping Li, Ruimin Shen, <strong>Ruogu Fang</strong>, Qiang Wu</div>
                                                            <div class="pubcite"> IET Image Processing. pp. 22, DOI:<a href="http://digital-library.theiet.org/content/journals/10.1049/iet-ipr.2017.0273">10.1049/iet-ipr.2017.0273</a>, Online ISSN 1751-9667, 2017.</div>
                                                            <div class="pubdate">Publication Year: 2017</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Optical coherence tomography (OCT) is an immersive technique for depth analysis of retinal layers. Automatic choroid layer segmentation is a challenging task because of the low contrast inputs. Existing methodologies carried choroid layer segmentation manually or semi-automatically. In this paper, we proposed automated choroid layer segmentation based on normalized cut algorithm, which aims at extracting the global impression of images and treats the segmentation as a graph partitioning problem. Due to the complexity of the layering structure of retinal layers and choroid layer, we employed a series of preprocessing to make the cut more deterministic and accurate. The proposed method divided the image into several patches and ran the normalized cut on every image patch separately. The aim was to avoid insignificant vertical cuts and focus on horizontal cutting. After processing every patch, we acquired a global cut on the original image by combining all the patches. Later we measured the choroidal thickness which is highly helpful in the diagnosis of several retinal diseases. The results were computed on a total of 525 images of 21 real patients. Experimental results showed that the mean relative error rate of the proposed method was around 0.4 as the compared the manual segmentation performed by the experts.</p>
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2017">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                            <a href="https://link.springer.com/chapter/10.1007/978-3-319-67389-9_12" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/MLMI.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">STAR: Spatio-Temporal Architecture for super-Resolution in Low-Dose CT Perfusion.</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor">Yao Xiao, Ajay Gupta, Pina C. Sanelli, <u><b>Ruogu Fang</b></u></div>
                                                            <div class="pubcite">Machine Learning in Medical Imaging (MIML), Medical Image Computing and Computer Assisted Intervention (MICCAI), Lecture Notes in Computer Science book series (LNCS, volume 10541), pp 97-105, Quebec, Canada, Sep. 2017.</div>
                                                            <div class="pubdate">Publication Year: 2017</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Computed tomography perfusion (CTP) is one of the most widely used imaging modality for cerebrovascular disease diagnosis and treatment, especially in emergency situations. While cerebral CTP is ca- pable of quantifying the blood flow dynamics by continuous scanning at a focused region of the brain, the associated excessive radiation increases the patients' risk levels of developing cancer. To reduce the necessary radiation dose in CTP, decreasing the temporal sampling frequency is one promising direction. In this paper, we propose STAR, an end-to- end Spatio-Temporal Architecture for super-Resolution to significantly reduce the necessary scanning time and therefore radiation exposure. The inputs into STAR are multi-directional 2D low-resolution spatio- temporal patches at different cross sections over space and time. Via training multiple direction networks followed by a conjoint reconstruc- tion network, our approach can produce high-resolution spatio-temporal volumes. The experiment results demonstrate the capability of STAR to maintain the image quality and accuracy of cerebral hemodynamic parameters at only one-third of the original scanning time.</p>
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2017">
                                                    <div class="pubmain">
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/BMES.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Cardiovascular Disease Prediction and Risk Factor Mining with RFMiner. </h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor">Yao Xiao, <u><b>Ruogu Fang</b></u></div>
                                                            <div class="pubcite"> Biomedical Engineering Society Annual Meeting (BMES), 2017 Annual Meeting, Phoenix, Arizona.</div>
                                                            <div class="pubdate">Publication Year: 2017</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2017">
                                                    <div class="pubmain">
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/BMES.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Accelerated Brain Perfusion Imaging via Spatio-Temporal Super-Resolution.</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor">Yao Xiao, <u><b>Ruogu Fang</b></u></div>
                                                            <div class="pubcite"> Biomedical Engineering Society Annual Meeting (BMES), 2017 Annual Meeting, Phoenix, Arizona.</div>
                                                            <div class="pubdate">Publication Year: 2017</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2017">
                                                    <div class="pubmain">
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/BMES.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">A Simple and Realistic Simulation Method for Low-Dose CT.</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor">Peng Liu, <u><b>Ruogu Fang</b></u>></div>
                                                            <div class="pubcite"> Biomedical Engineering Society Annual Meeting (BMES), 2017 Annual Meeting, Phoenix, Arizona.</div>
                                                            <div class="pubdate">Publication Year: 2017</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2017">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                            <a href="http://ieeexplore.ieee.org/stamp/stamp.jsp?arnumber=8010656" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/IEEEHealth.png" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">RFMiner: Risk Factors Discovery and Mining for Preventive Cardiovascular Health.</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor">Yao Xiao, <u><b>Ruogu Fang</b></u></div>
                                                            <div class="pubcite">The Second IEEE/ACM Conference on Connected Health: Applications, Systems and Engineering Technologies (CHASE), Philadelphia, PA</div>
                                                            <div class="pubdate">Publication Year: 2017</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Cardiovascular disease is one of the leading causes of death in the United States. It is critical to identify the risk factors associated with cardiovascular diseases and to alert individuals before they experience a heart attack. In this paper we propose RFMiner, a risk factor discovery and mining framework for identifying significant risk factors using integrated measures. We provide the blueprints for accurately predicting the possibility of heart attacks in the near future while identifying notable risk factors - especially the factors which are not well recognized.</p>
                                                    </div>
                                                </div>

                                                <div class="item mix Journal" data-year="2017">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                            <!--<a href="http://users.cis.fiu.edu/~rfang/publications/NC16_TENDER.pdf" class="tooltips" title="Download" target="_blank">
                                                            <i class="fa fa-cloud-download"></i>
                                                            </a> -->
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/Pattern_Recognition.gif" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Indexing and Mining Large-Scale Neuron Databases using Maximum Inner Product Search.</h4>
                                                            <span class="label label-warning">Journal</span>
                                                            <div class="pubauthor">Zhongyu Li, <strong>Ruogu Fang</strong>, Fumin Shen, Amin Katouzian, Shaoting Zhang.</div>
                                                            <div class="pubcite"> Pattern Recognition, vol. 63, pp. 680-688</div>
                                                            <div class="pubdate">Publication Year: 2017</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Morphological retrieval is an effective approach to explore large-scale neuronal databases, as the morphology is correlated with neuronal types, regions, functions, etc. In this paper, we focus on the neuron identification and analysis via morphological retrieval. In our proposed framework, multiple features are extracted to represent 3D neuron data. Because each feature reflects different levels of similarity between neurons, we group features into different hierarchies to compute the similarity matrix. Then, compact binary codes are generated from hierarchical features for efficient similarity search. Since neuronal cells usually have tree-topology structure, it is hard to distinguish different types of neurons simply via traditional binary coding or hashing methods based on Euclidean distance metric and/or linear hyperplanes. Therefore, we employ an asymmetric binary coding strategy based on the maximum inner product search (MIPS), which not only makes it easier to learn the binary coding functions, but also preserves the non-linear characteristics of the neuron morphological data. We evaluate the proposed method on more than 17,000 neurons, by validating the retrieved neurons with associated cell types and brain regions. Experimental results show the superiority of our approach in neuron morphological retrieval compared with other state-of-the-art methods. Moreover, we demonstrate its potential use cases in the identification and analysis of neuron characteristics from large neuron databases.</p>
                                                    </div>
                                                </div>

                                                <div class="item mix Journal" data-year="2017">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                            <a href="http://www.sciencedirect.com/science/article/pii/S0925231216313480" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                            <!--
                                                            <a href="http://users.cis.fiu.edu/~rfang/publications/NC16_TENDER.pdf" class="tooltips" title="Download" target="_blank">
                                                                <i class="fa fa-cloud-download"></i>
                                                            </a>
                                                            -->
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/Neurocomputing.png" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Abdominal Adipose Tissues Extraction Using Multi-Scale Deep Neural Network.</h4>
                                                            <span class="label label-warning">Journal</span>
                                                            <div class="pubauthor">Fei Jiang, Huating Li, Xuhong Hou, Bin Sheng, Ruimin Shen, Xiao-Yang Liu, Weiping Jia, Ping Li, <u><b>Ruogu Fang</b></u></div>
                                                            <div class="pubcite">NeuroComputing, vol. 229, pp. 23-33</div>
                                                            <div class="pubdate">Publication Year: 2017</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Segmentation of abdominal adipose tissues (AAT) into subcutaneous adipose tissues (SAT) and visceral adipose tissues (VAT) is of crucial interest for managing the obesity. Previous methods with raw or hand-crafted features rarely work well on large-scale subject cohorts, because of the inhomogeneous image intensities, artifacts and the diverse distributions of VAT. In this paper, we propose a novel two-stage coarse-to-fine algorithm for AAT seg- mentation. In the first stage, we formulate the AAT segmentation task as a pixel-wise classification problem. First, three types of features, intensity, spatial and contextual fea- tures, are adopted. Second, a new type of deep neural network, named multi-scale deep neural network (MSDNN), is provided to extract high-level features. In the second stage, to improve the segmentation accuracy, we refine coarse segmentation results by determining the internal boundary of SAT based on coarse segmentation results and continuous of SAT internal boundary. Finally, we demonstrate the efficacy of our algorithm for both 2D and 3D cases on a wide population range. Compared with other algorithms, our method is not only more suitable for large-scale dataset, but also achieves better segmentation results. Fur- thermore, our system takes about 2 seconds to segment an abdominal image, which implies potential clinical applications.</p>
                                                    </div>
                                                </div>




                                                
                                                <!-- 2016-->
                                                <div class="item mix Conference" data-year="2016">
                                                    <div class="pubmain">
                                                        <div class="pubassets">

                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/WIML.png" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Towards High-Throughput Abnormal Brain Screening in MRI Images</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor"> Maryamossadat Aghili, <u><b>Ruogu Fang</b></u></div>
                                                            <div class="pubcite"> Women in Machine Learning Workshop, Neural Information Processing Systems (NIPS), Barcelona, Spain.</div>
                                                            <div class="pubdate">Publication Year: 2016</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                </div>

                                                <div class="item mix Journal" data-year="2016">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                            <a href="http://www.sciencedirect.com/science/article/pii/S0925231216313753" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                            <a href="http://users.cis.fiu.edu/~rfang/publications/NC16_TENDER.pdf" class="tooltips" title="Download" target="_blank">
                                                                <i class="fa fa-cloud-download"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/Neurocomputing.png" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">TENDER: TEnsor Non-local Deconvolution Enabled Radiation Reduction in CT Perfusion.</h4>
                                                            <span class="label label-warning">Journal</span>
                                                            <div class="pubauthor"><u><b>Ruogu Fang</b></u>, Ajay Gupta, Junzhou Huang, Pina Sanelli.</div>
                                                            <div class="pubcite"> NeuroComputing, vol. 229, pp. 13-22</div>
                                                            <div class="pubdate">Publication Year: 2016</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Stroke is the leading cause of long-term disability and the second leading cause of mortality in the world, and exerts an enormous burden on the public health. CT remains one of the most widely used imaging modality for stroke diagnosis. However when coupled with CT perfusion, the excessive radiation exposure in repetitive imaging to assess treatment response and prognosis has raised significant public concerns regarding its potential hazards to both short- and longterm health outcomes. Tensor total variation has been proposed to reduce the necessary radiation dose in CT perfusion without comprising the image quality by fusing the information of the local anatomical structure with the temporal blood flow model. However the local search in the framework fails to leverage the non-local information in the spatio-temporal data. In this paper, we propose TENDER, an efficient framework of non-local tensor deconvolution to maintain the accuracy of the hemodynamic parameters and the diagnostic reliability in low radiation dose CT perfusion. The tensor total variation is extended using non-local spatio-temporal cubics for regularization to integrate contextual and non-local information. We also propose an efficient framework consisting of fast nearest neighbor search, accelerated optimization and parallel computing to improve the efficiency and scalability of the non-local spatio-temporal algorithm. Evaluations on clinical data of subjects with cerebrovascular disease and normal subjects demonstrate the advantage of non-local tensor deconvolution for reducing radiation dose in CT perfusion.</p>
                                                    </div>
                                                </div>

                                                <div class="item mix Journal" data-year="2016">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                            <a href="http://dl.acm.org/citation.cfm?id=2932707" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                            <a href="http://users.cis.fiu.edu/~rfang/publications/ACM16_Survey.pdf" class="tooltips" title="Download" target="_blank">
                                                                <i class="fa fa-cloud-download"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/ACM.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Computational Health Informatics in the Big Data Age: A Survey</h4>
                                                            <span class="label label-warning">Journal</span>
                                                            <div class="pubauthor"><u><b>Ruogu Fang*</b></u>, Samira Pouyanfar*, Yimin Yang, Shu-Ching Chen, S. S. Iyengar (* indicates equal contributions)</div>
                                                            <div class="pubcite"> Journal CSUR, ACM Computing Survey, Volume 49 Issue 1, Article No. 12</div>
                                                            <div class="pubdate">Publication Year: 2016</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>The explosive growth and widespread accessibility of digital health data have led to a surge of research activity in the healthcare and data sciences fields. The conventional approaches for health data management have achieved limited success as they are incapable of handling the huge amount of complex data with high volume, high velocity, and high variety. This article presents a comprehensive overview of the existing challenges, techniques, and future directions for computational health informatics in the big data age, with a structured analysis of the historical and state-of-the-art methods. We have summarized the challenges into four Vs (i.e., volume, velocity, variety, and veracity) and proposed a systematic data-processing pipeline for generic big data in health informatics, covering data capturing, storing, sharing, analyzing, searching, and decision support. Specifically, numerous techniques and algorithms in machine learning are categorized and compared. On the basis of this material, we identify and discuss the essential prospects lying ahead for computational health informatics in this big data age.</p>
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2016">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/WoundHealing.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Hemodynamic Imaging of Lower Extremity Ulcers.</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor">Rebecca Kwasinski, Cristianne Fernandez, Kevin Leiva, Edwin Robledo, Yuanyuan Zhu, Penelope Kallis, Francesco-Perez Clavijo, <strong>Ruogu Fang</strong>, Robert Kirsner, Anuradha Godavarty.</div>
                                                            <div class="pubcite"> Innovations in Wound Healing</div>
                                                            <div class="pubdate">Publication Year: 2016</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Clinicians employ visual inspection of the wound and reduction in its size over time to monitor its healing process. Although these are standard clinical assessments, there is a need to develop a physiological approach that can map sub-surface tissue oxygenation at and around the wound region.  Recently, a non-contact, portable, hand-held near infrared optical scanner (NIROS) has been developed to functionally image wound sites and differentiate healing from non-healing in lower extremity ulcers. Past studies using NIROS focused on differentiating healing from non-healing wounds based on NIR optical contrast between the wound and healthy surrounding tissue. However, these studies did not showcase the physiological changes in tissue oxygenation. Herein, NIROS has been modified to perform multi wavelength imaging in order to obtain the oxy and deoxy- hemoglobin maps of the wound and its surroundings. Clinical studies are currently performed at two clinical sites in Miami on lower extremity ulcers (2, diabetic foot ulcers (DFUs) and 4 venous leg ulcers (VLUs to date). Preliminary results have shown changes in oxy- and deoxy-hemoglobin maps of the wound and background across weeks of the treatment process. Image segmentation studies quantified regions of varied tissue oxygenation around and beneath the wound to potentially determine sub-surface healing regions. Ongoing efforts involve systematic 8-week imaging studies to obtain physiological indicators of healing from hemodynamic studies of DFUs and VLUs.</p>
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2016">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                        <!--
                                                        <a href="http://users.cis.fiu.edu/~rfang/publications/BMES16_SR.pdf" class="tooltips" title="Download" target="_blank">
                                                            <i class="fa fa-cloud-download"></i>
                                                        </a>
                                                        -->
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/BMES.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">CT perfusion image super-resolution using a deep convolutional network.</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor">Paul Naghshineh, Peng Liu, <u><b>Ruogu Fang</b></u></div>
                                                            <div class="pubcite">BMES, Biomedical Engineering Society Annual Meeting in Minneapolis, Minnesota.</div>
                                                            <div class="pubdate">Publication Year: 2016</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2016">
                                                    <div class="pubmain">
                                                        <div class="pubassets">

                                                    <!--    <a href="http://users.cis.fiu.edu/~rfang/publications/BMES16_physiological.pdf" class="tooltips" title="Download" target="_blank">
                                                            <i class="fa fa-cloud-download"></i>
                                                        </a>-->
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/BMES.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Physiological Assessment of Wound Healing using a Near-Infrared Optical Scanner.</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor">Anuradha Godavarty, Rebecca Kwasinki, Cristianne Fernandez, Yuanyuan Zhu, Edwin Robledo, F. Perez-Clavijo, <u><b>Ruogu Fang</b></u></div>
                                                            <div class="pubcite"> Biomedical Engineering Society Annual Meeting (BMES) in Minneapolis, Minnesota.</div>
                                                            <div class="pubdate">Publication Year: 2016</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2016">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                            <a href="http://ieeexplore.ieee.org/xpl/login.jsp?tp=&arnumber=7493340&url=http%3A%2F%2Fieeexplore.ieee.org%2Fxpls%2Fabs_all.jsp%3Farnumber%3D7493340" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/IEEEImaging.png" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle"> Maximum Inner Product Search for Morphological Retrieval of Large-Scale Neuron Data</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor">Zhongyu Li, Fumin Shen, <strong>Ruogu Fang</strong>, Sailesh Conjeti, Amin Katouzian, Shaoting Zhang.</div>
                                                            <div class="pubcite">ISBI, IEEE International Symposium on Biomedical Imaging, Prague, Czech Republic</div>
                                                            <div class="pubdate">Publication Year: 2016</div>
                                                        </div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Morphological retrieval is an effective approach to explore neurons' databases, as the morphology is correlated with neuronal types, regions, functions, etc. In this paper, we focus on the neuron identification and analysis via morphological retrieval. In our proposed framework, both global and local features are extracted to represent 3D neuron data. Then, compacted binary codes are generated from original features for efficient similarity search. As neuron cells usually have tree-topology structure, it is hard to distinguish different types of neuron simply via traditional binary coding or hashing methods based on Euclidean distance metric and/or linear hyperplanes. Thus, we propose a novel binary coding method based on the maximum inner product search (MIPS), which is not only more easier to learn the binary coding function, but also preserves the non-linear characteristics of neuron morphology data. We evaluate the proposed method on more than 17,000 neurons, by validating the retrieved neurons with associated cell types and brain regions. Experimental results show the superiority of our approach in neuron morphological retrieval compared with other state-of-the-art methods. Moreover, we demonstrate its potential use case in the identification and analysis of neuron characteristics.</p>
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2016">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                            <a href="http://ieeexplore.ieee.org/xpl/articleDetails.jsp?tp=&arnumber=7493372&url=http%3A%2F%2Fieeexplore.ieee.org%2Fiel7%2F7486633%2F7493185%2F07493372.pdf%3Farnumber%3D7493372" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/IEEEImaging.png" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle"> Direct Estimation of Permeability Maps for Low-Dose CT Perfusion </h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor"><u><b>Ruogu Fang</b></u>, Ajay Gupta, Pina C. Sanelli</div>
                                                            <div class="pubcite"> ISBI, IEEE International Symposium on Biomedical Imaging, Prague, Czech Republic</div>
                                                            <div class="pubdate">Publication Year: 2016</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>With the goal of achieving low radiation exposure from
                                                        medical imaging, computed tomography perfusion (CTP)
                                                        introduces challenging problems for both image reconstruction
                                                        and perfusion parameter estimation in the qualitative
                                                        and quantitative analyses. Conventional approaches address
                                                        the reconstruction and the estimation processes separately.
                                                        Since the hemodynamic parameter maps have much lower
                                                        dimensionality than the original sinogram data, estimating
                                                        hemodynamic parameters directly from sinogram will further
                                                        reduce radiation exposure and save computational resources
                                                        to reconstruct the intermediate time-series images. In this
                                                        work, we propose the first direct estimation framework for
                                                        CTP that integrates the time-series image reconstruction,
                                                        contrast conversion, hematocrit correction and hemodynamic
                                                        parameter estimation in one optimization function, which is
                                                        solved using an efficient algorithm. Evaluations on the digital
                                                        brain perfusion phantom and a clinical acute stroke subject
                                                        demonstrate that the proposed direct estimation framework
                                                        boosts the estimation accuracy remarkably in CTP scanning
                                                        with lower radiation exposure.</p>
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2016">
                                                    <div class="pubmain">
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/IEEEImaging.png" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Automatic Segmentation of Lower Extremity Ulcers in Near-Infrared Optical Imaging</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor"><strong>Ruogu Fang</strong>, Xing Pang, Arash Dadkhah, Jiali Lei, Elizabeth Solis, Suset Rodriguesz, Francisco Perez-Clavijo, Stephen Wigley, Charles Buscemi, Anuradha Godvarty.</div>
                                                            <div class="pubcite">ISBI, IEEE International Symposium on Biomedical Imaging, Prague, Czech Republic</div>
                                                            <div class="pubdate">Publication Year: 2016</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2016">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                            <a href="http://8.18.37.105/abstract.cfm?uri=OTS-2016-JTu3A.43" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/OSA.png" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Near-Infrared Optical Imaging and Wound Segmentation in Lower Extremity Ulcers</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor">Xing Pang, Arash Dadkhah, Jaili Lei, Elizabeth Solis, Suset Rodriguez, Francisco Perez-Clavijo, Stephen Wigley, <strong>Ruogu Fang</strong>, Anuradha Godvarty.</div>
                                                            <div class="pubcite">OSA, Optical Society of America Annual Meeting</div>
                                                            <div class="pubdate">Publication Year: 2016</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Near-Infrared (NIR) optical imaging can reveal tissue oxygenation of the wound, complementing the visual inspection of the surface granulation. Herein, graph cuts algorithm is applied to segment NIR images of the wound from its peripheries.</p>
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2016">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                            <a href="http://proceedings.spiedigitallibrary.org/proceeding.aspx?articleid=2501771" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/SPIE.png" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Wound Size Measurement of Lower Extremity Ulcers Using Segmentation Algorithms</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor">Arash Dadkhah, Xing Pang, Elizabeth Solis, <strong>Ruogu Fang</strong>, Anuradha Godvarty.</div>
                                                            <div class="pubcite"> SPIE Proceedings in Photonics West, San Francisco</div>
                                                            <div class="pubdate">Publication Year: 2016</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Lower extremity ulcers are one of the most common complications that not only affect many people around the world but also have huge impact on economy since a large amount of resources are spent for treatment and prevention of the diseases. Clinical studies have shown that reduction in the wound size of 40% within 4 weeks is an acceptable progress in the healing process. Quantification of the wound size plays a crucial role in assessing the extent of healing and determining the treatment process. To date, wound healing is visually inspected and the wound size is measured from surface images. The extent of wound healing internally may vary from the surface. A near-infrared (NIR) optical imaging approach has been developed for non-contact imaging of wounds internally and differentiating healing from non-healing wounds. Herein, quantitative wound size measurements from NIR and white light images are estimated using a graph cuts and region growing image segmentation algorithms. The extent of the wound healing from NIR imaging of lower extremity ulcers in diabetic subjects are quantified and compared across NIR and white light images. NIR imaging and wound size measurements can play a significant role in potentially predicting the extent of internal healing, thus allowing better treatment plans when implemented for periodic imaging in future.</p>
                                                    </div>
                                                </div>

                                                <!-- 2015-->
                                                <div class="item mix Journal" data-year="2015">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                            <a href="http://users.cis.fiu.edu/~rfang/projects/ttv.html" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>

                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/MCV.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Efficient 4D Non-Local Tensor Total-Variation for Low-Dose CT Perfusion Deconvolution</h4>
                                                            <span class="label label-warning">Journal</span>
                                                            <div class="pubauthor"><u><b>Ruogu Fang</b></u>, Ming Ni, Junzhou Huang, Qianmu Li, Tao Li</div>
                                                            <div class="pubcite">Medical Computer Vision: Algorithms for Big DataChapter, no.16, Lecture Notes in Computer Science</div>
                                                            <div class="pubdate">Publication Year: 2015</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Tensor total variation deconvolution has been recently proposed as a robust framework to accurately estimate the hemodynamic parameters in low-dose CT perfusion by fusing the local anatomicalstructure correlation and temporal blood flow continuation. However
                                                            the locality property in the current framework constrains the search for
                                                            anatomical structure similarities to the local neighborhood, missing the
                                                            global and long-range correlations in the whole anatomical structure.
                                                            This limitation has led to noticeable absence or artifact of delicate structures,
                                                            including the critical indicators for the clinical diagnosis of cerebrovascular
                                                            diseases. In this paper, we propose an extension of the TTV
                                                            framework by introducing 4D non-local tensor total variation into the deconvolution
                                                            to bridge the gap between non-adjacent regions of the same
                                                            tissue classes. The non-local regularization using tensor total variation
                                                            term is imposed on the spatio-temporal flow-scaled residue functions.
                                                            An efficient algorithm and implementation of the non-local tensor total
                                                            variation (NL-TTV) reduces the time complexity with fast similarity
                                                            computation, accelerated optimization and parallel operations. Extensive
                                                            evaluations on the clinical data with cerebrovascular diseases and normal subjects demonstrate the importance of non-local linkage and long-range connections for low-dose CT perfusion deconvolution.</p>
                                                    </div>
                                                </div>

                                                <div class="item mix Journal" data-year="2015">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                            <a href="http://ieeexplore.ieee.org/xpl/articleDetails.jsp?arnumber=7046356" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/IEEEMedical.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Robust Low-dose CT Perfusion Deconvolution via Tensor Total-Variation Regularization</h4>
                                                            <span class="label label-warning">Journal</span>
                                                            <div class="pubauthor"><u><b>Ruogu Fang</b></u>, Shaoting Zhang, Tsuhan Chen, Pina C. Sanelli.</div>
                                                            <div class="pubcite">TMI, IEEE Transaction on Medical Imaging, vol.34, no.7, pp.1533-1548</div>
                                                            <div class="pubdate">Publication Year: 2015</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Acute brain diseases such as acute strokes and
                                                        transit ischemic attacks are the leading causes of mortality and
                                                        morbidity worldwide, responsible for 9% of total death every
                                                        year. ‘Time is brain' is a widely accepted concept in acute
                                                        cerebrovascular disease treatment. Efficient and accurate computational
                                                        framework for hemodynamic parameters estimation
                                                        can save critical time for thrombolytic therapy. Meanwhile the
                                                        high level of accumulated radiation dosage due to continuous
                                                        image acquisition in CT perfusion (CTP) raised concerns on
                                                        patient safety and public health. However, low-radiation leads to
                                                        increased noise and artifacts which require more sophisticated
                                                        and time-consuming algorithms for robust estimation. In this
                                                        paper, we focus on developing a robust and efficient framework
                                                        to accurately estimate the perfusion parameters at low radiation
                                                        dosage. Specifically, we present a tensor total-variation (TTV)
                                                        technique which fuses the spatial correlation of the vascular
                                                        structure and the temporal continuation of the blood signal flow.
                                                        An efficient algorithm is proposed to find the solution with fast
                                                        convergence and reduced computational complexity. Extensive
                                                        evaluations are carried out in terms of sensitivity to noise levels,
                                                        estimation accuracy, contrast preservation, and performed on
                                                        digital perfusion phantom estimation, as well as in-vivo clinical
                                                        subjects. Our framework reduces the necessary radiation dose
                                                        to only 8% of the original level and outperforms the state-of-art
                                                        algorithms with peak signal-to-noise ratio improved by 32%. It
                                                        reduces the oscillation in the residue functions, corrects overestimation
                                                        of cerebral blood flow (CBF) and under-estimation of
                                                        mean transit time (MTT), and maintains the distinction between
                                                        the deficit and normal regions.</p>
                                                    </div>
                                                </div>

                                                <div class="item mix Journal" data-year="2015">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                            <a href="http://www.sciencedirect.com/science/article/pii/S0895611115000981#" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/CMIG.gif" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Guest Editorial: Sparsity Techniques in Medical Imaging.</h4>
                                                            <span class="label label-warning">Journal</span>
                                                            <div class="pubauthor"><u><b>Ruogu Fang</b></u>, Tsuhan Chen, Dimitris Metaxas, Pina Sanelli, Shaoting Zhang.</div>
                                                            <div class="pubcite">CMIG, Elsevier Journal of Computerized Medical Imaging and Graphics, vol. 46, no. 1</div>
                                                            <div class="pubdate">Publication Year: 2015</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>With the advent of the age for big data and complex structure,
                                                        sparsity has been an important modeling tool in compressed
                                                        sensing, machine learning, image processing, neuroscience and
                                                        statistics. In the medical imaging field, sparsity methods have been
                                                        successfully used in image reconstruction, image enhancement,
                                                        image segmentation, anomaly detection, disease classification, and
                                                        image database retrieval. Developing more powerful sparsity models
                                                        for a large range of medical imaging and medical image analysis
                                                        problems as well as efficient optimization and learning algorithm
                                                        will keep being a main research topic in this field. The goal of this
                                                        special issue is to publish original and high quality papers on innovation
                                                        research and development in medical imaging and medical
                                                        image analysis using sparsity techniques. This special issue will
                                                        help advance the scientific research within the field of sparsity
                                                        methods for medical imaging.</p>
                                                    </div>
                                                </div>

                                                <div class="item mix Journal" data-year="2015">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                            <a href="http://www.sciencedirect.com/science/article/pii/S0895611115000890" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/IEEEMedical.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Tissue-Specific Sparse Deconvolution for Brain CT Perfusion. </h4>
                                                            <span class="label label-warning">Journal</span>
                                                            <div class="pubauthor"><u><b>Ruogu Fang</b></u>,Haodi Jiang, Junzhou Huang.</div>
                                                            <div class="pubcite"> CMIG, Elsevier Journal of Computerized Medical Imaging and Graphics, vol.46, no.1, pp. 64-72</div>
                                                            <div class="pubdate">Publication Year: 2015</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Enhancing perfusion maps in low-dose computed tomography perfusion (CTP)
                                                        for cerebrovascular disease diagnosis is a challenging task, especially for lowcontrast
                                                        tissue categories where infarct core and ischemic penumbra usually
                                                        occur. Sparse perfusion deconvolution has been recently proposed to effectively
                                                        improve the image quality and diagnostic accuracy of low-dose perfusion CT by
                                                        extracting the complementary information from the high-dose perfusion maps
                                                        to restore the low-dose using a joint spatio-temporal model. However the lowcontrast
                                                        tissue classes where infarct core and ischemic penumbra are likely to
                                                        occur in cerebral perfusion CT tend to be over-smoothed, leading to loss of
                                                        essential biomarkers. In this paper, we propose a tissue-specific sparse deconvolution
                                                        approach to preserve the subtle perfusion information in the low-contrast
                                                        tissue classes. We first build tissue-specific dictionaries from segmentations of
                                                        high-dose perfusion maps using online dictionary learning, and then perform
                                                        deconvolution-based hemodynamic parameters estimation for block-wise tissue
                                                        segments on the low-dose CTP data. Extensive validation on clinical datasets
                                                        of patients with cerebrovascular disease demonstrates the superior performance
                                                        of our proposed method compared to state-of-art, and potentially improve diagnostic accuracy by increasing the differentiation between normal and ischemic
                                                        tissues in the brain</p>
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2015">
                                                    <div class="pubmain">
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/WoundHealing.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Wound Segmentation in Near-Infrared Optical Imaging </h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor"><strong>Ruogu Fang</strong>, Xing Pang, Arash Dadkhah, Jiali Lei, Elizabeth SOlis, Suset ROdriguez, Francisco Perez-Calvijo, Stephen Wigley, Charles Buscemi, Anuradha Godvarty.</div>
                                                            <div class="pubcite"> Innovation in Wound Healing, Hawks Cay, FL</div>
                                                            <div class="pubdate">Publication Year: 2015</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2015">
                                                    <div class="pubmain">
                                                        <!--
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                            <a href="http://www.sciencedirect.com/science/article/pii/S1057740812000290" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                            <a href="http://faculty-gsb.stanford.edu/aaker/pages/documents/CultivatingAdmirationinBrands_JCP2012.pdf" class="tooltips" title="Download" target="_blank">
                                                                <i class="fa fa-cloud-download"></i>
                                                            </a>
                                                        </div>-->
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/MICCAI.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Fast Preconditioning for Accelerated Multi-Contrast MRI Reconstruction</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor">Ruoyu Li, Yeqing Li, <strong>Ruogu Fang</strong>,Shaoting Zhang, Hao Pan, Junzhou Huang</div>
                                                            <div class="pubcite"> MICCAI'15, In Proc. of the 18th Annual International Conference on Medical Image Computing and Computer Assisted Intervention, Munich, Germany</div>
                                                            <div class="pubdate">Publication Year: 2015</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <!--
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Although a substantial amount of research has examined the constructs of warmth and competence, far less has examined how these constructs develop and what benefits may accrue when warmth and competence are cultivated. Yet there are positive consequences, both emotional and behavioral, that are likely to occur when brands hold perceptions of both. In this paper, we shed light on when and how warmth and competence are jointly promoted in brands, and why these reputations matter.</p>
                                                    </div>-->
                                                </div>

                                                <div class="item mix Conference" data-year="2015">
                                                    <div class="pubmain">
                                                        <div class="pubassets">

                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/MICCAI.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle"> Efficient 4D Non-Local Tensor Total-Variation for Low-Dose CT Perfusion Deconvolution</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor"> <u><b>Ruogu Fang</b></u>, Ming Ni, Junzhou Huang, Qianmu Li, and Tao Li </div>
                                                            <div class="pubcite"> The 18th Annual International Conference on Medical Image Computing and Computer Assisted Intervention, Workshop on Medical Computer Vision, Munich, Germany</div>
                                                            <div class="pubdate">Publication Year: 2015</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2015">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/BMES.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Robust Low-Dose CT Perfusion Deconvolution via Non-Local Tensor Total Variation</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor"><u><b>Ruogu Fang</b></u>, Ming Ni, Junzhou Huang, Qianmu Li, Tao Li</div>
                                                            <div class="pubcite"> BMES, Biomedical Engineering Society Annual Meeting, Tampa, FL</div>
                                                            <div class="pubdate">Publication Year: 2015</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Introduction</h4>
                                                        <p>Stroke and cerebrovascular diseases are the leading cause of serious, long-term disability in the
                                                        United States. Computed tomography perfusion (CTP) is one of the most widely accepted imaging modality for
                                                        stroke care. However, the high radiation exposure of CTP has lead to increased cancer risk. Tensor total variation
                                                        (TTV)[1] has been proposed to stabilize the quantification of perfusion parameters by integrating the anatomical
                                                        structure correlation. Yet the locality limitation of the neighborhood region has led to noticeable absence or
                                                        inflation of the delicate structures which are critical indicators for the clinical diagnosis. In this work, we propose
                                                        a non-local tensor total variation (NL-TTV) deconvolution method to by incorporating the long-range dependency
                                                        and the global connections in the spatio-temporal domain</p>
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2015">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/MRM.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">4-D Spatio-Temporal MR Perfusion Deconvolution via Tensor Total Variation.</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor"><u><b>Ruogu Fang</b></u></div>
                                                            <div class="pubcite">ISMRM'15, International Society for Magnetic Resonance in Medicine (ISMRM) Annual Meeting, Toronto, Canada. Oral presentation</div>
                                                            <div class="pubdate">Publication Year: 2015</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Introduction</h4>
                                                        <p>4-D dynamic susceptibility contrast (DSC) magnetic resonance imaging (MRI) is a well-established perfusion technique for non-invasive characterization of tissue
                                                        dynamics, with promising applications in assessing a wide range of diseases, as well as monitoring response of therapeutic interventions). DSC-MRI provides critical
                                                        real-time information by tracking the first-pass of an injected contrast-agent (e.g. gadolinium) with T2*-weighted MRI. The spatio-temproal data, consisting of contrast
                                                        concentration signals for each voxel of a volume, are deconvolved from the arterial input function (AIF) and then post-processed to generate perfusion parameter maps,
                                                        typically including the cerebral blood flow (CBF), cerebral blood volume (CBV), mean transit time (MTT) and time to peak (TTP). The most popular deconvolution
                                                        method is truncated singular value decomposition (TSVD)1,2 and its variants3
                                                        , which fail to exploit the spatio-temporal nature of the 4D data with both the anatomical
                                                        structure and the temporal continuation. This work adapts and demonstrates the feasibility of a 4-D tensor total variation (TTV) deconvolution approach, which has
                                                        been proposed for CT perfusion4
                                                        , to brain MR perfusion, with evaluation on synthetic data and clinical DSC-MRI data for glioblastomas, the most common type of
                                                        brain cancer. The method is guaranteed to convergence to global optimal because of the convex cost function and presents a more elegant framework of total variation
                                                        for the deconvolution, compared to recent efforts5,6 which either do not have a global optimal solution for the non-convex case or need to handcraft spatial and temporal
                                                        regularization terms.</p>
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2015">
                                                    <div class="pubmain">
                                                    <!-- <div class="pubassets">
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                            <a href="http://www.sciencedirect.com/science/article/pii/S1057740812000290" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                            <a href="http://faculty-gsb.stanford.edu/aaker/pages/documents/CultivatingAdmirationinBrands_JCP2012.pdf" class="tooltips" title="Download" target="_blank">
                                                                <i class="fa fa-cloud-download"></i>
                                                            </a>
                                                        </div>-->
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/Neurocomputing.png" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Leveraging Coupled Multi-Index for Scalable Retrieval of Mammographic Masses. </h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor">Menglin Jiang, Shaoting Zhang, <strong>Ruogu Fang</strong>, Dimitris Metaxas.</div>
                                                            <div class="pubcite"> ISBI'15, The IEEE International Symposium on Biomedical Imaging (ISBI), NYC, USA. Oral presentation 130 / 714 = 18%</div>
                                                            <div class="pubdate">Publication Year: 2015</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <!--
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Although a substantial amount of research has examined the constructs of warmth and competence, far less has examined how these constructs develop and what benefits may accrue when warmth and competence are cultivated. Yet there are positive consequences, both emotional and behavioral, that are likely to occur when brands hold perceptions of both. In this paper, we shed light on when and how warmth and competence are jointly promoted in brands, and why these reputations matter.</p>
                                                    </div>-->
                                                </div>

                                                <div class="item mix Conference" data-year="2015">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/Neurocomputing.png" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">A Spatio-Temporal Low-rank Total Variation Approach for Denoising Arterial Spin Labeling MRI Data</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor"><u><b>Ruogu Fang</b></u>,Junzhou Huang, Wen-Ming Luh.</div>
                                                            <div class="pubcite">ISBI'15, The IEEE International Symposium on Biomedical Imaging (ISBI), NYC, USA</div>
                                                            <div class="pubdate">Publication Year: 2015</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Arterial spin labeling MRI (ASL-MRI) can provide quantitative
                                                        signals correlated to the cerebral blood flow and neural
                                                        activity. However, the low signal-to-noise ratio in ASL
                                                        requires repeated acquisitions to improve the signal reliability,
                                                        leading to prolonged scanning time. At fewer repetitions,
                                                        noise and corruptions arise due to motion and physiological
                                                        artifacts, introducing errors into the cerebral blood flow estimation.
                                                        We propose to recover the ASL-MRI data from
                                                        the noisy and corrupted observations at shorter scanning time
                                                        with a spatio-temporal low-rank total variation method. The
                                                        low-rank approximation uses the similarity of the repetitive
                                                        scans, and the total variation regularization considers the local
                                                        spatial consistency. We compare with the state-of-art robust
                                                        M-estimator for ASL cerebral blood flow map estimation.
                                                        Validation on simulated and real data demonstrate the
                                                        robustness of the proposed method at fewer scanning repetitions
                                                        and with random corruption</p>
                                                    </div>
                                                </div>





                                                <!-- 2014-->
                                                <div class="item mix Journal" data-year="2014">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                            <a href="http://www.sciencedirect.com/science/article/pii/S136184151300145X" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>

                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/MIA.gif" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Improving Low-Dose Blood-Brain Barrier Permeability Quantification Using Sparse High-Dose Induced Prior for Patlak Model.</h4>
                                                            <span class="label label-warning">Journal</span>
                                                            <div class="pubauthor"><u><b>Ruogu Fang</b></u>, Kolbeinn Karlsson, Tsuhan Chen, Pina C. Sanelli.</div>
                                                            <div class="pubcite">MedIA'14, Medical Image Analysis, Volume 18, Issue 6, Pages 866-880</div>
                                                            <div class="pubdate">Publication Year: 2014</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Blood-brain barrier permeability (BBBP) measurements extracted from the perfusion computed tomography (PCT) using the
                                                        Patlak model can be a valuable indicator to predict hemorrhagic transformation in patients with acute stroke. Unfortunately, the
                                                        standard Patlak model based PCT requires excessive radiation exposure, which raised attention on radiation safety. Minimizing
                                                        radiation dose is of high value in clinical practice but can degrade the image quality due to the introduced severe noise. The purpose
                                                        of this work is to construct high quality BBBP maps from low-dose PCT data by using the brain structural similarity between
                                                        different individuals and the relations between the high- and low-dose maps. The proposed sparse high-dose induced (shd-Patlak)
                                                        model performs by building a high-dose induced prior for the Patlak model with a set of location adaptive dictionaries, followed
                                                        by an optimized estimation of BBBP map with the prior regularized Patlak model. Evaluation with the simulated low-dose clinical
                                                        brain PCT datasets clearly demonstrate that the shd-Patlak model can achieve more significant gains than the standard Patlak model
                                                        with improved visual quality, higher fidelity to the gold standard and more accurate details for clinical analysis.</p>
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2014">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                            <a href="http://users.cis.fiu.edu/~rfang/projects/ttv.html" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/MICCAI.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Tensor Total-Variation Regularized Deconvolution for Efficient Low-Dose CT Perfusion.</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor"><u><b>Ruogu Fang</b></u>,Pina Sanelli, Shaoting Zhang, Tsuhan Chen.</div>
                                                            <div class="pubcite">MICCAI'14, The 17th Annual International Conference on Medical Image Computing and Computer Assisted Intervention, Boston, USA</div>
                                                            <div class="pubdate">Publication Year: 2014</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Acute brain diseases such as acute stroke and transit ischemic attacks
                                                        are the leading causes of mortality and morbidity worldwide, responsible for 9%
                                                        of total death every year. ‘Time is brain' is a widely accepted concept in acute
                                                        cerebrovascular disease treatment. Efficient and accurate computational framework
                                                        for hemodynamic parameters estimation can save critical time for thrombolytic
                                                        therapy. Meanwhile the high level of accumulated radiation dosage due to
                                                        continuous image acquisition in CT perfusion (CTP) raised concerns on patient
                                                        safety and public health. However, low-radiation will lead to increased noise and
                                                        artifacts which require more sophisticated and time-consuming algorithms for
                                                        robust estimation. We propose a novel efficient framework using tensor totalvariation
                                                        (TTV) regularization to achieve both high efficiency and accuracy in
                                                        deconvolution for low-dose CTP. The method reduces the necessary radiation
                                                        dose to only 8% of the original level and outperforms the state-of-art algorithms
                                                        with estimation error reduced by 40%. It also corrects over-estimation of cerebral
                                                        blood flow (CBF) and under-estimation of mean transit time (MTT), at both normal
                                                        and reduced sampling rate. An efficient</p>
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2014">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/MICCAI.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Anisotropic Tensor Total Variation Regularization For Low Dose Low CT Perfusion Deconvolution.</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor"><u><b>Ruogu Fang</b></u>, Tsuhan Chen, Pina C. Sanelli.</div>
                                                            <div class="pubcite">MICCAI'14, The 17th Annual International Conference on Medical Image Computing and Computer Assisted Intervention, Workshop on Sparsity Techniques in Medical Imaging, Boston, USA</div>
                                                            <div class="pubdate">Publication Year: 2014</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Tensor total variation (TTV) regularized deconvolution has been proposed
                                                        for robust low radiation dose CT perfusion. In this paper, we extended TTV
                                                        algorithm with anisotropic regularization weighting for the temporal and spatial
                                                        dimension. We evaluated TTV algorithm on synthetic dataset for bolus delay,
                                                        uniform region variability and contrast preservation, and on clinical dataset for
                                                        reduced sampling rate with visual and quantitative comparison. The extensive
                                                        experiments demonstrated promising results of TTV compared to baseline and
                                                        state-of-art algorithms in low-dose and low sampling rate CTP deconvolution
                                                        with insensitivity to bolus delay. This work further demonstrates the effectiveness
                                                        and potential of TTV algorithm's clinical usage for cerebrovascular diseases
                                                        with significantly reduced radiation exposure and improved patient safety.</p>
                                                    </div>
                                                </div>

                                                <div class="item mix Thesis" data-year="2014">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/Cornell.png" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Towards Robust Deconvolution in Medical Imaging: Informatics, Diagnosis and Treatment.</h4>
                                                            <span class="label label-danger">PhD Thesis</span>
                                                            <div class="pubauthor"><strong> Ruogu Fang</strong></div>
                                                            <div class="pubcite">School of Electrical and Computer Engineering, Cornell University</div>
                                                            <div class="pubdate">Publication Year: 2014</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Introduction</h4>
                                                        <p>Deconvolution in medical imaging is an inverse process to estimate the
                                                        hemodynamic parameters, such as the blood flow, blood volume and mean
                                                        transit time, from the measurements in the medical images, as illustrated in Figure
                                                        1.1. A traditional machinery employed for estimating these hemodynamic
                                                        parameters is to compute the impulse response function of each voxel independently
                                                        using matrix factorization or curve fitting. For instance, singular value
                                                        decomposition (SVD) is widely used in current commercial softwares to inverse
                                                        the convolution process. </p>
                                                    </div>
                                                </div>

                                                <!-- 2013-->
                                                <div class="item mix Journal" data-year="2013">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                            <a href="http://www.sciencedirect.com/science/article/pii/S1361841513000194" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/MIA.gif" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Towards Robust Deconvolution of Low-Dose Perfusion CT: Sparse Perfusion Deconvolution Using Online Dictionary Learning</h4>
                                                            <span class="label label-warning">Journal</span>
                                                            <div class="pubauthor"><u><b>Ruogu Fang</b></u>, Tsuhan Chen, Pina C. Sanelli.</div>
                                                            <div class="pubcite">MedIA'13, Medical Image Analysis, Volume 17, Issue 4, Pages 417-428(5 Year Impact Factor: 4.512)</div>
                                                            <div class="pubdate">Publication Year: 2013</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Computed tomography perfusion (CTP) is an important functional imaging modality in the evaluation of cerebrovascular diseases,
                                                        particularly in acute stroke and vasospasm. However, the post-processed parametric maps of blood flow tend to be noisy,
                                                        especially in low-dose CTP, due to the noisy contrast enhancement profile and the oscillatory nature of the results generated by
                                                        the current computational methods. In this paper, we propose a robust sparse perfusion deconvolution method (SPD) to estimate
                                                        cerebral blood flow in CTP performed at low radiation dose. We first build a dictionary from high-dose perfusion maps using online
                                                        dictionary learning and then perform deconvolution-based hemodynamic parameters estimation on the low-dose CTP data. Our
                                                        method is validated on clinical data of patients with normal and pathological CBF maps. The results show that we achieve superior
                                                        performance than existing methods, and potentially improve the differentiation between normal and ischemic tissue in the brain.</p>
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2013">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/MICCAI.jpg" class="attachment-medium size-medim wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle"> Tissue-Specific Sparse Deconvolution for Low-Dose CT Perfusion</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor"><u><b>Ruogu Fang</b></u>>, Tsuhan Chen, Pina C. Sanelli</div>
                                                            <div class="pubcite">MICCAI'13, The 16th Annual International Conference on Medical Image Computing and Computer Assisted Intervention, Japan</div>
                                                            <div class="pubdate">Publication Year: 2013</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Sparse perfusion deconvolution has been recently proposed to effectively
                                                        improve the image quality and diagnostic accuracy of low-dose perfusion
                                                        CT by extracting the complementary information from the high-dose perfusion
                                                        maps to restore the low-dose using a joint spatio-temporal model. However the
                                                        low-contrast tissue classes where infarct core and ischemic penumbra usually occur
                                                        in cerebral perfusion CT tend to be over-smoothed, leading to loss of essential
                                                        biomarkers. In this paper, we extend this line of work by introducing tissuespecific
                                                        sparse deconvolution to preserve the subtle perfusion information in the
                                                        low-contrast tissue classes by learning tissue-specific dictionaries for each tissue
                                                        class, and restore the low-dose perfusion maps by joining the tissue segments
                                                        reconstructed from the corresponding dictionaries. Extensive validation on clinical
                                                        datasets of patients with cerebrovascular disease demonstrates the superior
                                                        performance of our proposed method with the advantage of better differentiation
                                                        between abnormal and normal tissue in these patients.</p>
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2013">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="https://www2.securecms.com/ICIP2013/" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/ICIP.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Kinship Classification by Modeling Facial Feature Heredity</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor"><u><b>Ruogu Fang</b></u>, Andrew C. Gallagher, Tsuhan Chen, Alexander Loui</div>
                                                            <div class="pubcite">IEEE International Conference on Image Processing, Melbourne, Australia</div>
                                                            <div class="pubdate">Publication Year: 2013</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                </div>
                                                
                                                <!-- 2012-->
                                                <div class="item mix Conference" data-year="2012">
                                                    <div class="pubmain">
                                                        <div class="pubassets">

                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/MICCAI.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Sparsity-Based Deconvolution of Low-Dose Perfusion CT Using Learned Dictionaries.</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor"><u><b>Ruogu Fang</b></u>, Tsuhan Chen, Pina C. Sanelli.</div>
                                                            <div class="pubcite">MICCAI'12, The 15th Annual International Conference on Medical Image Computing and Computer Assisted Intervention, Nice, France. Lecture Notes in Computer Science Volume 7510, 2012, pp 272-280.</div>
                                                            <div class="pubdate">Publication Year: 2012</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2012">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/IEEEImaging.png" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Sparsity-Based Deconvolution Of Low-Dose Brain Perfusion CT In Subarachnoid Hemorrhage Patients</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor"><u><b>Ruogu Fang</b></u>, Tsuhan Chen, Pina C. Sanelli.</div>
                                                            <div class="pubcite">ISBI'12, The 9th International Symposium on Biomedical Imaging, pp. 872-875, Barcelona, Spain. Oral presentation.</div>
                                                            <div class="pubdate">Publication Year: 2012</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Functional imaging serves as an important supplement to
                                                        anatomical imaging modalities such as MR and CT in
                                                        modern health care. In perfusion CT (CTP), hemodynamic
                                                        parameters are derived from the tracking of the first-pass of
                                                        the contrast bolus entering a tissue region of interest. In
                                                        practice, however, the post-processed parametric maps tend
                                                        to be noisy, especially in low-dose CTP, in part due to the
                                                        noisy contrast enhancement profile and oscillatory nature of
                                                        results generated by current computational methods. In this
                                                        paper, we propose a sparsity-based perfusion parameter
                                                        deconvolution approach that consists of a non-linear
                                                        processing based on sparsity prior in terms of residue
                                                        function dictionaries. Our simulated results from
                                                        numericaldata and experiments in aneurysmal subarachnoid
                                                        hemorrhage patients with clinical vasospasm show that the
                                                        algorithm improves the quality and reduces the noise of the
                                                        perfusion parametric maps in low-dose CTP, compared to
                                                        state-of-the-art methods</p>
                                                    </div>
                                                </div>

                                                <div class="item mix Conference" data-year="2012">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/SPIE.png" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Radiation dose reduction in computed tomography perfusion using spatial-temporal Bayesian methods.</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor"><u><b>Ruogu Fang</b></u>, Ashish Raj, Tsuhan Chen, Pina C. Sanelli</div>
                                                            <div class="pubcite">SPIE'12, In Proceedings of SPIE Medical Imaging, Volume 8313, Paper #831345</div>
                                                            <div class="pubdate">Publication Year: 2012</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>In current computed tomography (CT) examinations, the associated X-ray radiation dose is of significant concern to
                                                        patients and operators, especially CT perfusion (CTP) imaging that has higher radiation dose due to its cine scanning
                                                        technique. A simple and cost-effective means to perform the examinations is to lower the milliampere-seconds (mAs)
                                                        parameter as low as reasonably achievable in data acquisition. However, lowering the mAs parameter will unavoidably
                                                        increase data noise and degrade CT perfusion maps greatly if no adequate noise control is applied during image
                                                        reconstruction. To capture the essential dynamics of CT perfusion, a simple spatial-temporal Bayesian method that uses
                                                        a piecewise parametric model of the residual function is used, and then the model parameters are estimated from a
                                                        Bayesian formulation of prior smoothness constraints on perfusion parameters. From the fitted residual function, reliable
                                                        CTP parameter maps are obtained from low dose CT data. The merit of this scheme exists in the combination of
                                                        analytical piecewise residual function with Bayesian framework using a simpler prior spatial constrain for CT perfusion
                                                        application. On a dataset of 22 patients, this dynamic spatial-temporal Bayesian model yielded an increase in signal-tonoise-ratio
                                                        (SNR) of 78% and a decrease in mean-square-error (MSE) of 40% at low dose radiation of 43mA. </p>
                                                    </div>
                                                </div>

                                                <div class="item mix Patents" data-year="2012">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                            <a href="http://www.google.com/patents/WO2012027259A2?cl=en" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                        <img width="125" src="img/pubs/Patent.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                        <h4 class="pubtitle">System and Method For Interactive Segmentation On Mobile Devices in a Cloud Computing Environment</h4>
                                                        <span class="label label-info">Patents</span>
                                                        <div class="pubauthor"><strong> Ruogu Fang</strong>,Leo Grady, Gianluca Paladini.</div>
                                                        <div class="pubcite">Siemens Corporation. Patent No: US20130272587 A1, WO2012027259 A2, WO2012027259 A3, approved on 4/19/2012</div>
                                                        <div class="pubdate">Publication Year: 2012</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>A mobile device (160) for medical image analysis is disclosed. The mobile device (160) includes a display (162), a communication module (218), a memory (204) configured to store processor-executable instructions (224) and a processor (202) in communication with the display (162), the communication module (218) and the memory (204). The processor (202) being configured to execute the processor-executable instructions (224) to implement a compression routine to generate a compressed representation of a medical image stored in the memory (204), transmit the compressed representation to a remote device (110) via the communication module (218), receive segmented results from the remote device (110), wherein the segmented results are derived from a reconstruction of the compressed representation generated at the remote device (110), and present, via the display (162), a segmented medical image based on the received segmented results.</p>
                                                    </div>
                                                </div>

                                                <!-- 2011-->
                                                <div class="item mix Conference" data-year="2011">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/MICCAI.jpg" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Segmentation of Liver Tumor Using Efficient Global Optimal Tree Metrics Graph Cuts</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor"><u><b>Ruogu Fang</b></u>, Ramin Zabih, Ashish Raj, Tsuhan Chen</div>
                                                            <div class="pubcite">MICCAI'11, Abdominal Imaging, International Conference on Medical Image Computing and Computer Assisted Intervention, pp. 51-59</div>
                                                            <div class="pubdate">Publication Year: 2011</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p> We propose a novel approach that applies global optimal tree-metrics
                                                        graph cuts algorithm on multi-phase contrast enhanced contrast enhanced MRI
                                                        for liver tumor segmentation. To address the difficulties caused by low
                                                        contrasted boundaries and high variability in liver tumor segmentation, we first
                                                        extract a set of features in multi-phase contrast enhanced MRI data and use
                                                        color-space mapping to reveal spatial-temporal information invisible in MRI
                                                        intensity images. Then we apply efficient tree-metrics graph cut algorithm on
                                                        multi-phase contrast enhanced MRI data to obtain global optimal labeling in an
                                                        unsupervised framework. Finally we use tree-pruning method to reduce the
                                                        number of available labels for liver tumor segmentation. Experiments on realworld
                                                        clinical data show encouraging results. This approach can be applied to
                                                        various medical imaging modalities and organs.  </p>
                                                    </div>
                                                </div>

                                                <!-- 2010-->
                                                <div class="item mix Conference" data-year="2010">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                            <a href="https://www.newscientist.com/article/mg21228424.900-facial-recognition-software-spots-family-resemblance/#.U9VZz1ZMYhG" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/IEEE.png" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Towards Computational Models of Kinship Verification.</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor"><u><b>Ruogu Fang</b></u>,  Kevin D. Tang, Noah Snavely, Tsuhan Chen.</div>
                                                            <div class="pubcite">ICIP'10, The 17th IEEE International Conference on Image Processing, Hong Kong. Oral presentation ICIP 2010 Best Paper Award</div>
                                                            <div class="pubdate">Publication Year: 2010</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>We tackle the challenge of kinship verification using novel
                                                        feature extraction and selection methods, automatically
                                                        classifying pairs of face images as “related” or “unrelated”
                                                        (in terms of kinship). First, we conducted a controlled online
                                                        search to collect frontal face images of 150 pairs of public
                                                        figures and celebrities, along with images of their parents or
                                                        children. Next, we propose and evaluate a set of low-level
                                                        image features that for use in this classification problem.
                                                        After selecting the most discriminative inherited facial
                                                        features, we demonstrate a classification accuracy of 70.67%
                                                        on a test set of image pairs using K-Nearest-Neighbors.
                                                        Finally, we present an evaluation of human performance on
                                                        this problem. </p>
                                                    </div>
                                                </div>
                                                
                                                <div class="item mix Conference" data-year="2010">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            <a href="#" class="pubcollapse"><i class="fa fa-expand"></i></a>
                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/IEEE.png" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle">Tree-Metrics Graph Cuts For Brain MRI Segmentation With Tree Cutting.</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor"><u><b>Ruogu Fang</b></u>, Joyce Yu-hsin Chen, Ramin Zabih, Tsuhan Chen.</div>
                                                            <div class="pubcite">WNYIPW'10, IEEE Western New York Image Processing Workshop, pp. 10-13, Rochesester, NY. USA. Oral presentation</div>
                                                            <div class="pubdate">Publication Year: 2010</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p> We tackle the problem of brain MRI image segmentation using
                                                        the tree-metric graph cuts (TM) algorithm, a novel image
                                                        segmentation algorithm, and introduce a “tree-cutting”
                                                        method to interpret the labeling returned by the TM algorithm
                                                        as tissue classification for the input brain MRI image.
                                                        The approach has three steps: 1) pre-processing, which
                                                        generates a tree of labels as input to the TM algorithm; 2) a
                                                        sweep of the TM algorithm, which returns a globally optimal
                                                        labeling with respect to the tree of labels; 3) post-processing,
                                                        which involves running the “tree-cutting” method to generate
                                                        a mapping from labels to tissue classes (GM, WM, CSF),
                                                        producing a meaningful brain MRI segmentation. The TM
                                                        algorithm produces a globally optimal labeling on tree metrics
                                                        in one sweep, unlike conventional methods such as EMS
                                                        and EM-style geo-cuts, which iterate the expectation maximization
                                                        algorithm to find hidden patterns and produce only
                                                        locally optimal labelings. When used with the “tree-cutting”
                                                        method, the TM algorithm produces brain MRI segmentations
                                                        that are as good as the Unified Segmentation algorithm used
                                                        by SPM8, using a much weaker prior. Comparison with the
                                                        current approaches shows that our method is faster and that
                                                        our overall segmentation accuracy is better. </p>
                                                    </div>
                                                </div>

                                                <!-- 2009-->
                                                <div class="item mix Conference" data-year="2009">
                                                    <div class="pubmain">
                                                        <div class="pubassets">

                                                        </div>
                                                        <div class="pubthumbnail">
                                                            <img width="125" src="img/pubs/IEEE.png" class="attachment-medium size-medium wp-post-image" alt="" align="left" hspace="20">
                                                        </div>
                                                        <div class="pubcontent">
                                                            <h4 class="pubtitle"> Adaptive Scale Robust Feature Density Approximation For Visual Object Representation And Tracking</h4>
                                                            <span class="label label-primary">Conference</span>
                                                            <div class="pubauthor"> Chongyang Liu, <strong>Ruogu Fang</strong>, and Nelson H.C.Yung </div>
                                                            <div class="pubcite"> IEEE International Conference on Computer Vision Theory and Applications, Lisboa, Portugal</div>
                                                            <div class="pubdate">Publication Year: 2009</div>
                                                        </div>
                                                        <div class="clearfix"></div>
                                                    </div>
                                                </div>
                                            </div>
                                            <!-- Publications End -->

                                        </div>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>
                </div>

                <!--Teaching-->
                <div id="teaching" class="page">

                    <!--Title-->
                    <div class="pageheader">
                        <div class="headercontent">
                            <div class="section-container">
                                <h2 class="title">Teaching</h2>
                                <div class="row">
                                    <div class="col-md-12">
                                        <p></p>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>
                    <!--/Title-->

                    <div class="pagecontents">

                        <!--Current Teaching-->
                        <div class="section color-1">
                            <div class="section-container">
                                <div class="row">
                                    <div class="title text-center">
                                        <h3>Current Teaching</h3>
                                    </div>
                                    <ul class="ul-dates">
                                        <li>
                                                    <div class="dates">
                                                        <span>2021</span>
                                                        <span>Spring</span>
                                                    </div>
                                                    <div class="content">
                                                        <h4> BME6938 Multimodal Data Mining <a href="https://www.bme.ufl.edu/wp-content/uploads/2018/12/BME6938_Spring_2019_syllabus.pdf" class="tooltips" title="syllabus" target="_blank"><i class="fa fa-cloud-download"></i></a> <a href="https://ufl.instructure.com" class="tooltips" title="canvas" target="_blank"><i class="fa fa-external-link"></i></a></h4>
                                                        <p>Course Objectives: 1. Understand multimodal data mining in the biomedical domain; 2. Understand the concept, approaches, and limitations in analyzing different modalities of biomedical data. 3. Learn to use biomedical data programming libraries and skills to analyze multimodal biomedical data. </p>
                                                     <!--   <p>Instructor Evaluation: 4.87/5.00</p> -->
                                                    </div>
                                        </li>
                                    </ul>
                                </div>
                            </div>
                        </div>
                        <!--/Current Teaching-->

                        <!--Teaching History-->
                        <div class="section color-2">
                            <div class="section-container">
                                <div class="row">
                                    <div class="title text-center">
                                        <h3>Teaching History</h3>
                                    </div>
                                    <ul class="ul-dates-gray">
                                        <li>
                                            <ul class="ul-dates">
                                                <li>
                                            <div class="dates">
                                                <span>2020</span>
                                                <span>Fall</span>
                                            </div>
                                            <div class="content">
                                                <h4> BME3053C Computer Applications for BME <a href="https://www.bme.ufl.edu/wp-content/uploads/2018/10/BME3053C_Fang-1.pdf" class="tooltips" title="syllabus" target="_blank"><i class="fa fa-cloud-download"></i></a> <a href="https://ufl.instructure.com" class="tooltips" title="canvas" target="_blank"><i class="fa fa-external-link"></i></a></h4>
                                <!--            <p>Instructor Evaluation: 4.83/5.00, Course Evaluation: 4.53/5.00</p> -->
                                            </div>
                                        </li>
                                            <li>
                                                    <div class="dates">
                                                        <span>2019</span>
                                                        <span>Spring</span>
                                                    </div>
                                                    <div class="content">
                                                        <h4> BME6938 Multimodal Data Mining <a href="https://www.bme.ufl.edu/wp-content/uploads/2018/12/BME6938_Spring_2019_syllabus.pdf" class="tooltips" title="syllabus" target="_blank"><i class="fa fa-cloud-download"></i></a> <a href="https://ufl.instructure.com" class="tooltips" title="canvas" target="_blank"><i class="fa fa-external-link"></i></a></h4>
                                                        <p>Course Objectives: 1. Understand multimodal data mining in the biomedical domain; 2. Understand the concept, approaches, and limitations in analyzing different modalities of biomedical data. 3. Learn to use biomedical data programming libraries and skills to analyze multimodal biomedical data. </p>
                                                     <!--   <p>Instructor Evaluation: 4.87/5.00</p> -->
                                                    </div>
                                                </li>
                                                <li>
                                                    <div class="dates">
                                                        <span>2019</span>
                                                        <span>Fall</span>
                                                    </div>
                                                    <div class="content">
                                                        <h4> BME3053C Computer Applications for BME <a href="https://www.bme.ufl.edu/wp-content/uploads/2018/10/BME3053C-Syllabi-2.pdf" class="tooltips" title="syllabus" target="_blank"><i class="fa fa-cloud-download"></i></a> <a href="https://ufl.instructure.com" class="tooltips" title="canvas" target="_blank"><i class="fa fa-external-link"></i></a></h4>
                                                    </div>
                                                </li>
                                                <li>
                                                    <div class="dates">
                                                        <span>2018</span>
                                                        <span>Fall</span>
                                                    </div>
                                                    <div class="content">
                                                        <h4> BME3053C Computer Applications for BME <a href="https://www.bme.ufl.edu/course/computer-applications-for-bme-bme-3053c/" class="tooltips" title="syllabus" target="_blank"><i class="fa fa-cloud-download"></i></a> <a href="https://ufl.instructure.com" class="tooltips" title="canvas" target="_blank"><i class="fa fa-external-link"></i></a></h4>
                                                     <!--   <p>Instructor Evaluation: 4.00/5.00</p> -->
                                                    </div>
                                                </li>
                                                <li>
                                                    <div class="dates">
                                                        <span>2017</span>
                                                        <span>Fall</span>
                                                    </div>
                                                    <div class="content">
                                                        <h4> BME3053C Computer Applications for BME <a href="https://www.bme.ufl.edu/course/computer-applications-for-bme-bme-3053c/" class="tooltips" title="syllabus" target="_blank"><i class="fa fa-cloud-download"></i></a> <a href="https://ufl.instructure.com" class="tooltips" title="canvas" target="_blank"><i class="fa fa-external-link"></i></a></h4>
                                                        <p>The objectives this course are: 1) Develop a proficiency in the use of computer programming (specifically, MATLAB) to analyze biomedical measurements. 2) Develop an understanding of biomedical engineering problems that require quantitative analysis and visualization.</p>
                                                    </div>
                                                </li>
                                            </ul>
                                            <div class="dates">
                                                <span>2017</span>
                                                <span>Spring</span>
                                            </div>
                                            <div class="content">
                                                <h4> CAP 5771 Principles of Data Mining</h4>
                                            </div>
                                        </li>

                                        <li>
                                            <div class="dates">
                                                <span>2017</span>
                                                <span>Spring</span>
                                            </div>
                                            <div class="content">
                                                <h4> CAP 4770 Introduction to Data Mining</h4>
                                            </div>
                                        </li>

                                        <li>
                                            <div class="dates">
                                                <span>2016</span>
                                                <span>Fall</span>
                                            </div>
                                            <div class="content">
                                                <h4> CAP 4770 Introduction to Data Mining</h4>
                                            <!--    <p>Course Evaluation: 4.24/5.00, Instructor Evaluation: 4.34/5.00</p> -->
                                            </div>
                                        </li>

                                        <li>
                                            <div class="dates">
                                                <span>2016</span>
                                                <span>Spring</span>
                                            </div>
                                            <div class="content">
                                                <h4>CAP 5610 Machine Learning</h4>
                                            <!--    <p>Course Evaluation: 4.46/5.00, Instructor Evaluation: 4.39/5.00</p> -->

                                            </div>
                                        </li>

                                        <li>
                                            <div class="dates">
                                                <span>2015</span>
                                                <span>Fall</span>
                                            </div>
                                            <div class="content">
                                                <h4> CAP 4770 Introduction to Data Mining</h4>
                                                <p>Data Mining is the nontrivial extraction of implicit, previously unknown, and potentially useful information from data. It has gradually matured as a discipline merging ideas from statistics, machine learning, database and etc. This is an introductory course for junior/senior computer science undergraduate students on the topic of Data Mining. Topics include data mining applications, data preparation, data reduction and various data mining techniques (such as association, clustering, classification, anomaly detection).</p>
                                          <!--      <p>Course Evaluation: 4.53/5.00, Instructor Evaluation: 4.57/5.00</p> -->
                                            </div>
                                        </li>

                                        <li>
                                            <div class="dates">
                                                <span>2015</span>
                                                <span>Spring </span>
                                            </div>
                                            <div class="content">
                                                <h4>CAP 5610 Machine Learning</h4>
                                                <p>Machine learning is concerned with the question of how to make computers learn from experience. The ability to learn is not only central to most aspects of intelligent behavior, but machine learning techniques have become key components of many software systems. For examples, machine learning techniques are used to create spam filters, to analyze customer purchase data, to understand natural language, or to detect fraudulent credit card transactions.</p>
                                            </div>
                                        </li>
                                    </ul>
                                </div>
                            </div>
                        </div>
                    </div>
                </div>
                <!--/Teaching-->

                <!--Software-->
                <div id="software" class="page">
                    <div class="pageheader">
                        <div class="headercontent">
                            <div class="section-container">
                                <h2 class="title">Software</h2>
                                <div class="row">
                                    <div class="col-md-12">
                                        <p></p>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>
                    <div class="pagecontents">
                        <div class="section color-1" id="filters">
                            <div class="section-container">
                                <div class="row">
                                    <div class="col-md-3">
                                        <h3>Filter by type:</h3>
                                    </div>
                                    <div class="col-md-6">
                                        <select id="cd-dropdown" name="cd-dropdown" class="cd-select">
                                            <option class="filter" value="all" selected>All</option>
                                            <option class="filter" value="Code">Code</option>
                                            <option class="filter" value="Data">Data</option>
                                        </select>
                                    </div>
                                </div>
                            </div>
                        </div>
                        <div class="section color-2" id="pub-grid">
                            <div class="section-container">
                                <div class="row">
                                    <div class="col-md-12">
                                        <div class="pitems">
                                            <div class="item mix Code" data-year="2019">
                                                <div class="pubmain">
                                                    <div class="pubassets">
                                                        <a href="#" class="pubcollapse"><i class="fa fa-expand"></i></a>
                                                        <a href="https://github.com/cswin/AWC" class="tooltips" title="Github" target="_blank"><i class="fa fa-github"></i></a>
                                                    </div>
                                                    <h4 class="pubtitle">Domain Shift</h4>
                                                    <div class="pubcite">
                                                        <span class="label label-primary">Code</span>Adversarial Discriminative Adaptation and Ensembling Collaborative Learning for Domain Shift in Unsupervised Optic Disc and Cup Segmentation
                                                    </div>
                                                </div>
                                                <div class="pubdetails">
                                                    <h4>Citation</h4>
                                                    <ul>
                                                        <li>Peng Liu, Bin Kong, Zhongyu Li, Shaoting Zhang, and Ruogu Fang, 2019. CFEA: Collaborative Feature Ensembling Adaptation for Domain Adaptation in Unsupervised Optic Disc and Cup Segmentation. Medical Image Analysis and Computer Assisted Intervention.</li>
                                                    </ul>
                                                </div>
                                            </div>

                                            <div class="item mix Code" data-year="2016">
                                                <div class="pubmain">
                                                    <div class="pubassets">
                                                        <a href="#" class="pubcollapse"><i class="fa fa-expand"></i></a>
                                                        <a href="https://github.com/ruogufang/pct" class="tooltips" title="Github" target="_blank"><i class="fa fa-github"></i></a>
                                                    </div>
                                                        <h4 class="pubtitle">CT Perfusion Toolbox</h4>
                                                    <div class="pubcite">
                                                        <span class="label label-primary">Code</span>Toolbox for CT perfusion image processing and computation, including image loading, preprocessing, reconstruction, quantification, and example code.
                                                    </div>
                                                </div>
                                                <div class="pubdetails">
                                                    <h4>Citation</h4>
                                                    <ul>
                                                        <li>Fang, R., Chen, T. and Sanelli, P.C., 2013. Towards robust deconvolution of low-dose perfusion CT: Sparse perfusion deconvolution using online dictionary learning. Medical image analysis, 17(4), pp.417-428.</li>
                                                        <li>Fang, R., Zhang, S., Chen, T. and Sanelli, P.C., 2015. Robust low-dose CT perfusion deconvolution via tensor total-variation regularization. IEEE transactions on medical imaging, 34(7), pp.1533-1548.</li>
                                                    </ul>
                                                </div>
                                            </div>

                                            <div class="item mix Code" data-year="2015">
                                                <div class="pubmain">
                                                    <div class="pubassets">
                                                        <a href="#" class="pubcollapse"><i class="fa fa-expand"></i></a>
                                                        <a href="https://github.com/ruogufang/ttv" class="tooltips" title="Github" target="_blank"><i class="fa fa-github"></i></a>
                                                    </div>
                                                    <h4 class="pubtitle">Tensor Total Variation</h4>
                                                    <div class="pubcite">
                                                        <span class="label label-primary">Code</span>Toolbox for Tensor Total Variation quantfication of CT perfusion parameters in low-dose CT perfusion.
                                                    </div>
                                                </div>
                                                <div class="pubdetails">
                                                    <h4>Citation</h4>
                                                    <ul>
                                                        <li>Fang, R., Zhang, S., Chen, T. and Sanelli, P.C., 2015. Robust low-dose CT perfusion deconvolution via tensor total-variation regularization. IEEE transactions on medical imaging, 34(7), pp.1533-1548.</li>
                                                        <li>Fang, R., Sanelli, P.C., Zhang, S. and Chen, T., 2014, September. Tensor total-variation regularized deconvolution for efficient low-dose CT perfusion. In International Conference on Medical Image Computing and Computer-Assisted Intervention (pp. 154-161). Springer, Cham.</li>
                                                    </ul>
                                                </div>
                                            </div>

                                            <div class="item mix Code" data-year="2014">
                                                <div class="pubmain">
                                                    <div class="pubassets">
                                                        <a href="#" class="pubcollapse"><i class="fa fa-expand"></i></a>
                                                        <a href="https://github.com/ruogufang/SPD-Patlak" class="tooltips" title="Github" target="_blank"><i class="fa fa-github"></i></a>
                                                    </div>
                                                    <h4 class="pubtitle">Sparse High-Dose Induced Patlak Model for Permeability </h4>
                                                    <div class="pubcite">
                                                        <span class="label label-primary">Code</span>Sparse high-dose induced Patlak model for Permeability quantification in low-dose CT perfusion.
                                                    </div>
                                                </div>
                                                <div class="pubdetails">
                                                    <h4>Citation</h4>
                                                    <ul>
                                                        <li>Fang, R., Karlsson, K., Chen, T. and Sanelli, P.C., 2014. Improving low-dose blood–brain barrier permeability quantification using sparse high-dose induced prior for Patlak model. Medical image analysis, 18(6), pp.866-880.</li>
                                                    </ul>
                                                </div>
                                            </div>

                                            <div class="item mix Code" data-year="2013">
                                                <div class="pubmain">
                                                    <div class="pubassets">
                                                        <a href="#" class="pubcollapse"><i class="fa fa-expand"></i></a>
                                                        <a href="https://github.com/ruogufang/SPD" class="tooltips" title="Github" target="_blank"><i class="fa fa-github"></i></a>
                                                    </div>
                                                    <h4 class="pubtitle">Sparse Perfusion Deconvolution Toolbox </h4>
                                                    <div class="pubcite">
                                                        <span class="label label-primary">Code</span>Sparse perfusion deconvolution toolbox using dictionary learning and sparse coding for low-dose CT perfusion parameters quantification.
                                                    </div>
                                                </div>
                                                <div class="pubdetails">
                                                <h4>Citation</h4>
                                                    <ul>
                                                        <li>Fang, R., Chen, T. and Sanelli, P.C., 2012, October. Sparsity-based deconvolution of low-dose perfusion CT using learned dictionaries. In International Conference on Medical Image Computing and Computer-Assisted Intervention (pp. 272-280). Springer Berlin Heidelberg.</li>
                                                        <li>Fang, R., Chen, T. and Sanelli, P.C., 2013. Towards robust deconvolution of low-dose perfusion CT: Sparse perfusion deconvolution using online dictionary learning. Medical image analysis, 17(4), pp.417-428.</li>
                                                    </ul>
                                                </div>
                                            </div>

                                            <div class="item mix Data" data-year="2013">
                                                <div class="pubmain">
                                                    <div class="pubassets">
                                                        <a href="#" class="pubcollapse"><i class="fa fa-expand"></i></a>
                                                        <a href="http://chenlab.ece.cornell.edu/projects/KinshipClassification/" class="tooltips" title="webpage" target="_blank"><i class="fa fa-external-link"></i></a>
                                                    </div>
                                                    <h4 class="pubtitle">Kinship Classification Dataset</h4>
                                                    <div class="pubcite">
                                                        <span class="label label-primary">Data</span>Family101 Dataset: first large-scale dataset of families across several generations. It contains 101 different family with distinct family names, including 206 nuclear families, 607 individuals, with 14,816 images.
                                                    </div>
                                                </div>
                                                <div class="pubdetails">
                                                <h4>Citation</h4>
                                                <ul>
                                                    <li>Ruogu Fang, Andrew C. Gallagher, Tsuhan Chen, Alexander Loui. Kinship Classification by Modeling Facial Feature Heredity. IEEE International Conference on Image Processing, Australia, 2013 (ICIP '13)</li>
                                                </ul>

                                                </div>
                                            </div>

                                            <div class="item mix Data" data-year="2013">
                                                <div class="pubmain">
                                                    <div class="pubassets">
                                                        <a href="#" class="pubcollapse"><i class="fa fa-expand"></i></a>
                                                        <a href="http://chenlab.ece.cornell.edu/projects/KinshipVerification/" class="tooltips" title="webpage" target="_blank"><i class="fa fa-external-link"></i></a>
                                                    </div>
                                                    <h4 class="pubtitle">Kinship Verification Dataset</h4>
                                                    <div class="pubcite">
                                                        <span class="label label-primary">Data</span>Cornell Kinship Verification Dataset (143 pairs of parents and children)
                                                    </div>
                                                </div>
                                                <div class="pubdetails">
                                                <h4>Citation</h4>
                                                    <ul>
                                                        <li>Ruogu Fang, Kevin D. Tang, Noah Snavely, Tsuhan Chen. Towards Computational Models of Kinship Verification. IEEE International Conference on Image Processing, Hong Kong, September 2010 (ICIP '10)</li>
                                                    </ul>
                                                </div>
                                            </div>
                                        </div>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>
                </div>
                <!--/Software-->

                <!--Gallery-->
                <div id="gallery" class="page">
                    <div class="pagecontents">
                        <div class="section color-3" id="gallery-header">
                            <div class="section-container">
                                <div class="row">
                                    <div class="col-md-3">
                                        <h2>Gallery</h2>
                                    </div>
                                    <div class="col-md-9">
                                        <p></p>
                                    </div>
                                </div>
                            </div>
                        </div>
                        <div class="section color-3" id="gallery-large">
                            <div class="section-container">
                                <ul id="grid" class="grid">
                                    <li>
                                        <div>
                                            <img alt="image" src="img/gallery/award-2.jpg">
                                            <a href="img/gallery/award-2.jpg" class="popup-with-move-anim">
                                            <div class="over">
                                                <div class="comein">
                                                    <h3>SBMT Annual Meeting</h3>
                                                    <div class="comein-bg"></div>
                                                </div>
                                            </div>
                                            </a>
                                        </div>
                                    </li>
                                    <li>
                                        <div>
                                            <img alt="image" src="img/gallery/smilegrouppresent.jpg">
                                            <a href="img/gallery/smilegrouppresent.jpg" class="popup-with-move-anim">
                                            <div class="over">
                                                <div class="comein">
                                                    <h3>SMILE Group</h3>
                                                    <p>Welcome to join us!</p>
                                                    <div class="comein-bg"></div>
                                                </div>
                                            </div>
                                            </a>
                                        </div>
                                    </li>
                                    <li>
                                        <div>
                                            <img alt="image" src="img/gallery/BME3053CPoster.jpg">
                                            <a href="img/gallery/BME3053CPoster.jpg" class="popup-with-move-anim">
                                            <div class="over">
                                                <div class="comein">
                                                    <h3>Biomedical Computer Application Course Poster Presentation</h3>
                                                    <div class="comein-bg"></div>
                                                </div>
                                            </div>
                                            </a>
                                        </div>
                                    </li>
                                    <li>
                                        <div>
                                            <img alt="image" src="img/gallery/secplaceComApp.jpg">
                                            <a href="img/gallery/secplaceComApp.jpg" class="popup-with-move-anim">
                                            <div class="over">
                                                <div class="comein">
                                                    <h3>Second Place in Computer Application Presentation</h3>
                                                    <p></p>
                                                    <div class="comein-bg"></div>
                                                </div>
                                            </div>
                                            </a>
                                        </div>
                                    </li>
                                    <li>
                                        <div>
                                            <img alt="image" src="img/gallery/award-1.jpg">
                                            <a href="img/gallery/award-1.jpg" class="popup-with-move-anim">
                                            <div class="over">
                                                <div class="comein">
                                                    <p>Robin Sidhu Memorial Young Scientist Award</p>
                                                    <div class="comein-bg"></div>
                                                </div>
                                            </div>
                                            </a>
                                        </div>
                                    </li>
                                    <li>
                                        <div>
                                            <img alt="image" src="img/gallery/firstplaceComApp.jpg">
                                            <a href="img/gallery/firstplaceComApp.jpg" class="popup-with-move-anim">
                                            <div class="over">
                                                <div class="comein">
                                                    <h3>First Place Computer Application Presentation</h3>
                                                    <p></p>
                                                    <div class="comein-bg"></div>
                                                </div>
                                            </div>
                                            </a>
                                        </div>
                                    </li>
                                    <li>
                                        <div>
                                            <img alt="image" src="img/gallery/2016spring-mlcourse.jpg">
                                            <a href="img/gallery/2016spring-mlcourse.jpg" class="popup-with-move-anim">
                                            <div class="over">
                                                <div class="comein">
                                                    <h3>Machine Learning Course</h3>
                                                    <div class="comein-bg"></div>
                                                </div>
                                            </div>
                                            </a>
                                        </div>
                                    </li>
                                    <li>
                                        <div>
                                            <img alt="image" src="img/gallery/firstplaceML.jpg">
                                            <a href="img/gallery/firstplaceML.jpg" class="popup-with-move-anim">
                                            <div class="over">
                                                <div class="comein">
                                                    <h3>First Place Machine Learning Presentation</h3>
                                                    <div class="comein-bg"></div>
                                                </div>
                                            </div>
                                            </a>
                                        </div>
                                    </li>
                                    <li>
                                        <div>
                                            <img alt="image" src="img/gallery/goodteam.jpg">
                                            <a href="img/gallery/goodteam.jpg" class="popup-with-move-anim">
                                            <div class="over">
                                                <div class="comein">
                                                    <h3>Third Place Machine Learning Course</h3>
                                                    <div class="comein-bg"></div>
                                                </div>
                                            </div>
                                            </a>
                                        </div>
                                    </li>
                                </ul>
                            </div>
                        </div>
                    </div>
                </div>
                <!--/Gallery-->

                <!--Join-->
                <div id="join" class="page">
                    <div class="pageheader">
                        <div class="headercontent">
                            <div class="section-container">
                                <h2 class="title">Join</h2>
                                <div class="row">
                                    <p><b>We are now actively recruiting! We have multiple fully funded PhD positions open on artificial intelligence in medicine, brain-inspired AI, and AI for neurological diseases.</b></p> 
                                    <p>We are constantly seeking self-motivated, creative, and dedicated undergraduate, master, and Ph.D. students from diverse background into Artificial Intelligence in Medical Data Science research.</p>
                                </div>
                            </div>
                        </div>
                    </div>
                    <div class="pagecontents">
                        <div class="section fixedbg parallax contact-office">
                            <div class="section-container">
                                <div class="row">
                                    <div class="col-md-7 col-md-offset-1">
                                        <h2 class="title">How to apply</h2>
                                            <p>Please send Dr. Fang an email including your CV, transcript, and a brief research statement of why you want to join SMILE lab as an undergraduate researcher, what related research projects you have been involved, and why you think you will be a good undergraduate researcher in our lab.</p>
                                        <p>We look forward to your smile at SMILE Lab to make your day!</p>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>
                </div>
                <!--/Join-->

                <!--Contact-->
                <div id="contact" class="page">
                    <div class="pageheader">
                        <div class="headercontent">
                            <div class="section-container">
                                <h2 class="title">Contact & Meet Me</h2>
                                <div class="row">
                                    <div class="col-md-8">
                                        <p>I would be happy to talk to you if you need my assistance in your research, have intention for collaboration, or need technology translation for your company. Also, welcome to join my SMILE research group. Email is the best way to get in touch with me. Please feel free to contact me!</p>
                                    </div>
                                    <div class="col-md-4">
                                        <ul class="list-unstyled">
                                            <li>
                                                <strong><i class="fa fa-phone"></i>&nbsp;&nbsp;</strong>
                                                <span>office: (352) 294-1375</span>
                                            </li>
                                            <li>
                                                <strong><i class="fa fa-envelope"></i>&nbsp;&nbsp;</strong>
                                                <span>ruogu.fang@bme.ufl.edu</span>
                                            </li>
                                            <li>
                                                <!-- <span><a href="http://smile.cs.fiu.edu" target="_blank">SMILE Research group</a></span> -->
                                            </li>
                                        </ul>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>
                    <div class="pagecontents">
                        <div class="section fixedbg parallax contact-office">
                            <div class="section-container">
                                <div class="row">
                                    <div class="col-md-7 col-md-offset-1">
                                        <h2 class="title">At My Office</h2>
                                        <p>J287 Biomedical Science Building<br>University of Florida<br>1275 Center Dr.<br>Gainesville, FL, 32611.</p>
                                        <p></p>
                                    </div>
                                    <div class="col-md-3 text-center hidden-xs hidden-sm">
                                        <i class="fa fa-coffee icon-huge"></i>
                                    </div>
                                </div>
                            </div>
                        </div>
                        <!--
                        <div class="section color-1">
                            <div class="section-container">
                                <div class="row">
                                    <div class="col-md-4 text-center hidden-xs hidden-sm">
                                        <i class="fa fa-stethoscope icon-huge"></i>
                                    </div>
                                    <div class="col-md-8">
                                        <h2 class="title">At My Work</h2>
                                        <p>You can find me at my Work located at Stanford University Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
                                        <p>I am at my office every day from 7:00 until 10:00 am, but you may consider a call to fix an appointment.</p>
                                    </div>
                                </div>
                            </div>
                        </div>
                        <div class="section fixedbg parallax contact-lab">
                            <div class="section-container">
                                <div class="row">
                                   <div class="col-md-7 col-md-offset-1">
                                        <h2 class="title">At My Lab</h2>
                                        <p>You can find me at my office located at Stanford University Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
                                        <p>I am at my office every day from 7:00 until 10:00 am, but you may consider a call to fix an appointment.</p>
                                    </div>
                                    <div class="col-md-3 text-center hidden-xs hidden-sm">
                                        <i class="fa fa-superscript icon-huge"></i>
                                    </div>
                                </div>
                            </div>
                        </div>-->
                    </div>
                </div>
                <div id="overlay"></div>
            </div>
        </div>
    </body>
</html>
